
  <html>
  <head>
    <style>
table {
  font-family: arial, sans-serif;
  border-collapse: collapse;
  width: 100%;
}

td, th {
  border: 1px solid #cccccc;
  text-align: left;
  padding: 8px;
}

</style>
</head>
<body>
<table>
<tr><th>ORF ID</th><th>Gene name</th><th width="300">Function</th><td><table><tr><th width="200">Image</th><th width="100">PDB ID</th><th width="500">Structure Title</th><th width=200>Authors</th></tr></table></td></tr><tr><td>MSMEG_0060</td><td>eccB1</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/cy/5cyu/5cyu_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5CYU">5CYU</a></td><td width="500"> Structures of EccB1 and EccD1 from the core complex of the mycobacterial ESX-1 type VII secretion system.</td><td width=200><ul><li>Wagner, J.M.</li><li>Chan, S.</li><li>Evans, T.J.</li><li>Kahng, S.</li><li>Kim, J.</li><li>Arbing, M.A.</li><li>Eisenberg, D.</li><li>Korotkov, K.V.</li></ul></td></tr></table></td></tr><tr><td>Rv3869</td><td>eccB1</td><td width="300">Unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/cy/5cyu/5cyu_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5CYU">5CYU</a></td><td width="500"> Structures of EccB1 and EccD1 from the core complex of the mycobacterial ESX-1 type VII secretion system.</td><td width=200><ul><li>Wagner, J.M.</li><li>Chan, S.</li><li>Evans, T.J.</li><li>Kahng, S.</li><li>Kim, J.</li><li>Arbing, M.A.</li><li>Eisenberg, D.</li><li>Korotkov, K.V.</li></ul></td></tr></table></td></tr><tr><td>Rv1102c</td><td>mazF3</td><td width="300">Sequence-specific mRNA cleavage</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/cc/5cca/5cca_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5CCA">5CCA</a></td><td width="500"> Crystal structure of Mtb toxin</td><td width=200><ul><li>Cascio, D.</li><li>Arbing, M.</li><li>de Serrano, V.</li><li>Eisenberg, D.</li><li>Miallau, L.</li><li>TB Structural Genomics Consortium (TBSGC)</li></ul></td></tr></table></td></tr><tr><td>Rv1404</td><td>Rv1404</td><td width="300">Involved in transcriptional mechanism</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ny/2nyx/2nyx_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2NYX">2NYX</a></td><td width="500"> Crystal structure of RV1404 from Mycobacterium tuberculosis</td><td width=200><ul><li>Yu, M.</li><li>Bursey, E.H.</li><li>Radhakannan, R.</li><li>Kim, C.-Y.</li><li>Kaviratne, T.</li><li>Woodruff, T.</li><li>Segelke, B.W.</li><li>Lekin, T.</li><li>Toppani, D.</li><li>Terwilliger, T.C.</li><li>Hung, L.-W.</li></ul></td></tr></table></td></tr><tr><td>Rv0265c</td><td>Rv0265c</td><td width="300">Thought to be involved in iron transport across the membrane (import).</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/pm/4pm4/4pm4_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4PM4">4PM4</a></td><td width="500"> Structure of a putative periplasmic iron siderophore binding protein (Rv0265c) from Mycobacterium tuberculosis H37Rv</td><td width=200><ul><li>Arbing, M.A.</li><li>Chan, S.</li><li>Tran, N.</li><li>Kuo, E.</li><li>Lu, J.</li><li>Harris, L.R.</li><li>Zhou, T.T.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Mb3932c</td><td>Mb3932c</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/o6/4o6g/4o6g_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4O6G">4O6G</a></td><td width="500"> 1.55 angstrom resolution X-ray crystal structure of Rv3902c from Mycobacterium tuberculosis.</td><td width=200><ul><li>Reddy, B.G.</li><li>Moates, D.B.</li><li>Kim, H.B.</li><li>Green, T.J.</li><li>Kim, C.Y.</li><li>Terwilliger, T.C.</li><li>Delucas, L.J.</li></ul></td></tr></table></td></tr><tr><td>Rv3902c</td><td>Rv3902c</td><td width="300">Unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/o6/4o6g/4o6g_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4O6G">4O6G</a></td><td width="500"> 1.55 angstrom resolution X-ray crystal structure of Rv3902c from Mycobacterium tuberculosis.</td><td width=200><ul><li>Reddy, B.G.</li><li>Moates, D.B.</li><li>Kim, H.B.</li><li>Green, T.J.</li><li>Kim, C.Y.</li><li>Terwilliger, T.C.</li><li>Delucas, L.J.</li></ul></td></tr></table></td></tr><tr><td>Rv0632c</td><td>echA3</td><td width="300">Could possibly oxidize fatty acids using specific components [catalytic activity: (3S)-3-hydroxyacyl-CoA = trans-2(or 3)-enoyl-CoA + H(2)O].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/fn/4fnd/4fnd_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4FND">4FND</a></td><td width="500"> Crystal structure and mechanism of the prokaryotic enoyl CoA isomerase</td><td width=200><ul><li>Bruning, J.B.</li><li>Gao, N.</li><li>Hernandez, E.D.</li><li>Li, H.</li><li>Dang, N.</li><li>Hung, L.W.</li><li>Moran, S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/fn/4fnb/4fnb_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4FNB">4FNB</a></td><td width="500"> Crystal Structure and Mechanism of the Prokaryotic Enoyl CoA Isomerase</td><td width=200><ul><li>Bruning, J.B.</li><li>Gao, N.</li><li>Hernandez, E.D.</li><li>Li, H.</li><li>Dang, N.</li><li>Hung, L.W.</li><li>Moran, S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/fn/4fn8/4fn8_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4FN8">4FN8</a></td><td width="500"> Crystal structure and mechanism of the prokaryotic enoyl CoA isomerase (ECI)</td><td width=200><ul><li>Bruning, J.B.</li><li>Gao, N.</li><li>Hernandez, E.D.</li><li>Li, H.</li><li>Dang, N.</li><li>Hung, L.W.</li><li>Moran, S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/fn/4fn7/4fn7_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4FN7">4FN7</a></td><td width="500"> "Crystal Structure and Mechanism of the Prokaryotic Enoyl-CoA Isomerase (ECI)"</td><td width=200><ul><li>Bruning, J.B.</li><li>Gao, N.</li><li>Hernandez, E.D.</li><li>Li, H.</li><li>Dang, N.</li><li>Hung, L.W.</li><li>Moran, S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv0632c</td><td>echA3</td><td width="300">Could possibly oxidize fatty acids using specific components [catalytic activity: (3S)-3-hydroxyacyl-CoA = trans-2(or 3)-enoyl-CoA + H(2)O].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/fn/4fnd/4fnd_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4FND">4FND</a></td><td width="500"> Crystal structure and mechanism of the prokaryotic enoyl CoA isomerase</td><td width=200><ul><li>Bruning, J.B.</li><li>Gao, N.</li><li>Hernandez, E.D.</li><li>Li, H.</li><li>Dang, N.</li><li>Hung, L.W.</li><li>Moran, S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/fn/4fnb/4fnb_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4FNB">4FNB</a></td><td width="500"> Crystal Structure and Mechanism of the Prokaryotic Enoyl CoA Isomerase</td><td width=200><ul><li>Bruning, J.B.</li><li>Gao, N.</li><li>Hernandez, E.D.</li><li>Li, H.</li><li>Dang, N.</li><li>Hung, L.W.</li><li>Moran, S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/fn/4fn8/4fn8_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4FN8">4FN8</a></td><td width="500"> Crystal structure and mechanism of the prokaryotic enoyl CoA isomerase (ECI)</td><td width=200><ul><li>Bruning, J.B.</li><li>Gao, N.</li><li>Hernandez, E.D.</li><li>Li, H.</li><li>Dang, N.</li><li>Hung, L.W.</li><li>Moran, S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/fn/4fn7/4fn7_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4FN7">4FN7</a></td><td width="500"> "Crystal Structure and Mechanism of the Prokaryotic Enoyl-CoA Isomerase (ECI)"</td><td width=200><ul><li>Bruning, J.B.</li><li>Gao, N.</li><li>Hernandez, E.D.</li><li>Li, H.</li><li>Dang, N.</li><li>Hung, L.W.</li><li>Moran, S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv1070c</td><td>echA8</td><td width="300">Could possibly oxidize fatty acids using specific components [catalytic activity: (3S)-3-hydroxyacyl-CoA = trans-2(or 3)-enoyl-CoA + H(2)O].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/fj/4fjw/4fjw_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4FJW">4FJW</a></td><td width="500"> Crystal Structure and Mechanism of the Prokaryotic Crotonase</td><td width=200><ul><li>Bruning, J.B.</li><li>Delgado, E.</li><li>Ghosh, S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/q0/3q0j/3q0j_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3Q0J">3Q0J</a></td><td width="500"> Crystal Structure and Mechanism of the Prokaryotic Crotonase</td><td width=200><ul><li>Bruning, J.B.</li><li>Delgado, E.</li><li>Ghosh, S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/pz/3pzk/3pzk_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3PZK">3PZK</a></td><td width="500"> Crystal Structure of the Prokaryotic Crotonase</td><td width=200><ul><li>Bruning, J.B.</li><li>Delgado, E.</li><li>Ghosh, S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/q0/3q0g/3q0g_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3Q0G">3Q0G</a></td><td width="500"> Crystal Structure of the Prokaryotic Crotonase</td><td width=200><ul><li>Bruning, J.B.</li><li>Delgado, E.</li><li>Ghosh, S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv1070c</td><td>echA8</td><td width="300">Could possibly oxidize fatty acids using specific components [catalytic activity: (3S)-3-hydroxyacyl-CoA = trans-2(or 3)-enoyl-CoA + H(2)O].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/fj/4fjw/4fjw_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4FJW">4FJW</a></td><td width="500"> Crystal Structure and Mechanism of the Prokaryotic Crotonase</td><td width=200><ul><li>Bruning, J.B.</li><li>Delgado, E.</li><li>Ghosh, S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/q0/3q0j/3q0j_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3Q0J">3Q0J</a></td><td width="500"> Crystal Structure and Mechanism of the Prokaryotic Crotonase</td><td width=200><ul><li>Bruning, J.B.</li><li>Delgado, E.</li><li>Ghosh, S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/pz/3pzk/3pzk_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3PZK">3PZK</a></td><td width="500"> Crystal Structure of the Prokaryotic Crotonase</td><td width=200><ul><li>Bruning, J.B.</li><li>Delgado, E.</li><li>Ghosh, S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/q0/3q0g/3q0g_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3Q0G">3Q0G</a></td><td width="500"> Crystal Structure of the Prokaryotic Crotonase</td><td width=200><ul><li>Bruning, J.B.</li><li>Delgado, E.</li><li>Ghosh, S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv3841</td><td>bfrB</td><td width="300">Involved in iron storage; ferritin is an intracellular molecule that stores iron in a soluble  nontoxic  readily available form. The functional molecule  which is composed of 24 chains  is roughly spherical and contains a central cavity in which the polymeric ferric iron core is deposited.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/un/3uno/3uno_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3UNO">3UNO</a></td><td width="500"> Mycobacterium tuberculosis ferritin homolog, BfrB</td><td width=200><ul><li>McMath, L.M.</li><li>Contreras, H.</li><li>Goulding, C.W.</li></ul></td></tr></table></td></tr><tr><td>MSMEG_6760</td><td>MSMEG_6760</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ui/3uid/3uid_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3UID">3UID</a></td><td width="500"> Crystal structure of the toxin Msmeg_6760, the structural homolog of Mycobacterium tuberculosis Rv2035, a novel type II toxin involved in the hypoxic response.</td><td width=200><ul><li>Bajaj, R.A.</li><li>Arbing, M.A.</li><li>Shin, A.</li><li>Cascio, D.</li><li>Miallau, L.</li></ul></td></tr></table></td></tr><tr><td>Rv3582c</td><td>ispD</td><td width="300">Involved in the deoxyxylulose-5-phosphate pathway (DXP) of isoprenoid biosynthesis (at the third step). Catalyzes the formation of 4-diphosphocytidyl-2C-methyl-D-erythritol (CDP-me) from CTP and 2C-methyl-D-erythritol 4-phosphate (MEP).</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/q8/3q80/3q80_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3Q80">3Q80</a></td><td width="500"> Crystal Structure of Mycobacterium tuberculosis 2-C-methyl-D-erythritol 4-phosphate cytidyltransferase (IspD): a candidate antitubercular drug target</td><td width=200><ul><li>Reddy, M.C.M.</li><li>Bruning, J.B.</li><li>Thurman, C.</li><li>Ioerger, T.R.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/q7/3q7u/3q7u_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3Q7U">3Q7U</a></td><td width="500"> Crystal Structure of Mycobacterium tuberculosis 2-C-methyl-D-erythritol 4-phosphate cytidyltransferase (IspD): a candidate antitubercular drug target</td><td width=200><ul><li>Reddy, M.C.M.</li><li>Bruning, J.B.</li><li>Thurman, C.</li><li>Ioerger, T.R.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ok/3okr/3okr_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3OKR">3OKR</a></td><td width="500"> Crystal Structure of the Mycobacterium tuberculosis 2-C-methyl-D-erythritol 4-phosphate cytidyltransferase (IspD):  an antitubercular drug target</td><td width=200><ul><li>Sacchettini, J.C.</li><li>Reddy, M.C.M.</li><li>Bruning, J.B.</li><li>Thurman, C.</li></ul></td></tr></table></td></tr><tr><td>Rv3582c</td><td>ispD</td><td width="300">Involved in the deoxyxylulose-5-phosphate pathway (DXP) of isoprenoid biosynthesis (at the third step). Catalyzes the formation of 4-diphosphocytidyl-2C-methyl-D-erythritol (CDP-me) from CTP and 2C-methyl-D-erythritol 4-phosphate (MEP).</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/q8/3q80/3q80_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3Q80">3Q80</a></td><td width="500"> Crystal Structure of Mycobacterium tuberculosis 2-C-methyl-D-erythritol 4-phosphate cytidyltransferase (IspD): a candidate antitubercular drug target</td><td width=200><ul><li>Reddy, M.C.M.</li><li>Bruning, J.B.</li><li>Thurman, C.</li><li>Ioerger, T.R.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/q7/3q7u/3q7u_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3Q7U">3Q7U</a></td><td width="500"> Crystal Structure of Mycobacterium tuberculosis 2-C-methyl-D-erythritol 4-phosphate cytidyltransferase (IspD): a candidate antitubercular drug target</td><td width=200><ul><li>Reddy, M.C.M.</li><li>Bruning, J.B.</li><li>Thurman, C.</li><li>Ioerger, T.R.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ok/3okr/3okr_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3OKR">3OKR</a></td><td width="500"> Crystal Structure of the Mycobacterium tuberculosis 2-C-methyl-D-erythritol 4-phosphate cytidyltransferase (IspD):  an antitubercular drug target</td><td width=200><ul><li>Sacchettini, J.C.</li><li>Reddy, M.C.M.</li><li>Bruning, J.B.</li><li>Thurman, C.</li></ul></td></tr></table></td></tr><tr><td>MSMEG_0620</td><td>MSMEG_0620</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/q4/3q4h/3q4h_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3Q4H">3Q4H</a></td><td width="500"> Heterologous expression of mycobacterial Esx complexes in Escherichia coli for structural studies is facilitated by the use of maltose binding protein fusions.</td><td width=200><ul><li>Arbing, M.A.</li><li>Chan, S.</li><li>Harris, L.</li><li>Kuo, E.</li><li>Zhou, T.T.</li><li>Ahn, C.J.</li><li>Nguyen, L.</li><li>He, Q.</li><li>Lu, J.</li><li>Menchavez, P.T.</li><li>Shin, A.</li><li>Holton, T.</li><li>Sawaya, M.R.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv0287</td><td>esxG</td><td width="300">Function unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/q4/3q4h/3q4h_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3Q4H">3Q4H</a></td><td width="500"> Heterologous expression of mycobacterial Esx complexes in Escherichia coli for structural studies is facilitated by the use of maltose binding protein fusions.</td><td width=200><ul><li>Arbing, M.A.</li><li>Chan, S.</li><li>Harris, L.</li><li>Kuo, E.</li><li>Zhou, T.T.</li><li>Ahn, C.J.</li><li>Nguyen, L.</li><li>He, Q.</li><li>Lu, J.</li><li>Menchavez, P.T.</li><li>Shin, A.</li><li>Holton, T.</li><li>Sawaya, M.R.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>MSMEG_0621</td><td>MSMEG_0621</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/q4/3q4h/3q4h_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3Q4H">3Q4H</a></td><td width="500"> Heterologous expression of mycobacterial Esx complexes in Escherichia coli for structural studies is facilitated by the use of maltose binding protein fusions.</td><td width=200><ul><li>Arbing, M.A.</li><li>Chan, S.</li><li>Harris, L.</li><li>Kuo, E.</li><li>Zhou, T.T.</li><li>Ahn, C.J.</li><li>Nguyen, L.</li><li>He, Q.</li><li>Lu, J.</li><li>Menchavez, P.T.</li><li>Shin, A.</li><li>Holton, T.</li><li>Sawaya, M.R.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv0288</td><td>esxH</td><td width="300">Function unknown. May be involved in virulence.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/q4/3q4h/3q4h_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3Q4H">3Q4H</a></td><td width="500"> Heterologous expression of mycobacterial Esx complexes in Escherichia coli for structural studies is facilitated by the use of maltose binding protein fusions.</td><td width=200><ul><li>Arbing, M.A.</li><li>Chan, S.</li><li>Harris, L.</li><li>Kuo, E.</li><li>Zhou, T.T.</li><li>Ahn, C.J.</li><li>Nguyen, L.</li><li>He, Q.</li><li>Lu, J.</li><li>Menchavez, P.T.</li><li>Shin, A.</li><li>Holton, T.</li><li>Sawaya, M.R.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv0014c</td><td>pknB</td><td width="300">Involved in signal transduction (via phosphorylation). Thought to regulate cell division/differentiation. Can phosphorylate the peptide substrate myelin basic protein (MBP) [catalytic activity: ATP + a protein = ADP + a phosphoprotein].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/or/3orp/3orp_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3ORP">3ORP</a></td><td width="500"> Allosteric activation mechanism of the Mycobacterium tuberculosis receptor Ser/Thr kinase, PknB</td><td width=200><ul><li>Lombana, T.N.</li><li>Echols, N.</li><li>Good, M.C.</li><li>Thomsen, N.D.</li><li>Ng, H.-L.</li><li>Greenstein, A.E.</li><li>Falick, A.M.</li><li>King, D.S.</li><li>Alber, T.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/or/3ort/3ort_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3ORT">3ORT</a></td><td width="500"> Allosteric activation mechanism of the Mycobacterium tuberculosis receptor Ser/Thr kinase, PknB</td><td width=200><ul><li>Lombana, T.N.</li><li>Echols, N.</li><li>Good, M.C.</li><li>Thomsen, N.D.</li><li>Ng, H.-L.</li><li>Greenstein, A.E.</li><li>Falick, A.M.</li><li>King, D.S.</li><li>Alber, T.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/e1/5e10/5e10_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5E10">5E10</a></td><td width="500"> Structural and Genetic Analyses of the Mycobacterium tuberculosis Protein Kinase B Sensor Domain Identify a Potential Ligand-binding Site.</td><td width=200><ul><li>Prigozhin, D.M.</li><li>Papavinasasundaram, K.G.</li><li>Baer, C.E.</li><li>Murphy, K.C.</li><li>Moskaleva, A.</li><li>Chen, T.Y.</li><li>Alber, T.</li><li>Sassetti, C.M.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/or/3orl/3orl_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3ORL">3ORL</a></td><td width="500"> Allosteric activation mechanism of the Mycobacterium tuberculosis receptor Ser/Thr kinase, PknB</td><td width=200><ul><li>Lombana, T.N.</li><li>Echols, N.</li><li>Good, M.C.</li><li>Thomsen, N.D.</li><li>Ng, H.-L.</li><li>Greenstein, A.E.</li><li>Falick, A.M.</li><li>King, D.S.</li><li>Alber, T.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/e1/5e12/5e12_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5E12">5E12</a></td><td width="500"> Structural and Genetic Analyses of the Mycobacterium tuberculosis Protein Kinase B Sensor Domain Identify a Potential Ligand-binding Site.</td><td width=200><ul><li>Prigozhin, D.M.</li><li>Papavinasasundaram, K.G.</li><li>Baer, C.E.</li><li>Murphy, K.C.</li><li>Moskaleva, A.</li><li>Chen, T.Y.</li><li>Alber, T.</li><li>Sassetti, C.M.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/e0/5e0z/5e0z_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5E0Z">5E0Z</a></td><td width="500"> Structural and Genetic Analyses of the Mycobacterium tuberculosis Protein Kinase B Sensor Domain Identify a Potential Ligand-binding Site.</td><td width=200><ul><li>Prigozhin, D.M.</li><li>Papavinasasundaram, K.G.</li><li>Baer, C.E.</li><li>Murphy, K.C.</li><li>Moskaleva, A.</li><li>Chen, T.Y.</li><li>Alber, T.</li><li>Sassetti, C.M.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/e0/5e0y/5e0y_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5E0Y">5E0Y</a></td><td width="500"> Structural and Genetic Analyses of the Mycobacterium tuberculosis Protein Kinase B Sensor Domain Identify a Potential Ligand-binding Site.</td><td width=200><ul><li>Prigozhin, D.M.</li><li>Papavinasasundaram, K.G.</li><li>Baer, C.E.</li><li>Murphy, K.C.</li><li>Moskaleva, A.</li><li>Chen, T.Y.</li><li>Alber, T.</li><li>Sassetti, C.M.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/or/3oro/3oro_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3ORO">3ORO</a></td><td width="500"> Allosteric activation mechanism of the Mycobacterium tuberculosis receptor Ser/Thr protein kinase, PknB.</td><td width=200><ul><li>Lombana, T.N.</li><li>Echols, N.</li><li>Good, M.C.</li><li>Thomsen, N.D.</li><li>Ng, H.L.</li><li>Greenstein, A.E.</li><li>Falick, A.M.</li><li>King, D.S.</li><li>Alber, T.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/or/3orm/3orm_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3ORM">3ORM</a></td><td width="500"> Allosteric activation mechanism of the Mycobacterium tuberculosis receptor Ser/Thr protein kinase, PknB.</td><td width=200><ul><li>Lombana, T.N.</li><li>Echols, N.</li><li>Good, M.C.</li><li>Thomsen, N.D.</li><li>Ng, H.L.</li><li>Greenstein, A.E.</li><li>Falick, A.M.</li><li>King, D.S.</li><li>Alber, T.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/or/3ork/3ork_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3ORK">3ORK</a></td><td width="500"> Allosteric activation mechanism of the Mycobacterium tuberculosis receptor Ser/Thr kinase, PknB</td><td width=200><ul><li>Lombana, T.N.</li><li>Echols, N.</li><li>Good, M.C.</li><li>Thomsen, N.D.</li><li>Ng, H.-L.</li><li>Greenstein, A.E.</li><li>Falick, A.M.</li><li>King, D.S.</li><li>Alber, T.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/f6/3f61/3f61_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3F61">3F61</a></td><td width="500"> Auto-activation mechanism of the Mycobacterium tuberculosis PknB receptor Ser/Thr kinase.</td><td width=200><ul><li>Mieczkowski, C.</li><li>Iavarone, A.T.</li><li>Alber, T.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/f6/3f69/3f69_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3F69">3F69</a></td><td width="500"> Auto-activation mechanism of the Mycobacterium tuberculosis PknB receptor Ser/Thr kinase.</td><td width=200><ul><li>Mieczkowski, C.</li><li>Iavarone, A.T.</li><li>Alber, T.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/or/3ori/3ori_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3ORI">3ORI</a></td><td width="500"> Allosteric activation mechanism of the Mycobacterium tuberculosis receptor Ser/Thr protein kinase, PknB.</td><td width=200><ul><li>Lombana, T.N.</li><li>Echols, N.</li><li>Good, M.C.</li><li>Thomsen, N.D.</li><li>Ng, H.L.</li><li>Greenstein, A.E.</li><li>Falick, A.M.</li><li>King, D.S.</li><li>Alber, T.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/o6/1o6y/1o6y_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1O6Y">1O6Y</a></td><td width="500"> Crystal Structure of the Catalytic Domain of the Pknb Serine/Threonine Kinase from Mycobacterium Tuberculosis</td><td width=200><ul><li>Ortiz-Lombardia, M.</li><li>Pompeo, F.</li><li>Boitel, B.</li><li>Alzari, P.M.</li></ul></td></tr></table></td></tr><tr><td>MSMEG_0435</td><td>MSMEG_0435</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/mm/3mml/3mml_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3MML">3MML</a></td><td width="500"> Crystal Structure of Allphanate Hydrolase Complex from M. smegmatis, Msmeg0435-Msmeg0436</td><td width=200><ul><li>Kaufmann, M.</li><li>Chernishof, I.</li><li>Shin, A.</li><li>Germano, D.</li><li>Sawaya, M.R.</li><li>Waldo, G.S.</li><li>Arbing, M.A.</li><li>Perry, J.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>MSMEG_0436</td><td>MSMEG_0436</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/mm/3mml/3mml_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3MML">3MML</a></td><td width="500"> Crystal Structure of Allphanate Hydrolase Complex from M. smegmatis, Msmeg0435-Msmeg0436</td><td width=200><ul><li>Kaufmann, M.</li><li>Chernishof, I.</li><li>Shin, A.</li><li>Germano, D.</li><li>Sawaya, M.R.</li><li>Waldo, G.S.</li><li>Arbing, M.A.</li><li>Perry, J.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv1411c</td><td>lprG</td><td width="300">Unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/mh/3mh8/3mh8_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3MH8">3MH8</a></td><td width="500"> Mycobacterium tuberculosis lipoprotein LprG (Rv1411c) binds triacylated glycolipid agonists of Toll-like receptor 2.</td><td width=200><ul><li>Drage, M.G.</li><li>Tsai, H.C.</li><li>Pecora, N.D.</li><li>Cheng, T.Y.</li><li>Arida, A.R.</li><li>Shukla, S.</li><li>Rojas, R.E.</li><li>Seshadri, C.</li><li>Moody, D.B.</li><li>Boom, W.H.</li><li>Sacchettini, J.C.</li><li>Harding, C.V.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/zr/4zra/4zra_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4ZRA">4ZRA</a></td><td width="500"> Mycobacterial Metabolic Syndrome: LprG and Rv1410 Regulate Triacylglyceride Levels, Growth Rate and Virulence in Mycobacterium tuberculosis.</td><td width=200><ul><li>Martinot, A.J.</li><li>Farrow, M.</li><li>Bai, L.</li><li>Layre, E.</li><li>Cheng, T.Y.</li><li>Tsai, J.H.</li><li>Iqbal, J.</li><li>Annand, J.W.</li><li>Sullivan, Z.A.</li><li>Hussain, M.M.</li><li>Sacchettini, J.</li><li>Moody, D.B.</li><li>Seeliger, J.C.</li><li>Rubin, E.J.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/mh/3mha/3mha_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3MHA">3MHA</a></td><td width="500"> Mycobacterium tuberculosis lipoprotein LprG (Rv1411c) binds triacylated glycolipid agonists of Toll-like receptor 2.</td><td width=200><ul><li>Drage, M.G.</li><li>Tsai, H.C.</li><li>Pecora, N.D.</li><li>Cheng, T.Y.</li><li>Arida, A.R.</li><li>Shukla, S.</li><li>Rojas, R.E.</li><li>Seshadri, C.</li><li>Moody, D.B.</li><li>Boom, W.H.</li><li>Sacchettini, J.C.</li><li>Harding, C.V.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/mh/3mh9/3mh9_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3MH9">3MH9</a></td><td width="500"> Mycobacterium tuberculosis lipoprotein LprG (Rv1411c) binds triacylated glycolipid agonists of Toll-like receptor 2.</td><td width=200><ul><li>Drage, M.G.</li><li>Tsai, H.C.</li><li>Pecora, N.D.</li><li>Cheng, T.Y.</li><li>Arida, A.R.</li><li>Shukla, S.</li><li>Rojas, R.E.</li><li>Seshadri, C.</li><li>Moody, D.B.</li><li>Boom, W.H.</li><li>Sacchettini, J.C.</li><li>Harding, C.V.</li></ul></td></tr></table></td></tr><tr><td>Rv1411c</td><td>lprG</td><td width="300">Unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/mh/3mh8/3mh8_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3MH8">3MH8</a></td><td width="500"> Mycobacterium tuberculosis lipoprotein LprG (Rv1411c) binds triacylated glycolipid agonists of Toll-like receptor 2.</td><td width=200><ul><li>Drage, M.G.</li><li>Tsai, H.C.</li><li>Pecora, N.D.</li><li>Cheng, T.Y.</li><li>Arida, A.R.</li><li>Shukla, S.</li><li>Rojas, R.E.</li><li>Seshadri, C.</li><li>Moody, D.B.</li><li>Boom, W.H.</li><li>Sacchettini, J.C.</li><li>Harding, C.V.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/zr/4zra/4zra_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4ZRA">4ZRA</a></td><td width="500"> Mycobacterial Metabolic Syndrome: LprG and Rv1410 Regulate Triacylglyceride Levels, Growth Rate and Virulence in Mycobacterium tuberculosis.</td><td width=200><ul><li>Martinot, A.J.</li><li>Farrow, M.</li><li>Bai, L.</li><li>Layre, E.</li><li>Cheng, T.Y.</li><li>Tsai, J.H.</li><li>Iqbal, J.</li><li>Annand, J.W.</li><li>Sullivan, Z.A.</li><li>Hussain, M.M.</li><li>Sacchettini, J.</li><li>Moody, D.B.</li><li>Seeliger, J.C.</li><li>Rubin, E.J.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/mh/3mha/3mha_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3MHA">3MHA</a></td><td width="500"> Mycobacterium tuberculosis lipoprotein LprG (Rv1411c) binds triacylated glycolipid agonists of Toll-like receptor 2.</td><td width=200><ul><li>Drage, M.G.</li><li>Tsai, H.C.</li><li>Pecora, N.D.</li><li>Cheng, T.Y.</li><li>Arida, A.R.</li><li>Shukla, S.</li><li>Rojas, R.E.</li><li>Seshadri, C.</li><li>Moody, D.B.</li><li>Boom, W.H.</li><li>Sacchettini, J.C.</li><li>Harding, C.V.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/mh/3mh9/3mh9_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3MH9">3MH9</a></td><td width="500"> Mycobacterium tuberculosis lipoprotein LprG (Rv1411c) binds triacylated glycolipid agonists of Toll-like receptor 2.</td><td width=200><ul><li>Drage, M.G.</li><li>Tsai, H.C.</li><li>Pecora, N.D.</li><li>Cheng, T.Y.</li><li>Arida, A.R.</li><li>Shukla, S.</li><li>Rojas, R.E.</li><li>Seshadri, C.</li><li>Moody, D.B.</li><li>Boom, W.H.</li><li>Sacchettini, J.C.</li><li>Harding, C.V.</li></ul></td></tr></table></td></tr><tr><td>Rv0626</td><td>vapB5</td><td width="300">Possible ribonuclease</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/db/3dbo/3dbo_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3DBO">3DBO</a></td><td width="500"> Structure and Proposed Activity of a Member of the VapBC Family of Toxin-Antitoxin Systems: VapBC-5 FROM MYCOBACTERIUM TUBERCULOSIS.</td><td width=200><ul><li>Miallau, L.</li><li>Faller, M.</li><li>Chiang, J.</li><li>Arbing, M.</li><li>Guo, F.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv0626</td><td>vapB5</td><td width="300">Possible ribonuclease</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/db/3dbo/3dbo_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3DBO">3DBO</a></td><td width="500"> Structure and Proposed Activity of a Member of the VapBC Family of Toxin-Antitoxin Systems: VapBC-5 FROM MYCOBACTERIUM TUBERCULOSIS.</td><td width=200><ul><li>Miallau, L.</li><li>Faller, M.</li><li>Chiang, J.</li><li>Arbing, M.</li><li>Guo, F.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv0627</td><td>vapC5</td><td width="300">Unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/db/3dbo/3dbo_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3DBO">3DBO</a></td><td width="500"> Structure and Proposed Activity of a Member of the VapBC Family of Toxin-Antitoxin Systems: VapBC-5 FROM MYCOBACTERIUM TUBERCULOSIS.</td><td width=200><ul><li>Miallau, L.</li><li>Faller, M.</li><li>Chiang, J.</li><li>Arbing, M.</li><li>Guo, F.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv0627</td><td>vapC5</td><td width="300">Unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/db/3dbo/3dbo_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3DBO">3DBO</a></td><td width="500"> Structure and Proposed Activity of a Member of the VapBC Family of Toxin-Antitoxin Systems: VapBC-5 FROM MYCOBACTERIUM TUBERCULOSIS.</td><td width=200><ul><li>Miallau, L.</li><li>Faller, M.</li><li>Chiang, J.</li><li>Arbing, M.</li><li>Guo, F.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv1657</td><td>argR</td><td width="300">Regulates arginine biosynthesis genes</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ca/3cag/3cag_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3CAG">3CAG</a></td><td width="500"> Structure of the C-terminal domain of the arginine repressor protein from Mycobacterium tuberculosis.</td><td width=200><ul><li>Cherney, L.T.</li><li>Cherney, M.M.</li><li>Garen, C.R.</li><li>Lu, G.J.</li><li>James, M.N.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/bu/3bue/3bue_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3BUE">3BUE</a></td><td width="500"> Structure of the C-terminal domain of the arginine repressor protein from Mycobacterium tuberculosis.</td><td width=200><ul><li>Cherney, L.T.</li><li>Cherney, M.M.</li><li>Garen, C.R.</li><li>Lu, G.J.</li><li>James, M.N.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/zf/2zfz/2zfz_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2ZFZ">2ZFZ</a></td><td width="500"> Structure of the C-terminal domain of the arginine repressor protein from Mycobacterium tuberculosis.</td><td width=200><ul><li>Cherney, L.T.</li><li>Cherney, M.M.</li><li>Garen, C.R.</li><li>Lu, G.J.</li><li>James, M.N.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/er/3ere/3ere_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3ERE">3ERE</a></td><td width="500"> Crystal structure of the arginine repressor protein in complex with the DNA operator from Mycobacterium tuberculosis.</td><td width=200><ul><li>Cherney, L.T.</li><li>Cherney, M.M.</li><li>Garen, C.R.</li><li>Lu, G.J.</li><li>James, M.N.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/la/3lap/3lap_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3LAP">3LAP</a></td><td width="500"> crystal structure of the intermediate complex of the arginine repressor from Mycobacterium tuberculosis bound with its DNA operator reveals detailed mechanism of arginine repression.</td><td width=200><ul><li>Cherney, L.T.</li><li>Cherney, M.M.</li><li>Garen, C.R.</li><li>James, M.N.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/fh/3fhz/3fhz_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3FHZ">3FHZ</a></td><td width="500"> The structure of the arginine repressor from Mycobacterium tuberculosis bound with its DNA operator and Co-repressor, L-arginine.</td><td width=200><ul><li>Cherney, L.T.</li><li>Cherney, M.M.</li><li>Garen, C.R.</li><li>James, M.N.</li></ul></td></tr></table></td></tr><tr><td>Rv1657</td><td>argR</td><td width="300">Regulates arginine biosynthesis genes</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ca/3cag/3cag_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3CAG">3CAG</a></td><td width="500"> Structure of the C-terminal domain of the arginine repressor protein from Mycobacterium tuberculosis.</td><td width=200><ul><li>Cherney, L.T.</li><li>Cherney, M.M.</li><li>Garen, C.R.</li><li>Lu, G.J.</li><li>James, M.N.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/bu/3bue/3bue_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3BUE">3BUE</a></td><td width="500"> Structure of the C-terminal domain of the arginine repressor protein from Mycobacterium tuberculosis.</td><td width=200><ul><li>Cherney, L.T.</li><li>Cherney, M.M.</li><li>Garen, C.R.</li><li>Lu, G.J.</li><li>James, M.N.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/zf/2zfz/2zfz_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2ZFZ">2ZFZ</a></td><td width="500"> Structure of the C-terminal domain of the arginine repressor protein from Mycobacterium tuberculosis.</td><td width=200><ul><li>Cherney, L.T.</li><li>Cherney, M.M.</li><li>Garen, C.R.</li><li>Lu, G.J.</li><li>James, M.N.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/er/3ere/3ere_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3ERE">3ERE</a></td><td width="500"> Crystal structure of the arginine repressor protein in complex with the DNA operator from Mycobacterium tuberculosis.</td><td width=200><ul><li>Cherney, L.T.</li><li>Cherney, M.M.</li><li>Garen, C.R.</li><li>Lu, G.J.</li><li>James, M.N.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/la/3lap/3lap_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3LAP">3LAP</a></td><td width="500"> crystal structure of the intermediate complex of the arginine repressor from Mycobacterium tuberculosis bound with its DNA operator reveals detailed mechanism of arginine repression.</td><td width=200><ul><li>Cherney, L.T.</li><li>Cherney, M.M.</li><li>Garen, C.R.</li><li>James, M.N.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/fh/3fhz/3fhz_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3FHZ">3FHZ</a></td><td width="500"> The structure of the arginine repressor from Mycobacterium tuberculosis bound with its DNA operator and Co-repressor, L-arginine.</td><td width=200><ul><li>Cherney, L.T.</li><li>Cherney, M.M.</li><li>Garen, C.R.</li><li>James, M.N.</li></ul></td></tr></table></td></tr><tr><td>Rv3117</td><td>cysA3</td><td width="300">May be a sulfotransferase involved in the formation of thiosulfate [catalytic activity: thiosulfate + cyanide = sulfite + thiocyanate].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/aa/3aax/3aax_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3AAX">3AAX</a></td><td width="500"> The crystal structure of probable thiosulfate sulfurtransferase CysA3 (Rv3117) from Mycobacterium tuberculosis</td><td width=200><ul><li>Sankaranarayanan, R.</li><li>Witholt, S.J.</li><li>Cherney, M.M.</li><li>Garen, C.R.</li><li>Cherney, L.T.</li><li>James, M.N.G.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/aa/3aay/3aay_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3AAY">3AAY</a></td><td width="500"> The crystal structure of probable thiosulfate sulfurtransferase CysA3 (Rv3117) from Mycobacterium tuberculosis</td><td width=200><ul><li>Sankaranarayanan, R.</li><li>Witholt, S.J.</li><li>Cherney, M.M.</li><li>Garen, C.R.</li><li>Cherney, L.T.</li><li>James, M.N.G.</li></ul></td></tr></table></td></tr><tr><td>Rv2579</td><td>dhaA</td><td width="300">Generates a primary alcohol and halide from 1-haloalkane and H2O [catalytic activity: 1-haloalkane + H2O = a primary alcohol + halide].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/o2/2o2i/2o2i_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2O2I">2O2I</a></td><td width="500"> Crystal structure of haloalkane dehalogenase Rv2579 from Mycobacterium tuberculosis complexed with 1,3-propandiol</td><td width=200><ul><li>Mazumdar, P.A.</li><li>Hulecki, J.</li><li>Cherney, M.M.</li><li>Garen, C.R.</li><li>James, M.N.G.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/qv/2qvb/2qvb_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2QVB">2QVB</a></td><td width="500"> X-ray crystal structure of Mycobacterium tuberculosis haloalkane dehalogenase Rv2579.</td><td width=200><ul><li>Mazumdar, P.A.</li><li>Hulecki, J.C.</li><li>Cherney, M.M.</li><li>Garen, C.R.</li><li>James, M.N.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/o2/2o2h/2o2h_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2O2H">2O2H</a></td><td width="500"> Crystal structure of haloalkane dehalogenase Rv2579 from Mycobacterium tuberculosis complexed with 1,2-dichloroethane</td><td width=200><ul><li>Mazumdar, P.A.</li><li>Hulecki, J.</li><li>Cherney, M.M.</li><li>Garen, C.R.</li><li>James, M.N.G.</li></ul></td></tr></table></td></tr><tr><td>Rv2579</td><td>dhaA</td><td width="300">Generates a primary alcohol and halide from 1-haloalkane and H2O [catalytic activity: 1-haloalkane + H2O = a primary alcohol + halide].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/o2/2o2i/2o2i_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2O2I">2O2I</a></td><td width="500"> Crystal structure of haloalkane dehalogenase Rv2579 from Mycobacterium tuberculosis complexed with 1,3-propandiol</td><td width=200><ul><li>Mazumdar, P.A.</li><li>Hulecki, J.</li><li>Cherney, M.M.</li><li>Garen, C.R.</li><li>James, M.N.G.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/qv/2qvb/2qvb_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2QVB">2QVB</a></td><td width="500"> X-ray crystal structure of Mycobacterium tuberculosis haloalkane dehalogenase Rv2579.</td><td width=200><ul><li>Mazumdar, P.A.</li><li>Hulecki, J.C.</li><li>Cherney, M.M.</li><li>Garen, C.R.</li><li>James, M.N.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/o2/2o2h/2o2h_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2O2H">2O2H</a></td><td width="500"> Crystal structure of haloalkane dehalogenase Rv2579 from Mycobacterium tuberculosis complexed with 1,2-dichloroethane</td><td width=200><ul><li>Mazumdar, P.A.</li><li>Hulecki, J.</li><li>Cherney, M.M.</li><li>Garen, C.R.</li><li>James, M.N.G.</li></ul></td></tr></table></td></tr><tr><td>Rv2844</td><td>Rv2844</td><td width="300">Function unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ib/2ib0/2ib0_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2IB0">2IB0</a></td><td width="500"> Crystal structure of a conserved hypothetical protein, rv2844, from Mycobacterium tuberculosis</td><td width=200><ul><li>Yu, M.</li><li>Bursey, E.H.</li><li>Radhakannan, T.</li><li>Kim, C.Y.</li><li>Kaviratne, T.</li><li>Woodruff, T.</li><li>Segelke, B.W.</li><li>Lekin, T.</li><li>Toppani, D.</li><li>Terwilliger, T.C.</li><li>Hung, L.W.</li></ul></td></tr></table></td></tr><tr><td>Rv2874</td><td>dipZ</td><td width="300">Could be involved in cytochrome-C biogenesis.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/hy/2hyx/2hyx_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2HYX">2HYX</a></td><td width="500"> Structure of the C-terminal domain of DipZ from Mycobacterium tuberculosis</td><td width=200><ul><li>Goldstone, D.</li><li>Baker, E.N.</li><li>Metcalf, P.</li></ul></td></tr></table></td></tr><tr><td>Rv2874</td><td>dipZ</td><td width="300">Could be involved in cytochrome-C biogenesis.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/hy/2hyx/2hyx_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2HYX">2HYX</a></td><td width="500"> Structure of the C-terminal domain of DipZ from Mycobacterium tuberculosis</td><td width=200><ul><li>Goldstone, D.</li><li>Baker, E.N.</li><li>Metcalf, P.</li></ul></td></tr></table></td></tr><tr><td>Rv1848</td><td>ureA</td><td width="300">Involved in the conversion of urea to NH3 [catalytic activity: urea + H2O = CO2 + 2 NH3]</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/fv/2fvh/2fvh_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2FVH">2FVH</a></td><td width="500"> Crystal Structure of Rv1848, a Urease Gamma Subunit UreA (Urea amidohydrolase), from Mycobacterium Tuberculosis</td><td width=200><ul><li>Bursey, E.H.</li><li>Yu, M.</li><li>Segelke, B.W.</li><li>Lekin, T.</li><li>Rupp, B.</li><li>Kim, C.Y.</li><li>Rho, B.S.</li><li>Terwilliger, T.C.</li><li>Hung, L.-W.</li></ul></td></tr></table></td></tr><tr><td>Rv0390</td><td>Rv0390</td><td width="300">Function unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/fs/2fsx/2fsx_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2FSX">2FSX</a></td><td width="500"> Crystal Structure of Rv0390 from Mycobacterium tuberculosis</td><td width=200><ul><li>Bursey, E.H.</li><li>Radhakannan, T.</li><li>Yu, M.</li><li>Segelke, B.W.</li><li>Lekin, T.</li><li>Toppani, D.</li><li>Chang, Y.-B.</li><li>Kaviratne, T.</li><li>Woodruff, T.</li><li>Terwilliger, T.C.</li><li>Hung, L.-W.</li></ul></td></tr></table></td></tr><tr><td>Rv2717c</td><td>Rv2717c</td><td width="300">Function unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/fr/2fr2/2fr2_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2FR2">2FR2</a></td><td width="500"> Crystal structure of Rv2717c from Mycobacterium tuberculosis</td><td width=200><ul><li>Hung, L.W.</li><li>Yu, M.</li><li>Bursey, E.H.</li><li>Radhakannan, T.</li><li>Segelke, B.W.</li><li>Lekin, T.</li><li>Chang, Y.B.</li><li>Terwilliger, T.C.</li><li>Kaviratne, A.</li><li>Woodruff, T.</li></ul></td></tr></table></td></tr><tr><td>Rv2302</td><td>Rv2302</td><td width="300">Function unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/a7/2a7y/2a7y_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2A7Y">2A7Y</a></td><td width="500"> Solution structure of the conserved hypothetical protein Rv2302 from Mycobacterium tuberculosis.</td><td width=200><ul><li>Buchko, G.W.</li><li>Kim, C.Y.</li><li>Terwilliger, T.C.</li><li>Kennedy, M.A.</li></ul></td></tr></table></td></tr><tr><td>Rv0983</td><td>pepD</td><td width="300">Function unknown; possibly hydrolyzes peptides and/or proteins (seems to cleave preferentially after serine residues).</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/y8/1y8t/1y8t_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1Y8T">1Y8T</a></td><td width="500"> Possible role for HtrA homologs in mycobacterium tuberculosis</td><td width=200><ul><li>Palaninathan, S.K.</li><li>MohamedMohaideen, N.N.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv0983</td><td>pepD</td><td width="300">Function unknown; possibly hydrolyzes peptides and/or proteins (seems to cleave preferentially after serine residues).</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/y8/1y8t/1y8t_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1Y8T">1Y8T</a></td><td width="500"> Possible role for HtrA homologs in mycobacterium tuberculosis</td><td width=200><ul><li>Palaninathan, S.K.</li><li>MohamedMohaideen, N.N.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv0793</td><td>Rv0793</td><td width="300">Function unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/y0/1y0h/1y0h_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1Y0H">1Y0H</a></td><td width="500"> The crystal structure of Rv0793, a hypothetical monooxygenase from M. tuberculosis</td><td width=200><ul><li>Lemieux, M.J.</li><li>Ference, C.</li><li>Cherney, M.M.</li><li>Wang, M.</li><li>Garen, C.</li><li>James, M.N.</li></ul></td></tr></table></td></tr><tr><td>Rv1155</td><td>Rv1155</td><td width="300">May be involved in biosynthesis of pyridoxine (vitamin B6) and pyridoxal phosphate. Oxidize PNP and PMP into pyridoxal 5'-phosphate (PLP) [catalytic activity: pyridoxamine 5'-phosphate + H2O + O2 = pyridoxal 5'-phosphate + NH3 + H2O2]</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/xx/1xxo/1xxo_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1XXO">1XXO</a></td><td width="500"> X-ray crystal structure of mycobacterium tuberculosis pyridoxine 5'-phosphate oxidase at 1.8 a resolution</td><td width=200><ul><li>Biswal, B.K.</li><li>Cherney, M.M.</li><li>Wang, M.</li><li>Garen, C.</li><li>James, M.N.G.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/aq/2aq6/2aq6_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2AQ6">2AQ6</a></td><td width="500"> Structures of Mycobacterium tuberculosispyridoxine 5'-phosphate oxidase and its complexes with flavin mononucleotide and pyridoxal 5'-phosphate.</td><td width=200><ul><li>Biswal, B.K.</li><li>Cherney, M.M.</li><li>Wang, M.</li><li>Garen, C.</li><li>James, M.N.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/y3/1y30/1y30_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1Y30">1Y30</a></td><td width="500"> Structures of Mycobacterium tuberculosispyridoxine 5'-phosphate oxidase and its complexes with flavin mononucleotide and pyridoxal 5'-phosphate.</td><td width=200><ul><li>Biswal, B.K.</li><li>Cherney, M.M.</li><li>Wang, M.</li><li>Garen, C.</li><li>James, M.N.</li></ul></td></tr></table></td></tr><tr><td>Rv2991</td><td>Rv2991</td><td width="300">Function unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/rf/1rfe/1rfe_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1RFE">1RFE</a></td><td width="500"> The crystal structure of Rv2991 from Mycobacterium tuberculosis: An F420binding protein with unknown function.</td><td width=200><ul><li>Benini, S.</li><li>Haouz, A.</li><li>Proux, F.</li><li>Alzari, P.</li><li>Wilson, K.</li></ul></td></tr></table></td></tr><tr><td>Rv1926c</td><td>mpt63</td><td width="300">Unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/lm/1lmi/1lmi_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1LMI">1LMI</a></td><td width="500"> Crystal structure of a major secreted protein of Mycobacterium tuberculosis-MPT63 at 
1.5-A resolution</td><td width=200><ul><li>Goulding, C.W.</li><li>Parseghian, A.</li><li>Sawaya, M.R.</li><li>Cascio, D.</li><li>Apostol, M.</li><li>Gennaro, M.L.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv1926c</td><td>mpt63</td><td width="300">Unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/lm/1lmi/1lmi_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1LMI">1LMI</a></td><td width="500"> Crystal structure of a major secreted protein of Mycobacterium tuberculosis-MPT63 at 
1.5-A resolution</td><td width=200><ul><li>Goulding, C.W.</li><li>Parseghian, A.</li><li>Sawaya, M.R.</li><li>Cascio, D.</li><li>Apostol, M.</li><li>Gennaro, M.L.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv3418c</td><td>groES</td><td width="300">Binds to CPN60 in the presence of mg-ATP and suppresses the ATPase activity of the latter.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/hx/1hx5/1hx5_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1HX5">1HX5</a></td><td width="500"> Three-dimensional Structure of Mycobacterium tuberculosis Chaperonin-10 Reveals a Partially Stable Conformation for its Mobile Loop</td><td width=200><ul><li>Taneja, B.</li><li>Mande, S.C.</li></ul></td></tr></table></td></tr><tr><td>Rv2247</td><td>accD6</td><td width="300">Involved in fatty acid biosynthesis (mycolic acids synthesis) [catalytic activity: ATP + propionyl-CoA + CO(2) + H(2)O = ADP + orthophosphate + methylmalonyl-CoA].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/tz/6tzv/6tzv_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6TZV">6TZV</a></td><td width="500"> Novel herbicidal derivatives that inhibit carboxyltransferase subunit of the acetyl-CoA carboxylase in Mycobacterium tuberculosis</td><td width=200><ul><li>Reddy, M.C.M.</li><li>Zhou, N.</li><li>Sacchettini, J.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/pr/6prw/6prw_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6PRW">6PRW</a></td><td width="500"> CRYSTAL STRUCTURE OF THE CARBOXYLTRANSFERASE SUBUNIT OF ACC (ACCD6) IN COMPLEX WITH INHIBITOR QUIZALOFOP-P DEVERIVATIVE FROM MYCOBACTERIUM TUBERCULOSIS</td><td width=200><ul><li>Reddy, M.C.M.</li><li>Nian, Z.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/pk/6pk2/6pk2_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6PK2">6PK2</a></td><td width="500"> Elucidating the Inhibition and specificity of binding of
herbicidal aryloxyphenoxypropionates derivatives to Mycobacterium tuberculosis carboxyltransferase
domain of acetyl-coenzyme A(AccD6).</td><td width=200><ul><li>Reddy, M.C.M.</li><li>Nian, Z.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/fb/4fb8/4fb8_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4FB8">4FB8</a></td><td width="500"> Structure, Activity, and Inhibition of the Carboxyltransferase beta-Subunit of Acetyl Coenzyme A Carboxylase (AccD6) from Mycobacterium tuberculosis.</td><td width=200><ul><li>Reddy, M.C.</li><li>Breda, A.</li><li>Bruning, J.B.</li><li>Sherekar, M.</li><li>Valluru, S.</li><li>Thurman, C.</li><li>Ehrenfeld, H.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/g2/4g2r/4g2r_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4G2R">4G2R</a></td><td width="500"> Structure, Activity, and Inhibition of the Carboxyltransferase beta-Subunit of Acetyl Coenzyme A Carboxylase (AccD6) from Mycobacterium tuberculosis.</td><td width=200><ul><li>Reddy, M.C.</li><li>Breda, A.</li><li>Bruning, J.B.</li><li>Sherekar, M.</li><li>Valluru, S.</li><li>Thurman, C.</li><li>Ehrenfeld, H.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv2247</td><td>accD6</td><td width="300">Involved in fatty acid biosynthesis (mycolic acids synthesis) [catalytic activity: ATP + propionyl-CoA + CO(2) + H(2)O = ADP + orthophosphate + methylmalonyl-CoA].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/tz/6tzv/6tzv_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6TZV">6TZV</a></td><td width="500"> Novel herbicidal derivatives that inhibit carboxyltransferase subunit of the acetyl-CoA carboxylase in Mycobacterium tuberculosis</td><td width=200><ul><li>Reddy, M.C.M.</li><li>Zhou, N.</li><li>Sacchettini, J.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/fb/4fb8/4fb8_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4FB8">4FB8</a></td><td width="500"> Structure, Activity, and Inhibition of the Carboxyltransferase beta-Subunit of Acetyl Coenzyme A Carboxylase (AccD6) from Mycobacterium tuberculosis.</td><td width=200><ul><li>Reddy, M.C.</li><li>Breda, A.</li><li>Bruning, J.B.</li><li>Sherekar, M.</li><li>Valluru, S.</li><li>Thurman, C.</li><li>Ehrenfeld, H.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/g2/4g2r/4g2r_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4G2R">4G2R</a></td><td width="500"> Structure, Activity, and Inhibition of the Carboxyltransferase beta-Subunit of Acetyl Coenzyme A Carboxylase (AccD6) from Mycobacterium tuberculosis.</td><td width=200><ul><li>Reddy, M.C.</li><li>Breda, A.</li><li>Bruning, J.B.</li><li>Sherekar, M.</li><li>Valluru, S.</li><li>Thurman, C.</li><li>Ehrenfeld, H.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv3574</td><td>kstR</td><td width="300">Involved in transcriptional mechanism. Predicted to control regulon involved in lipid metabolism</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ua/5ua2/5ua2_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5UA2">5UA2</a></td><td width="500"> Crystal structure of KstR in complex with cognate DNA operator</td><td width=200><ul><li>Ho, N.A.T.</li><li>Dawes, S.S.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ua/5ua1/5ua1_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5UA1">5UA1</a></td><td width="500"> Crystal structure of KstR in complex with cognate DNA operator</td><td width=200><ul><li>Ho, N.A.T.</li><li>Dawes, S.S.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/cx/5cxi/5cxi_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5CXI">5CXI</a></td><td width="500"> The Structure of the Transcriptional Repressor KstR in Complex with CoA Thioester Cholesterol Metabolites Sheds Light on the Regulation of Cholesterol Catabolism in Mycobacterium tuberculosis.</td><td width=200><ul><li>Ho, N.A.</li><li>Dawes, S.S.</li><li>Crowe, A.M.</li><li>Casabon, I.</li><li>Gao, C.</li><li>Kendall, S.L.</li><li>Baker, E.N.</li><li>Eltis, L.D.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/cx/5cxg/5cxg_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5CXG">5CXG</a></td><td width="500"> The Structure of the Transcriptional Repressor KstR in Complex with CoA Thioester Cholesterol Metabolites Sheds Light on the Regulation of Cholesterol Catabolism in Mycobacterium tuberculosis.</td><td width=200><ul><li>Ho, N.A.</li><li>Dawes, S.S.</li><li>Crowe, A.M.</li><li>Casabon, I.</li><li>Gao, C.</li><li>Kendall, S.L.</li><li>Baker, E.N.</li><li>Eltis, L.D.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/mn/3mnl/3mnl_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3MNL">3MNL</a></td><td width="500"> The Structure of the Transcriptional Repressor KstR in Complex with CoA Thioester Cholesterol Metabolites Sheds Light on the Regulation of Cholesterol Catabolism in Mycobacterium tuberculosis.</td><td width=200><ul><li>Ho, N.A.</li><li>Dawes, S.S.</li><li>Crowe, A.M.</li><li>Casabon, I.</li><li>Gao, C.</li><li>Kendall, S.L.</li><li>Baker, E.N.</li><li>Eltis, L.D.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/cw/5cw8/5cw8_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5CW8">5CW8</a></td><td width="500"> The Structure of the Transcriptional Repressor KstR in Complex with CoA Thioester Cholesterol Metabolites Sheds Light on the Regulation of Cholesterol Catabolism in Mycobacterium tuberculosis.</td><td width=200><ul><li>Ho, N.A.</li><li>Dawes, S.S.</li><li>Crowe, A.M.</li><li>Casabon, I.</li><li>Gao, C.</li><li>Kendall, S.L.</li><li>Baker, E.N.</li><li>Eltis, L.D.</li><li>Lott, J.S.</li></ul></td></tr></table></td></tr><tr><td>Rv3574</td><td>kstR</td><td width="300">Involved in transcriptional mechanism. Predicted to control regulon involved in lipid metabolism</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ua/5ua2/5ua2_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5UA2">5UA2</a></td><td width="500"> Crystal structure of KstR in complex with cognate DNA operator</td><td width=200><ul><li>Ho, N.A.T.</li><li>Dawes, S.S.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ua/5ua1/5ua1_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5UA1">5UA1</a></td><td width="500"> Crystal structure of KstR in complex with cognate DNA operator</td><td width=200><ul><li>Ho, N.A.T.</li><li>Dawes, S.S.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/cx/5cxi/5cxi_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5CXI">5CXI</a></td><td width="500"> The Structure of the Transcriptional Repressor KstR in Complex with CoA Thioester Cholesterol Metabolites Sheds Light on the Regulation of Cholesterol Catabolism in Mycobacterium tuberculosis.</td><td width=200><ul><li>Ho, N.A.</li><li>Dawes, S.S.</li><li>Crowe, A.M.</li><li>Casabon, I.</li><li>Gao, C.</li><li>Kendall, S.L.</li><li>Baker, E.N.</li><li>Eltis, L.D.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/cx/5cxg/5cxg_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5CXG">5CXG</a></td><td width="500"> The Structure of the Transcriptional Repressor KstR in Complex with CoA Thioester Cholesterol Metabolites Sheds Light on the Regulation of Cholesterol Catabolism in Mycobacterium tuberculosis.</td><td width=200><ul><li>Ho, N.A.</li><li>Dawes, S.S.</li><li>Crowe, A.M.</li><li>Casabon, I.</li><li>Gao, C.</li><li>Kendall, S.L.</li><li>Baker, E.N.</li><li>Eltis, L.D.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/mn/3mnl/3mnl_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3MNL">3MNL</a></td><td width="500"> The Structure of the Transcriptional Repressor KstR in Complex with CoA Thioester Cholesterol Metabolites Sheds Light on the Regulation of Cholesterol Catabolism in Mycobacterium tuberculosis.</td><td width=200><ul><li>Ho, N.A.</li><li>Dawes, S.S.</li><li>Crowe, A.M.</li><li>Casabon, I.</li><li>Gao, C.</li><li>Kendall, S.L.</li><li>Baker, E.N.</li><li>Eltis, L.D.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/cw/5cw8/5cw8_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5CW8">5CW8</a></td><td width="500"> The Structure of the Transcriptional Repressor KstR in Complex with CoA Thioester Cholesterol Metabolites Sheds Light on the Regulation of Cholesterol Catabolism in Mycobacterium tuberculosis.</td><td width=200><ul><li>Ho, N.A.</li><li>Dawes, S.S.</li><li>Crowe, A.M.</li><li>Casabon, I.</li><li>Gao, C.</li><li>Kendall, S.L.</li><li>Baker, E.N.</li><li>Eltis, L.D.</li><li>Lott, J.S.</li></ul></td></tr></table></td></tr><tr><td>Rv2192c</td><td>trpD</td><td width="300">Tryptophan biosynthesis</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c2/5c2l/5c2l_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5C2L">5C2L</a></td><td width="500"> Binding and mimicking of the phosphate-rich substrate, PRPP</td><td width=200><ul><li>Evans, G.L.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/by/5byt/5byt_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5BYT">5BYT</a></td><td width="500"> Binding and mimicking of the phosphate-rich substrate, PRPP.</td><td width=200><ul><li>Evans, G.L.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/bo/5bo3/5bo3_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5BO3">5BO3</a></td><td width="500"> Binding and mimicking of the phosphate-rich substrate, PRPP.</td><td width=200><ul><li>Evans, G.L.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/bo/5bo2/5bo2_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5BO2">5BO2</a></td><td width="500"> Binding and mimicking of the phosphate-rich substrate, PRPP.</td><td width=200><ul><li>Evans, G.L.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/bn/5bne/5bne_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5BNE">5BNE</a></td><td width="500"> Binding and mimicking of the phosphate-rich substrate, PRPP.</td><td width=200><ul><li>Evans, G.L.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/zt/4ztv/4ztv_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4ZTV">4ZTV</a></td><td width="500"> Bioactive non-anthranilate-like fragment-like inhibitor provides alternative starting point for inhibitor design against anthranilate phosphoribosyl transferase from Mycobacterium tuberculosis.</td><td width=200><ul><li>Evans, G.L.</li><li>Bulloch, E.M.M.</li><li>Eom, J.</li><li>Castell, A.</li><li>Short, F.L.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/zo/4zok/4zok_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4ZOK">4ZOK</a></td><td width="500"> Inhibitor bound in the active site of Mycobacterium tuberculosis anthranilate phosphoribosyltransferase (AnPRT; trpD).</td><td width=200><ul><li>Evans, G.L.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/zo/4zoj/4zoj_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4ZOJ">4ZOJ</a></td><td width="500"> Inhibitor bound in the active site of Mycobacterium tuberculosis anthranilate phosphoribosyltransferase (AnPRT; trpD).</td><td width=200><ul><li>Evans, G.L.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/zo/4zof/4zof_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4ZOF">4ZOF</a></td><td width="500"> Inhibitor bound in the active site of Mycobacterium tuberculosis anthranilate phosphoribosyltransferase (AnPRT; trpD).</td><td width=200><ul><li>Evans, G.L.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/qs/3qsa/3qsa_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3QSA">3QSA</a></td><td width="500"> Inhibition of Mycobacterium tuberculosis anthranilate phosphoribosyltransferase by blocking of an active site entrance tunnel.</td><td width=200><ul><li>Castell, A.</li><li>Short, L.F.</li><li>Evans, G.</li><li>Bulloch, E.M.</li><li>Cookson, T.</li><li>Parker, E.</li><li>Lee, C.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/qs/3qs8/3qs8_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3QS8">3QS8</a></td><td width="500"> The Substrate Capture Mechanism of Mycobacterium tuberculosis Anthranilate Phosphoribosyltransferase Provides a Mode for Inhibition.</td><td width=200><ul><li>Castell, A.</li><li>Short, F.L.</li><li>Evans, G.L.</li><li>Cookson, T.V.</li><li>Bulloch, E.M.</li><li>Joseph, D.D.</li><li>Lee, C.E.</li><li>Parker, E.J.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/qq/3qqs/3qqs_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3QQS">3QQS</a></td><td width="500"> The Substrate Capture Mechanism of Mycobacterium tuberculosis Anthranilate Phosphoribosyltransferase Provides a Mode for Inhibition.</td><td width=200><ul><li>Castell, A.</li><li>Short, F.L.</li><li>Evans, G.L.</li><li>Cookson, T.V.</li><li>Bulloch, E.M.</li><li>Joseph, D.D.</li><li>Lee, C.E.</li><li>Parker, E.J.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/zv/1zvw/1zvw_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1ZVW">1ZVW</a></td><td width="500"> The Crystal Structure of TrpD, a Metabolic Enzyme Essential for Lung Colonization by Mycobacterium tuberculosis, in Complex with its Substrate Phosphoribosylpyrophosphate.</td><td width=200><ul><li>Lee, C.E.</li><li>Goodfellow, C.</li><li>Javid-Majd, F.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c7/5c7s/5c7s_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5C7S">5C7S</a></td><td width="500"> Binding and mimicking of the phosphate-rich substrate, PRPP</td><td width=200><ul><li>Evans, G.L.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c1/5c1r/5c1r_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5C1R">5C1R</a></td><td width="500"> Binding and mimicking of the phosphate-rich substrate, PRPP.</td><td width=200><ul><li>Evans, G.L.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/m0/4m0r/4m0r_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4M0R">4M0R</a></td><td width="500"> Repurposing the Chemical Scaffold of the Anti-Arthritic Drug Lobenzarit to Target Tryptophan Biosynthesis in Mycobacterium tuberculosis.</td><td width=200><ul><li>Evans, G.L.</li><li>Gamage, S.A.</li><li>Bulloch, E.M.</li><li>Baker, E.N.</li><li>Denny, W.A.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ij/4ij1/4ij1_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4IJ1">4IJ1</a></td><td width="500"> Repurposing the Chemical Scaffold of the Anti-Arthritic Drug Lobenzarit to Target Tryptophan Biosynthesis in Mycobacterium tuberculosis.</td><td width=200><ul><li>Evans, G.L.</li><li>Gamage, S.A.</li><li>Bulloch, E.M.</li><li>Baker, E.N.</li><li>Denny, W.A.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/gk/4gkm/4gkm_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4GKM">4GKM</a></td><td width="500"> Repurposing the Chemical Scaffold of the Anti-Arthritic Drug Lobenzarit to Target Tryptophan Biosynthesis in Mycobacterium tuberculosis.</td><td width=200><ul><li>Evans, G.L.</li><li>Gamage, S.A.</li><li>Bulloch, E.M.</li><li>Baker, E.N.</li><li>Denny, W.A.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/gi/4giu/4giu_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4GIU">4GIU</a></td><td width="500"> Repurposing the Chemical Scaffold of the Anti-Arthritic Drug Lobenzarit to Target Tryptophan Biosynthesis in Mycobacterium tuberculosis.</td><td width=200><ul><li>Evans, G.L.</li><li>Gamage, S.A.</li><li>Bulloch, E.M.</li><li>Baker, E.N.</li><li>Denny, W.A.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/r6/3r6c/3r6c_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3R6C">3R6C</a></td><td width="500"> The Substrate Capture Mechanism of Mycobacterium tuberculosis Anthranilate Phosphoribosyltransferase Provides a Mode for Inhibition.</td><td width=200><ul><li>Castell, A.</li><li>Short, F.L.</li><li>Evans, G.L.</li><li>Cookson, T.V.</li><li>Bulloch, E.M.</li><li>Joseph, D.D.</li><li>Lee, C.E.</li><li>Parker, E.J.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/qr/3qr9/3qr9_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3QR9">3QR9</a></td><td width="500"> The Substrate Capture Mechanism of Mycobacterium tuberculosis Anthranilate Phosphoribosyltransferase Provides a Mode for Inhibition.</td><td width=200><ul><li>Castell, A.</li><li>Short, F.L.</li><li>Evans, G.L.</li><li>Cookson, T.V.</li><li>Bulloch, E.M.</li><li>Joseph, D.D.</li><li>Lee, C.E.</li><li>Parker, E.J.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/bp/2bpq/2bpq_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2BPQ">2BPQ</a></td><td width="500"> The Crystal Structure of Trpd, a Metabolic Enzyme Essential for Lung Colonization by Mycobacterium Tuberculosis, in Complex with its Substrate Phosphoribosylpyrophosphate</td><td width=200><ul><li>Lee, C.E.</li><li>Goodfellow, C.</li><li>Javid-Majd, F.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table></td></tr><tr><td>Rv2192c</td><td>trpD</td><td width="300">Tryptophan biosynthesis</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c2/5c2l/5c2l_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5C2L">5C2L</a></td><td width="500"> Binding and mimicking of the phosphate-rich substrate, PRPP</td><td width=200><ul><li>Evans, G.L.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/by/5byt/5byt_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5BYT">5BYT</a></td><td width="500"> Binding and mimicking of the phosphate-rich substrate, PRPP.</td><td width=200><ul><li>Evans, G.L.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/bn/5bne/5bne_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5BNE">5BNE</a></td><td width="500"> Binding and mimicking of the phosphate-rich substrate, PRPP.</td><td width=200><ul><li>Evans, G.L.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c7/5c7s/5c7s_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5C7S">5C7S</a></td><td width="500"> Binding and mimicking of the phosphate-rich substrate, PRPP</td><td width=200><ul><li>Evans, G.L.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c1/5c1r/5c1r_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5C1R">5C1R</a></td><td width="500"> Binding and mimicking of the phosphate-rich substrate, PRPP.</td><td width=200><ul><li>Evans, G.L.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/m0/4m0r/4m0r_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4M0R">4M0R</a></td><td width="500"> Repurposing the Chemical Scaffold of the Anti-Arthritic Drug Lobenzarit to Target Tryptophan Biosynthesis in Mycobacterium tuberculosis.</td><td width=200><ul><li>Evans, G.L.</li><li>Gamage, S.A.</li><li>Bulloch, E.M.</li><li>Baker, E.N.</li><li>Denny, W.A.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ij/4ij1/4ij1_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4IJ1">4IJ1</a></td><td width="500"> Repurposing the Chemical Scaffold of the Anti-Arthritic Drug Lobenzarit to Target Tryptophan Biosynthesis in Mycobacterium tuberculosis.</td><td width=200><ul><li>Evans, G.L.</li><li>Gamage, S.A.</li><li>Bulloch, E.M.</li><li>Baker, E.N.</li><li>Denny, W.A.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/gk/4gkm/4gkm_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4GKM">4GKM</a></td><td width="500"> Repurposing the Chemical Scaffold of the Anti-Arthritic Drug Lobenzarit to Target Tryptophan Biosynthesis in Mycobacterium tuberculosis.</td><td width=200><ul><li>Evans, G.L.</li><li>Gamage, S.A.</li><li>Bulloch, E.M.</li><li>Baker, E.N.</li><li>Denny, W.A.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/gi/4giu/4giu_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4GIU">4GIU</a></td><td width="500"> Repurposing the Chemical Scaffold of the Anti-Arthritic Drug Lobenzarit to Target Tryptophan Biosynthesis in Mycobacterium tuberculosis.</td><td width=200><ul><li>Evans, G.L.</li><li>Gamage, S.A.</li><li>Bulloch, E.M.</li><li>Baker, E.N.</li><li>Denny, W.A.</li><li>Lott, J.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/bp/2bpq/2bpq_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2BPQ">2BPQ</a></td><td width="500"> The Crystal Structure of Trpd, a Metabolic Enzyme Essential for Lung Colonization by Mycobacterium Tuberculosis, in Complex with its Substrate Phosphoribosylpyrophosphate</td><td width=200><ul><li>Lee, C.E.</li><li>Goodfellow, C.</li><li>Javid-Majd, F.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table></td></tr><tr><td>Rv2552c</td><td>aroE</td><td width="300">Possibly involved at the fourth step in the biosynthesis of chorismate within the biosynthesis of aromatic amino acids (the shikimate pathway) [catalytic activity: shikimate + NADP(+) = 5-dehydroshikimate + NADPH].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/p4/4p4n/4p4n_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4P4N">4P4N</a></td><td width="500"> Structure of Mycobacterium tuberculosis Shikimate Dehydrogenase</td><td width=200><ul><li>Lalgondar, M.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/p4/4p4l/4p4l_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4P4L">4P4L</a></td><td width="500"> Structure of Mycobacterium tuberculosis Shikimate Dehydrogenase</td><td width=200><ul><li>Lalgondar, M.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/p4/4p4g/4p4g_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4P4G">4P4G</a></td><td width="500"> Crystal Structure of Mycobacterium tuberculosis Shikimate Dehydrogenase</td><td width=200><ul><li>Lalgondar, M.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv2552c</td><td>aroE</td><td width="300">Possibly involved at the fourth step in the biosynthesis of chorismate within the biosynthesis of aromatic amino acids (the shikimate pathway) [catalytic activity: shikimate + NADP(+) = 5-dehydroshikimate + NADPH].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/p4/4p4n/4p4n_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4P4N">4P4N</a></td><td width="500"> Structure of Mycobacterium tuberculosis Shikimate Dehydrogenase</td><td width=200><ul><li>Lalgondar, M.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/p4/4p4l/4p4l_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4P4L">4P4L</a></td><td width="500"> Structure of Mycobacterium tuberculosis Shikimate Dehydrogenase</td><td width=200><ul><li>Lalgondar, M.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/p4/4p4g/4p4g_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4P4G">4P4G</a></td><td width="500"> Crystal Structure of Mycobacterium tuberculosis Shikimate Dehydrogenase</td><td width=200><ul><li>Lalgondar, M.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>MSMEG_6113</td><td>dacB</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/p0/4p0m/4p0m_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4P0M">4P0M</a></td><td width="500"> Subfamily-Specific Adaptations in the Structures of Two Penicillin-Binding Proteins from Mycobacterium tuberculosis.</td><td width=200><ul><li>Prigozhin, D.M.</li><li>Krieger, I.V.</li><li>Huizar, J.P.</li><li>Mavrici, D.</li><li>Waldo, G.S.</li><li>Hung, L.</li><li>Sacchettini, J.C.</li><li>Terwilliger, T.C.</li><li>Alber, T.</li></ul></td></tr></table></td></tr><tr><td>Rv2911</td><td>dacB2</td><td width="300">Involved in peptidoglycan synthesis (at final stages). Hydrolyzes the bound D-alanyl-D-alanine [catalytic activity: D-alanyl-D-alanine + H(2)O = 2 D-alanine].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/p0/4p0m/4p0m_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4P0M">4P0M</a></td><td width="500"> Subfamily-Specific Adaptations in the Structures of Two Penicillin-Binding Proteins from Mycobacterium tuberculosis.</td><td width=200><ul><li>Prigozhin, D.M.</li><li>Krieger, I.V.</li><li>Huizar, J.P.</li><li>Mavrici, D.</li><li>Waldo, G.S.</li><li>Hung, L.</li><li>Sacchettini, J.C.</li><li>Terwilliger, T.C.</li><li>Alber, T.</li></ul></td></tr></table></td></tr><tr><td>Rv2911</td><td>dacB2</td><td width="300">Involved in peptidoglycan synthesis (at final stages). Hydrolyzes the bound D-alanyl-D-alanine [catalytic activity: D-alanyl-D-alanine + H(2)O = 2 D-alanine].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/p0/4p0m/4p0m_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4P0M">4P0M</a></td><td width="500"> Subfamily-Specific Adaptations in the Structures of Two Penicillin-Binding Proteins from Mycobacterium tuberculosis.</td><td width=200><ul><li>Prigozhin, D.M.</li><li>Krieger, I.V.</li><li>Huizar, J.P.</li><li>Mavrici, D.</li><li>Waldo, G.S.</li><li>Hung, L.</li><li>Sacchettini, J.C.</li><li>Terwilliger, T.C.</li><li>Alber, T.</li></ul></td></tr></table></td></tr><tr><td>Rv3717</td><td>Rv3717</td><td width="300">Function unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/m6/4m6i/4m6i_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4M6I">4M6I</a></td><td width="500"> Structural and Biochemical Analyses of Mycobacterium tuberculosis N-Acetylmuramyl-L-alanine Amidase Rv3717 Point to a Role in Peptidoglycan Fragment Recycling.</td><td width=200><ul><li>Prigozhin, D.M.</li><li>Mavrici, D.</li><li>Huizar, J.P.</li><li>Vansell, H.J.</li><li>Alber, T.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/m6/4m6h/4m6h_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4M6H">4M6H</a></td><td width="500"> Structural and Biochemical Analyses of Mycobacterium tuberculosis N-Acetylmuramyl-L-alanine Amidase Rv3717 Point to a Role in Peptidoglycan Fragment Recycling.</td><td width=200><ul><li>Prigozhin, D.M.</li><li>Mavrici, D.</li><li>Huizar, J.P.</li><li>Vansell, H.J.</li><li>Alber, T.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/m6/4m6g/4m6g_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4M6G">4M6G</a></td><td width="500"> Structural and Biochemical Analyses of Mycobacterium tuberculosis N-Acetylmuramyl-L-alanine Amidase Rv3717 Point to a Role in Peptidoglycan Fragment Recycling.</td><td width=200><ul><li>Prigozhin, D.M.</li><li>Mavrici, D.</li><li>Huizar, J.P.</li><li>Vansell, H.J.</li><li>Alber, T.</li></ul></td></tr></table></td></tr><tr><td>Rv0099</td><td>fadD10</td><td width="300">Function unknown  but involvement in lipid degradation.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/is/4isb/4isb_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4ISB">4ISB</a></td><td width="500"> Structures of Mycobacterium tuberculosis FadD10 protein reveal a new type of adenylate-forming enzyme.</td><td width=200><ul><li>Liu, Z.</li><li>Ioerger, T.R.</li><li>Wang, F.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ir/4ir7/4ir7_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4IR7">4IR7</a></td><td width="500"> Structures of Mycobacterium tuberculosis FadD10 protein reveal a new type of adenylate-forming enzyme.</td><td width=200><ul><li>Liu, Z.</li><li>Ioerger, T.R.</li><li>Wang, F.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv2068c</td><td>blaC</td><td width="300">Hydrolyses beta-lactams to generate corresponding beta-amino acid [catalytic activity: a beta-lactam + H(2)O = a substituted beta-amino acid].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/eb/4ebp/4ebp_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4EBP">4EBP</a></td><td width="500"> Faropenem is Effective Against Mycobacterium tuberculosis in vitro and in vivo</td><td width=200><ul><li>Mire, J.A.</li><li>Pai, P.</li><li>Siddiqi, N.</li><li>Russell, D.H.</li><li>Rubin, E.J.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/eb/4ebn/4ebn_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4EBN">4EBN</a></td><td width="500"> Faropenem is Effective Against Mycobacterium tuberculosis in vitro and in vivo</td><td width=200><ul><li>Mire, J.A.</li><li>Pai, P.</li><li>Siddiqi, N.</li><li>Russell, D.H.</li><li>Rubin, E.J.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/eb/4ebl/4ebl_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4EBL">4EBL</a></td><td width="500"> Faropenem is Effective Against Mycobacterium tuberculosis in vitro and in vivo</td><td width=200><ul><li>Mire, J.A.</li><li>Pai, P.</li><li>Siddiqi, N.</li><li>Russell, D.H.</li><li>Rubin, E.J.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/vf/3vfh/3vfh_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3VFH">3VFH</a></td><td width="500"> BlaC-Specific Fluorogenic Probes for Rapid Tuberculosis Detection</td><td width=200><ul><li>Xie, H.</li><li>Mire, J.A.</li><li>Kong, Y.</li><li>Chang, M.</li><li>Sacchettini, J.C.</li><li>Cirillo, J.D.</li><li>Rao, J.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/vf/3vff/3vff_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3VFF">3VFF</a></td><td width="500"> BlaC-Specific Fluorogenic Probes for Rapid Tuberculosis Detection</td><td width=200><ul><li>Xie, H.</li><li>Mire, J.A.</li><li>Kong, Y.</li><li>Chang, M.</li><li>Sacchettini, J.C.</li><li>Cirillo, J.D.</li><li>Rao, J.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/gd/2gdn/2gdn_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2GDN">2GDN</a></td><td width="500"> Crystal Structure and Activity Studies of the Mycobacterium tuberculosis {beta}-Lactamase Reveal Its Critical Role in Resistance to {beta}-Lactam Antibiotics.</td><td width=200><ul><li>Wang, F.</li><li>Cassidy, C.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv2068c</td><td>blaC</td><td width="300">Hydrolyses beta-lactams to generate corresponding beta-amino acid [catalytic activity: a beta-lactam + H(2)O = a substituted beta-amino acid].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/eb/4ebp/4ebp_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4EBP">4EBP</a></td><td width="500"> Faropenem is Effective Against Mycobacterium tuberculosis in vitro and in vivo</td><td width=200><ul><li>Mire, J.A.</li><li>Pai, P.</li><li>Siddiqi, N.</li><li>Russell, D.H.</li><li>Rubin, E.J.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/eb/4ebn/4ebn_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4EBN">4EBN</a></td><td width="500"> Faropenem is Effective Against Mycobacterium tuberculosis in vitro and in vivo</td><td width=200><ul><li>Mire, J.A.</li><li>Pai, P.</li><li>Siddiqi, N.</li><li>Russell, D.H.</li><li>Rubin, E.J.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/eb/4ebl/4ebl_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4EBL">4EBL</a></td><td width="500"> Faropenem is Effective Against Mycobacterium tuberculosis in vitro and in vivo</td><td width=200><ul><li>Mire, J.A.</li><li>Pai, P.</li><li>Siddiqi, N.</li><li>Russell, D.H.</li><li>Rubin, E.J.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/vf/3vfh/3vfh_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3VFH">3VFH</a></td><td width="500"> BlaC-Specific Fluorogenic Probes for Rapid Tuberculosis Detection</td><td width=200><ul><li>Xie, H.</li><li>Mire, J.A.</li><li>Kong, Y.</li><li>Chang, M.</li><li>Sacchettini, J.C.</li><li>Cirillo, J.D.</li><li>Rao, J.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/vf/3vff/3vff_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3VFF">3VFF</a></td><td width="500"> BlaC-Specific Fluorogenic Probes for Rapid Tuberculosis Detection</td><td width=200><ul><li>Xie, H.</li><li>Mire, J.A.</li><li>Kong, Y.</li><li>Chang, M.</li><li>Sacchettini, J.C.</li><li>Cirillo, J.D.</li><li>Rao, J.</li></ul></td></tr></table></td></tr><tr><td>Rv3910</td><td>Rv3910</td><td width="300">Unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/uq/3uqc/3uqc_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3UQC">3UQC</a></td><td width="500"> A phosphorylated pseudokinase complex controls cell wall synthesis in mycobacteria.</td><td width=200><ul><li>Gee, C.L.</li><li>Papavinasasundaram, K.G.</li><li>Blair, S.R.</li><li>Baer, C.E.</li><li>Falick, A.M.</li><li>King, D.S.</li><li>Griffin, J.E.</li><li>Venghatakrishnan, H.</li><li>Zukauskas, A.</li><li>Wei, J.R.</li><li>Dhiman, R.K.</li><li>Crick, D.C.</li><li>Rubin, E.J.</li><li>Sassetti, C.M.</li><li>Alber, T.</li></ul></td></tr></table></td></tr><tr><td>MSMEG_3564</td><td>bfr</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/uo/3uoi/3uoi_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3UOI">3UOI</a></td><td width="500"> Mycobacterium tuberculosis bacterioferritin, BfrA</td><td width=200><ul><li>McMath, L.M.</li><li>Goulding, C.W.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/uo/3uof/3uof_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3UOF">3UOF</a></td><td width="500"> Mycobacterium tuberculosis bacterioferritin, BfrA</td><td width=200><ul><li>McMath, L.M.</li><li>Goulding, C.W.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/qb/3qb9/3qb9_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3QB9">3QB9</a></td><td width="500"> Mycobacterium tuberculosis bacterioferritin, BfrA</td><td width=200><ul><li>McMath, L.M.</li><li>Goulding, C.W.</li></ul></td></tr></table></td></tr><tr><td>Rv1876</td><td>bfrA</td><td width="300">Involved in iron storage (may perform analogous functions in iron detoxification and storage as that of animal ferritins); ferritin is an intracellular molecule that stores iron in a soluble  nontoxic  readily available form. The functional molecule  which is composed of 24 chains  is roughly spherical and contains a central cavity in which the polymeric ferric iron core is deposited.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/uo/3uoi/3uoi_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3UOI">3UOI</a></td><td width="500"> Mycobacterium tuberculosis bacterioferritin, BfrA</td><td width=200><ul><li>McMath, L.M.</li><li>Goulding, C.W.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/uo/3uof/3uof_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3UOF">3UOF</a></td><td width="500"> Mycobacterium tuberculosis bacterioferritin, BfrA</td><td width=200><ul><li>McMath, L.M.</li><li>Goulding, C.W.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/qb/3qb9/3qb9_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3QB9">3QB9</a></td><td width="500"> Mycobacterium tuberculosis bacterioferritin, BfrA</td><td width=200><ul><li>McMath, L.M.</li><li>Goulding, C.W.</li></ul></td></tr></table></td></tr><tr><td>Rv1876</td><td>bfrA</td><td width="300">Involved in iron storage (may perform analogous functions in iron detoxification and storage as that of animal ferritins); ferritin is an intracellular molecule that stores iron in a soluble  nontoxic  readily available form. The functional molecule  which is composed of 24 chains  is roughly spherical and contains a central cavity in which the polymeric ferric iron core is deposited.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/uo/3uoi/3uoi_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3UOI">3UOI</a></td><td width="500"> Mycobacterium tuberculosis bacterioferritin, BfrA</td><td width=200><ul><li>McMath, L.M.</li><li>Goulding, C.W.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/uo/3uof/3uof_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3UOF">3UOF</a></td><td width="500"> Mycobacterium tuberculosis bacterioferritin, BfrA</td><td width=200><ul><li>McMath, L.M.</li><li>Goulding, C.W.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/qb/3qb9/3qb9_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3QB9">3QB9</a></td><td width="500"> Mycobacterium tuberculosis bacterioferritin, BfrA</td><td width=200><ul><li>McMath, L.M.</li><li>Goulding, C.W.</li></ul></td></tr></table></td></tr><tr><td>Rv1611</td><td>trpC</td><td width="300">Tryptophan biosynthesis pathway (fourth step) [catalytic activity: 1-(2-carboxyphenylamino)-1-deoxy-D-ribulose 5-phosphate = 1-(indol-3-YL)glycerol 3-phosphate + CO(2) + H(2)O.]</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/t7/3t78/3t78_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3T78">3T78</a></td><td width="500"> Structural Insights and Inhibition of Mycobacterium tuberculosis Indole Glycerol Phosphate Synthase (IGPS): an Essential Enzyme for Tryptophan Biosynthesis</td><td width=200><ul><li>Reddy, M.C.M.</li><li>Bruning, J.B.</li><li>Thurman, C.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/t5/3t55/3t55_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3T55">3T55</a></td><td width="500"> Structural Insights and Inhibition of Mycobacterium tuberculosis Indole Glycerol Phosphate Synthase (IGPS): an Essential Enzyme for Tryptophan Biosynthesis</td><td width=200><ul><li>Reddy, M.C.M.</li><li>Bruning, J.B.</li><li>Thurman, C.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/t4/3t44/3t44_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3T44">3T44</a></td><td width="500"> Structural Insights and Inhibition of Mycobacterium tuberculosis Indole Glycerol Phosphate Synthase (IGPS): an Essential Enzyme for Tryptophan Biosynthesis</td><td width=200><ul><li>Reddy, M.C.M.</li><li>Bruning, J.B.</li><li>Thurman, C.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/t4/3t40/3t40_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3T40">3T40</a></td><td width="500"> Structural Insights and Inhibition of Mycobacterium tuberculosis Indole Glycerol Phosphate Synthase (IGPS): an Essential Enzyme for Tryptophan Biosynthesis</td><td width=200><ul><li>Reddy, M.C.M.</li><li>Bruning, J.B.</li><li>Thurman, C.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv1611</td><td>trpC</td><td width="300">Tryptophan biosynthesis pathway (fourth step) [catalytic activity: 1-(2-carboxyphenylamino)-1-deoxy-D-ribulose 5-phosphate = 1-(indol-3-YL)glycerol 3-phosphate + CO(2) + H(2)O.]</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/t7/3t78/3t78_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3T78">3T78</a></td><td width="500"> Structural Insights and Inhibition of Mycobacterium tuberculosis Indole Glycerol Phosphate Synthase (IGPS): an Essential Enzyme for Tryptophan Biosynthesis</td><td width=200><ul><li>Reddy, M.C.M.</li><li>Bruning, J.B.</li><li>Thurman, C.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/t5/3t55/3t55_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3T55">3T55</a></td><td width="500"> Structural Insights and Inhibition of Mycobacterium tuberculosis Indole Glycerol Phosphate Synthase (IGPS): an Essential Enzyme for Tryptophan Biosynthesis</td><td width=200><ul><li>Reddy, M.C.M.</li><li>Bruning, J.B.</li><li>Thurman, C.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/t4/3t44/3t44_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3T44">3T44</a></td><td width="500"> Structural Insights and Inhibition of Mycobacterium tuberculosis Indole Glycerol Phosphate Synthase (IGPS): an Essential Enzyme for Tryptophan Biosynthesis</td><td width=200><ul><li>Reddy, M.C.M.</li><li>Bruning, J.B.</li><li>Thurman, C.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/t4/3t40/3t40_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3T40">3T40</a></td><td width="500"> Structural Insights and Inhibition of Mycobacterium tuberculosis Indole Glycerol Phosphate Synthase (IGPS): an Essential Enzyme for Tryptophan Biosynthesis</td><td width=200><ul><li>Reddy, M.C.M.</li><li>Bruning, J.B.</li><li>Thurman, C.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/qj/3qja/3qja_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3QJA">3QJA</a></td><td width="500"> Small-molecule scaffolds for Mycobacterium tuberculosis Indole Glycerol Phosphate Synthase (IGPS) inhibitors identified by high-throughput screening and defined by X-ray crystallography</td><td width=200><ul><li>Reddy, M.C.M.</li><li>Bruning, J.B.</li><li>Thurman, C.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv2346c</td><td>esxO</td><td width="300">Function unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/og/3ogi/3ogi_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3OGI">3OGI</a></td><td width="500"> Heterologous expression of mycobacterial Esx complexes in Escherichia coli for structural studies is facilitated by the use of maltose binding protein fusions.</td><td width=200><ul><li>Arbing, M.A.</li><li>Chan, S.</li><li>Harris, L.</li><li>Kuo, E.</li><li>Zhou, T.T.</li><li>Ahn, C.J.</li><li>Nguyen, L.</li><li>He, Q.</li><li>Lu, J.</li><li>Menchavez, P.T.</li><li>Shin, A.</li><li>Holton, T.</li><li>Sawaya, M.R.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/gz/4gzr/4gzr_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4GZR">4GZR</a></td><td width="500"> Heterologous expression of mycobacterial Esx complexes in Escherichia coli for structural studies is facilitated by the use of maltose binding protein fusions.</td><td width=200><ul><li>Arbing, M.A.</li><li>Chan, S.</li><li>Harris, L.</li><li>Kuo, E.</li><li>Zhou, T.T.</li><li>Ahn, C.J.</li><li>Nguyen, L.</li><li>He, Q.</li><li>Lu, J.</li><li>Menchavez, P.T.</li><li>Shin, A.</li><li>Holton, T.</li><li>Sawaya, M.R.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv2347c</td><td>esxP</td><td width="300">Function unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/og/3ogi/3ogi_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3OGI">3OGI</a></td><td width="500"> Heterologous expression of mycobacterial Esx complexes in Escherichia coli for structural studies is facilitated by the use of maltose binding protein fusions.</td><td width=200><ul><li>Arbing, M.A.</li><li>Chan, S.</li><li>Harris, L.</li><li>Kuo, E.</li><li>Zhou, T.T.</li><li>Ahn, C.J.</li><li>Nguyen, L.</li><li>He, Q.</li><li>Lu, J.</li><li>Menchavez, P.T.</li><li>Shin, A.</li><li>Holton, T.</li><li>Sawaya, M.R.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/gz/4gzr/4gzr_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4GZR">4GZR</a></td><td width="500"> Heterologous expression of mycobacterial Esx complexes in Escherichia coli for structural studies is facilitated by the use of maltose binding protein fusions.</td><td width=200><ul><li>Arbing, M.A.</li><li>Chan, S.</li><li>Harris, L.</li><li>Kuo, E.</li><li>Zhou, T.T.</li><li>Ahn, C.J.</li><li>Nguyen, L.</li><li>He, Q.</li><li>Lu, J.</li><li>Menchavez, P.T.</li><li>Shin, A.</li><li>Holton, T.</li><li>Sawaya, M.R.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv1098c</td><td>fum</td><td width="300">Involved in the tricarboxylic acid cycle. Catalyzes the reversible hydration of fumarate to L-malate [catalytic activity: (S)-malate = fumarate + H2O]</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/no/3no9/3no9_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3NO9">3NO9</a></td><td width="500"> Crystal Structure of apo fumarate hydratase from Mycobacterium tuberculosis</td><td width=200><ul><li>Li, H.</li><li>Swanson, S.</li><li>Yu, M.</li><li>Hung, L.-H.</li><li>Sacchettini, J.S.</li></ul></td></tr></table></td></tr><tr><td>MAB_1250c</td><td>fumC</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/no/3no9/3no9_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3NO9">3NO9</a></td><td width="500"> Crystal Structure of apo fumarate hydratase from Mycobacterium tuberculosis</td><td width=200><ul><li>Li, H.</li><li>Swanson, S.</li><li>Yu, M.</li><li>Hung, L.-H.</li><li>Sacchettini, J.S.</li></ul></td></tr></table></td></tr><tr><td>Rv1098c</td><td>fum</td><td width="300">Involved in the tricarboxylic acid cycle. Catalyzes the reversible hydration of fumarate to L-malate [catalytic activity: (S)-malate = fumarate + H2O]</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/no/3no9/3no9_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3NO9">3NO9</a></td><td width="500"> Crystal Structure of apo fumarate hydratase from Mycobacterium tuberculosis</td><td width=200><ul><li>Li, H.</li><li>Swanson, S.</li><li>Yu, M.</li><li>Hung, L.-H.</li><li>Sacchettini, J.S.</li></ul></td></tr></table></td></tr><tr><td>Rv2919c</td><td>glnB</td><td width="300">In nitrogen-limiting conditions  when the ratio of GLN to 2-ketoglutarate decreases  P-II is uridylylated to P-II-UMP by GLND|Rv2918c. P-II-UMP allows the deadenylylation of glutamine synthetase (gs)  thus activating the enzyme. Converserly  in nitrogen excess P-II is deuridylated and promotes the adenylation of gs. P-II indirectly controls the transcription of the gs gene (GLNA: four copies in the genome). P-II prevents NR-II catalyzed conversion of NR-I to NR-I-phosphate  the transcriptional activator of GLNA. When P-II is uridylylated to P-II-UMP  these events are reversed.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/lf/3lf0/3lf0_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3LF0">3LF0</a></td><td width="500"> Crystal structures of the apo and ATP bound Mycobacterium tuberculosis nitrogen regulatory PII protein.</td><td width=200><ul><li>Shetty, N.D.</li><li>Reddy, M.C.</li><li>Palaninathan, S.K.</li><li>Owen, J.L.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/bz/3bzq/3bzq_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3BZQ">3BZQ</a></td><td width="500"> Crystal structures of the apo and ATP bound Mycobacterium tuberculosis nitrogen regulatory PII protein.</td><td width=200><ul><li>Shetty, N.D.</li><li>Reddy, M.C.</li><li>Palaninathan, S.K.</li><li>Owen, J.L.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv2919c</td><td>glnB</td><td width="300">In nitrogen-limiting conditions  when the ratio of GLN to 2-ketoglutarate decreases  P-II is uridylylated to P-II-UMP by GLND|Rv2918c. P-II-UMP allows the deadenylylation of glutamine synthetase (gs)  thus activating the enzyme. Converserly  in nitrogen excess P-II is deuridylated and promotes the adenylation of gs. P-II indirectly controls the transcription of the gs gene (GLNA: four copies in the genome). P-II prevents NR-II catalyzed conversion of NR-I to NR-I-phosphate  the transcriptional activator of GLNA. When P-II is uridylylated to P-II-UMP  these events are reversed.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/lf/3lf0/3lf0_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3LF0">3LF0</a></td><td width="500"> Crystal structures of the apo and ATP bound Mycobacterium tuberculosis nitrogen regulatory PII protein.</td><td width=200><ul><li>Shetty, N.D.</li><li>Reddy, M.C.</li><li>Palaninathan, S.K.</li><li>Owen, J.L.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/bz/3bzq/3bzq_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3BZQ">3BZQ</a></td><td width="500"> Crystal structures of the apo and ATP bound Mycobacterium tuberculosis nitrogen regulatory PII protein.</td><td width=200><ul><li>Shetty, N.D.</li><li>Reddy, M.C.</li><li>Palaninathan, S.K.</li><li>Owen, J.L.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv2122c</td><td>hisE</td><td width="300">Involved in histidine biosynthesis.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c9/3c90/3c90_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3C90">3C90</a></td><td width="500"> The 1.25 A resolution structure of phosphoribosyl-ATP pyrophosphohydrolase from Mycobacterium tuberculosis.</td><td width=200><ul><li>Javid-Majd, F.</li><li>Yang, D.</li><li>Ioerger, T.R.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/y6/1y6x/1y6x_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1Y6X">1Y6X</a></td><td width="500"> The 1.25 A resolution structure of phosphoribosyl-ATP pyrophosphohydrolase from Mycobacterium tuberculosis.</td><td width=200><ul><li>Javid-Majd, F.</li><li>Yang, D.</li><li>Ioerger, T.R.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv2122c</td><td>hisE</td><td width="300">Involved in histidine biosynthesis.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c9/3c90/3c90_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3C90">3C90</a></td><td width="500"> The 1.25 A resolution structure of phosphoribosyl-ATP pyrophosphohydrolase from Mycobacterium tuberculosis.</td><td width=200><ul><li>Javid-Majd, F.</li><li>Yang, D.</li><li>Ioerger, T.R.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>MSMEG_4901</td><td>rph</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/b4/3b4t/3b4t_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3B4T">3B4T</a></td><td width="500"> 2.1 A Crystal structure of RNase PH from Mycobacterium tuberculosis.</td><td width=200><ul><li>Antczak, A.J.</li><li>Lekin, T.</li><li>Segelke, B.W.</li><li>Berger, J.M.</li></ul></td></tr></table></td></tr><tr><td>Rv1340</td><td>rphA</td><td width="300">RNase PH is a phosphorolytic exoribonuclease that removes nucleotide residues following the -CCA terminus of tRNA and adds nucleotides to the ends of RNA molecules by using nucleoside diphosphates as substrates [catalytic activity: {tRNA}(N+1) + phosphate = {tRNA}(N) + a nucleoside diphosphate].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/b4/3b4t/3b4t_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3B4T">3B4T</a></td><td width="500"> 2.1 A Crystal structure of RNase PH from Mycobacterium tuberculosis.</td><td width=200><ul><li>Antczak, A.J.</li><li>Lekin, T.</li><li>Segelke, B.W.</li><li>Berger, J.M.</li></ul></td></tr></table></td></tr><tr><td>Rv1340</td><td>rphA</td><td width="300">RNase PH is a phosphorolytic exoribonuclease that removes nucleotide residues following the -CCA terminus of tRNA and adds nucleotides to the ends of RNA molecules by using nucleoside diphosphates as substrates [catalytic activity: {tRNA}(N+1) + phosphate = {tRNA}(N) + a nucleoside diphosphate].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/b4/3b4t/3b4t_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3B4T">3B4T</a></td><td width="500"> 2.1 A Crystal structure of RNase PH from Mycobacterium tuberculosis.</td><td width=200><ul><li>Antczak, A.J.</li><li>Lekin, T.</li><li>Segelke, B.W.</li><li>Berger, J.M.</li></ul></td></tr></table></td></tr><tr><td>Rv1938</td><td>ephB</td><td width="300">This enzyme acts on epoxides (alkene oxides  oxiranes) and arene oxides. Plays a role in xenobiotic metabolism by degrading potential toxic epoxides. Also determines steady-state levels of physiological mediators.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/zj/2zjf/2zjf_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2ZJF">2ZJF</a></td><td width="500"> The molecular structure of epoxide hydrolase B from Mycobacterium tuberculosis and its complex with a urea-based inhibitor.</td><td width=200><ul><li>Biswal, B.K.</li><li>Morisseau, C.</li><li>Garen, G.</li><li>Cherney, M.M.</li><li>Garen, C.</li><li>Niu, C.</li><li>Hammock, B.D.</li><li>James, M.N.</li></ul></td></tr></table></td></tr><tr><td>Rv1938</td><td>ephB</td><td width="300">This enzyme acts on epoxides (alkene oxides  oxiranes) and arene oxides. Plays a role in xenobiotic metabolism by degrading potential toxic epoxides. Also determines steady-state levels of physiological mediators.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/zj/2zjf/2zjf_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2ZJF">2ZJF</a></td><td width="500"> The molecular structure of epoxide hydrolase B from Mycobacterium tuberculosis and its complex with a urea-based inhibitor.</td><td width=200><ul><li>Biswal, B.K.</li><li>Morisseau, C.</li><li>Garen, G.</li><li>Cherney, M.M.</li><li>Garen, C.</li><li>Niu, C.</li><li>Hammock, B.D.</li><li>James, M.N.</li></ul></td></tr></table></td></tr><tr><td>Rv0760c</td><td>Rv0760c</td><td width="300">Function unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/z7/2z7a/2z7a_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2Z7A">2Z7A</a></td><td width="500"> Crystal structure of Mycobacterium tuberculosis Rv0760c at 1.50 A resolution, a structural homolog of Delta(5)-3-ketosteroid isomerase.</td><td width=200><ul><li>Cherney, M.M.</li><li>Garen, C.R.</li><li>James, M.N.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/z7/2z76/2z76_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2Z76">2Z76</a></td><td width="500"> Crystal structure of Mycobacterium tuberculosis Rv0760c at 1.50 A resolution, a structural homolog of Delta(5)-3-ketosteroid isomerase.</td><td width=200><ul><li>Cherney, M.M.</li><li>Garen, C.R.</li><li>James, M.N.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/z7/2z77/2z77_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2Z77">2Z77</a></td><td width="500"> Crystal structure of Mycobacterium tuberculosis Rv0760c at 1.50 A resolution, a structural homolog of Delta(5)-3-ketosteroid isomerase.</td><td width=200><ul><li>Cherney, M.M.</li><li>Garen, C.R.</li><li>James, M.N.</li></ul></td></tr></table></td></tr><tr><td>Rv3291c</td><td>lrpA</td><td width="300">Involved in transcriptional mechanism.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/qz/2qz8/2qz8_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2QZ8">2QZ8</a></td><td width="500"> Crystal structure of Mycobacterium tuberculosis LrpA, a leucine-responsive global regulator associated with starvation response.</td><td width=200><ul><li>Reddy, M.C.</li><li>Gokulan, K.</li><li>Jacobs, W.R.</li><li>Ioerger, T.R.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>MMAR_0373</td><td>ptpB</td><td width="300">involved in signal transduction (via dephosphorylation). can dephosphorylated in vitro the phosphotyrosine residue of myelin basic protein (MBP) at ph 7.0. could be involved in virulence by interfering with phosphotyrosine-mediated signals in macrophages [catalytic activity: protein tyrosine phosphate + H(2)O = protein tyrosine + phosphate].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/oz/2oz5/2oz5_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2OZ5">2OZ5</a></td><td width="500"> Structural Basis for Selective Inhibition of Mycobacterium tuberculosis Protein Tyrosine Phosphatase PtpB.</td><td width=200><ul><li>Grundner, C.</li><li>Perrin, D.</li><li>Hooft van Huijsduijnen, R.</li><li>Swinnen, D.</li><li>Gonzalez, J.</li><li>Gee, C.L.</li><li>Wells, T.N.</li><li>Alber, T.</li></ul></td></tr></table></td></tr><tr><td>Rv1885c</td><td>Rv1885c</td><td width="300">Involved in the shikimate pathway. Converts chorismate to prephenate in the biosynthesis of tyrosine and phenylalanine.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ao/2ao2/2ao2_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2AO2">2AO2</a></td><td width="500"> The 2.15 A crystal structure of Mycobacterium tuberculosis chorismate mutase reveals an unexpected gene duplication and suggests a role in host-pathogen interactions.</td><td width=200><ul><li>Qamra, R.</li><li>Prakash, P.</li><li>Aruna, B.</li><li>Hasnain, S.E.</li><li>Mande, S.C.</li></ul></td></tr></table></td></tr><tr><td>Rv2498c</td><td>citE</td><td width="300">Interconversion of acetate and oxaloacetate from citrate [catalytic activity: citrate = acetate + oxaloacetate].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/z6/1z6k/1z6k_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1Z6K">1Z6K</a></td><td width="500"> The structure and computational analysis of Mycobacterium tuberculosis protein CitE suggest a novel enzymatic function.</td><td width=200><ul><li>Goulding, C.W.</li><li>Bowers, P.M.</li><li>Segelke, B.</li><li>Lekin, T.</li><li>Kim, C.Y.</li><li>Terwilliger, T.C.</li><li>Eisenberg, D.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/u5/1u5v/1u5v_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1U5V">1U5V</a></td><td width="500"> Crystal structure of Mycobacterium tuberculosis citrate lyase beta subunit and its unusual triphosphate binding site</td><td width=200><ul><li>Goulding, C.W.</li><li>Lerkin, T.</li><li>Kim, C.Y.</li><li>Segelke, B.</li><li>Terwilliger, T.</li><li>Eisenberg, E.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/u5/1u5h/1u5h_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1U5H">1U5H</a></td><td width="500"> The structure and computational analysis of Mycobacterium tuberculosis protein CitE suggest a novel enzymatic function.</td><td width=200><ul><li>Goulding, C.W.</li><li>Bowers, P.M.</li><li>Segelke, B.</li><li>Lekin, T.</li><li>Kim, C.Y.</li><li>Terwilliger, T.C.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv2498c</td><td>citE</td><td width="300">Interconversion of acetate and oxaloacetate from citrate [catalytic activity: citrate = acetate + oxaloacetate].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/z6/1z6k/1z6k_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1Z6K">1Z6K</a></td><td width="500"> The structure and computational analysis of Mycobacterium tuberculosis protein CitE suggest a novel enzymatic function.</td><td width=200><ul><li>Goulding, C.W.</li><li>Bowers, P.M.</li><li>Segelke, B.</li><li>Lekin, T.</li><li>Kim, C.Y.</li><li>Terwilliger, T.C.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv2773c</td><td>dapB</td><td width="300">Involved in biosynthesis of diaminopimelate and lysine from aspartate semialdehyde (at the second step) [catalytic activity: 2 3 4 5-tetrahydrodipicolinate + NAD(P)(+) = 2 3-dihydrodipicolinate + NAD(P)H].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/yl/1yl6/1yl6_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1YL6">1YL6</a></td><td width="500"> The structure of dihydrodipicolinate reductase (DapB) from Mycobacterium tuberculosis in three crystal forms.</td><td width=200><ul><li>Janowski, R.</li><li>Kefala, G.</li><li>Weiss, M.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/yl/1yl5/1yl5_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1YL5">1YL5</a></td><td width="500"> The structure of dihydrodipicolinate reductase (DapB) from Mycobacterium tuberculosis in three crystal forms.</td><td width=200><ul><li>Janowski, R.</li><li>Kefala, G.</li><li>Weiss, M.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/yl/1yl7/1yl7_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1YL7">1YL7</a></td><td width="500"> The structure of dihydrodipicolinate reductase (DapB) from Mycobacterium tuberculosis in three crystal forms.</td><td width=200><ul><li>Janowski, R.</li><li>Kefala, G.</li><li>Weiss, M.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/p9/1p9l/1p9l_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1P9L">1P9L</a></td><td width="500"> The three-dimensional structures of the Mycobacterium tuberculosis dihydrodipicolinate reductase-NADH-2,6-PDC and -NADPH-2,6-PDC complexes. Structural and mutagenic analysis of relaxed nucleotide specificity.</td><td width=200><ul><li>Cirilli, M.</li><li>Zheng, R.</li><li>Scapin, G.</li><li>Blanchard, J.S.</li></ul></td></tr></table></td></tr><tr><td>Rv2238c</td><td>ahpE</td><td width="300">Detoxification of organic peroxides.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/xx/1xxu/1xxu_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1XXU">1XXU</a></td><td width="500"> Crystal Structure of AhpE from Mycobacterium tuberculosis, a 1-Cys Peroxiredoxin</td><td width=200><ul><li>Li, S.</li><li>Peterson, N.A.</li><li>Kim, M.Y.</li><li>Kim, C.Y.</li><li>Hung, L.W.</li><li>Yu, M.</li><li>Lekin, T.</li><li>Segelke, B.W.</li><li>Lott, J.S.</li><li>Baker, E.N.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/xv/1xvw/1xvw_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1XVW">1XVW</a></td><td width="500"> Crystal Structure of AhpE from Mycobacterium tuberculosis, a 1-Cys Peroxiredoxin</td><td width=200><ul><li>Li, S.</li><li>Peterson, N.A.</li><li>Kim, M.Y.</li><li>Kim, C.Y.</li><li>Hung, L.W.</li><li>Yu, M.</li><li>Lekin, T.</li><li>Segelke, B.W.</li><li>Lott, J.S.</li><li>Baker, E.N.</li></ul></td></tr></table></td></tr><tr><td>Rv1932</td><td>tpx</td><td width="300">Has antioxidant activity. Could remove peroxides or H(2)O(2)</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/xv/1xvq/1xvq_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1XVQ">1XVQ</a></td><td width="500"> Functional and Structural Characterization of a Thiol Peroxidase from Mycobacterium tuberculosis.</td><td width=200><ul><li>Rho, B.S.</li><li>Hung, L.W.</li><li>Holton, J.M.</li><li>Vigil, D.</li><li>Kim, S.I.</li><li>Park, M.S.</li><li>Terwilliger, T.C.</li><li>Pedelacq, J.D.</li></ul></td></tr></table></td></tr><tr><td>MSMEG_2414</td><td>coaD</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/tf/1tfu/1tfu_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1TFU">1TFU</a></td><td width="500"> Substrate-induced asymmetry and channel closure revealed by the apoenzyme structure of Mycobacterium tuberculosis phosphopantetheine adenylyltransferase.</td><td width=200><ul><li>Morris, V.K.</li><li>Izard, T.</li></ul></td></tr></table></td></tr><tr><td>Rv2965c</td><td>kdtB</td><td width="300">Involved in the coenzyme A (CoA) biosynthesis (at the fourth step). Reversibly transfers an adenylyl group from ATP to 4'-phosphopantetheine  yielding dephospho-CoA (DPCOA) and pyrophosphate [catalytic activity: ATP + pantetheine 4'-phosphate = diphosphate + dephospho-CoA].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/tf/1tfu/1tfu_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1TFU">1TFU</a></td><td width="500"> Substrate-induced asymmetry and channel closure revealed by the apoenzyme structure of Mycobacterium tuberculosis phosphopantetheine adenylyltransferase.</td><td width=200><ul><li>Morris, V.K.</li><li>Izard, T.</li></ul></td></tr></table></td></tr><tr><td>Rv2965c</td><td>kdtB</td><td width="300">Involved in the coenzyme A (CoA) biosynthesis (at the fourth step). Reversibly transfers an adenylyl group from ATP to 4'-phosphopantetheine  yielding dephospho-CoA (DPCOA) and pyrophosphate [catalytic activity: ATP + pantetheine 4'-phosphate = diphosphate + dephospho-CoA].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/tf/1tfu/1tfu_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1TFU">1TFU</a></td><td width="500"> Substrate-induced asymmetry and channel closure revealed by the apoenzyme structure of Mycobacterium tuberculosis phosphopantetheine adenylyltransferase.</td><td width=200><ul><li>Morris, V.K.</li><li>Izard, T.</li></ul></td></tr></table></td></tr><tr><td>Mb2989c</td><td>kdtB</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/tf/1tfu/1tfu_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1TFU">1TFU</a></td><td width="500"> Substrate-induced asymmetry and channel closure revealed by the apoenzyme structure of Mycobacterium tuberculosis phosphopantetheine adenylyltransferase.</td><td width=200><ul><li>Morris, V.K.</li><li>Izard, T.</li></ul></td></tr></table></td></tr><tr><td>Rv2793c</td><td>truB</td><td width="300">Formation of pseudouridine at position 55 in the psi GC loop of transfer RNAS [catalytic activity: uracil + D-ribose 5-phosphate = pseudouridine 5'-phosphate + H(2)O].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/sg/1sgv/1sgv_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1SGV">1SGV</a></td><td width="500"> Crystal structure of the apo forms of psi 55 tRNA pseudouridine synthase from Mycobacterium tuberculosis: a hinge at the base of the catalytic cleft.</td><td width=200><ul><li>Chaudhuri, B.N.</li><li>Chan, S.</li><li>Perry, L.J.</li><li>Yeates, T.O.</li></ul></td></tr></table></td></tr><tr><td>Rv2793c</td><td>truB</td><td width="300">Formation of pseudouridine at position 55 in the psi GC loop of transfer RNAS [catalytic activity: uracil + D-ribose 5-phosphate = pseudouridine 5'-phosphate + H(2)O].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/sg/1sgv/1sgv_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1SGV">1SGV</a></td><td width="500"> Crystal structure of the apo forms of psi 55 tRNA pseudouridine synthase from Mycobacterium tuberculosis: a hinge at the base of the catalytic cleft.</td><td width=200><ul><li>Chaudhuri, B.N.</li><li>Chan, S.</li><li>Perry, L.J.</li><li>Yeates, T.O.</li></ul></td></tr></table></td></tr><tr><td>Mb2816c</td><td>truB</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/sg/1sgv/1sgv_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1SGV">1SGV</a></td><td width="500"> Crystal structure of the apo forms of psi 55 tRNA pseudouridine synthase from Mycobacterium tuberculosis: a hinge at the base of the catalytic cleft.</td><td width=200><ul><li>Chaudhuri, B.N.</li><li>Chan, S.</li><li>Perry, L.J.</li><li>Yeates, T.O.</li></ul></td></tr></table></td></tr><tr><td>Rv3804c</td><td>fbpA</td><td width="300">Involved in cell wall mycoloylation. Proteins of the antigen 85 complex are responsible for the high affinity of mycobacteria to fibronectin. Possesses a mycolyltransferase activity required for the biogenesis of trehalose dimycolate (cord factor)  a dominant structure necessary for maintaining cell wall integrity.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/sf/1sfr/1sfr_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1SFR">1SFR</a></td><td width="500"> Mycobacterium tuberculosis Antigen 85A and 85C Structures Confirm Binding Orientation and Conserved Substrate Specificity</td><td width=200><ul><li>Ronning, D.R.</li><li>Vissa, V.</li><li>Besra, G.S.</li><li>Belisle, J.T.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv1626</td><td>Rv1626</td><td width="300">Sensor part of a two component regulatory system</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/sd/1sd5/1sd5_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1SD5">1SD5</a></td><td width="500"> The Crystal and Solution Structure of a Putative Transcriptional Antiterminator from Mycobacterium tuberculosis.</td><td width=200><ul><li>Morth, J.P.</li><li>Feng, V.</li><li>Perry, L.J.</li><li>Svergun, D.I.</li><li>Tucker, P.A.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/s8/1s8n/1s8n_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1S8N">1S8N</a></td><td width="500"> The Crystal and Solution Structure of a Putative Transcriptional Antiterminator from Mycobacterium tuberculosis.</td><td width=200><ul><li>Morth, J.P.</li><li>Feng, V.</li><li>Perry, L.J.</li><li>Svergun, D.I.</li><li>Tucker, P.A.</li></ul></td></tr></table></td></tr><tr><td>Rv1389</td><td>gmk</td><td width="300">Essential for recycling GMP and indirectly  CGMP [catalytic activity: ATP + GMP = ADP + GDP]</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/s4/1s4q/1s4q_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1S4Q">1S4Q</a></td><td width="500"> Crystal Structure of Guanylate Kinase from Mycobacterium tuberculosis</td><td width=200><ul><li>Chan, S.</li><li>Sawaya, M.R.</li><li>Perry, L.J.</li><li>Eisenberg, D.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/z8/1z8f/1z8f_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1Z8F">1Z8F</a></td><td width="500"> Crystal Structure of Guanylate Kinase from Mycobacterium tuberculosis (Rv1389)</td><td width=200><ul><li>Chan, S.</li><li>Sawaya, M.R.</li><li>Choi, B.</li><li>Zocchi, G.</li><li>Perry, L.J.</li></ul></td></tr></table></td></tr><tr><td>Rv1389</td><td>gmk</td><td width="300">Essential for recycling GMP and indirectly  CGMP [catalytic activity: ATP + GMP = ADP + GDP]</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/s4/1s4q/1s4q_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1S4Q">1S4Q</a></td><td width="500"> Crystal Structure of Guanylate Kinase from Mycobacterium tuberculosis</td><td width=200><ul><li>Chan, S.</li><li>Sawaya, M.R.</li><li>Perry, L.J.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Mb1424</td><td>gmk</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/s4/1s4q/1s4q_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1S4Q">1S4Q</a></td><td width="500"> Crystal Structure of Guanylate Kinase from Mycobacterium tuberculosis</td><td width=200><ul><li>Chan, S.</li><li>Sawaya, M.R.</li><li>Perry, L.J.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv0548c</td><td>menB</td><td width="300">Involved in menaquinone biosynthesis. Convert O-succinylbenzoyl-CoA (OSB-CoA) to 1 4-dihydroxy-2-naphthoic acid (DHNA) [catalytic activity: O-succinylbenzoyl-CoA = 1 4-dihydroxy-2-naphthoate + CoA].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/rj/1rjm/1rjm_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1RJM">1RJM</a></td><td width="500"> Structure of naphthoate synthase (MenB) from Mycobacterium tuberculosis in both native and product-bound forms.</td><td width=200><ul><li>Johnston, J.M.</li><li>Arcus, V.L.</li><li>Baker, E.N.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/q5/1q52/1q52_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1Q52">1Q52</a></td><td width="500"> Crystal structure of Mycobacterium tuberculosis MenB, a key enzyme in vitamin K2 biosynthesis.</td><td width=200><ul><li>Truglio, J.J.</li><li>Theis, K.</li><li>Feng, Y.</li><li>Gajda, R.</li><li>Machutta, C.</li><li>Tonge, P.J.</li><li>Kisker, C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/q5/1q51/1q51_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1Q51">1Q51</a></td><td width="500"> Crystal structure of Mycobacterium tuberculosis MenB, a key enzyme in vitamin K2 biosynthesis.</td><td width=200><ul><li>Truglio, J.J.</li><li>Theis, K.</li><li>Feng, Y.</li><li>Gajda, R.</li><li>Machutta, C.</li><li>Tonge, P.J.</li><li>Kisker, C.</li></ul></td></tr></table></td></tr><tr><td>Rv0548c</td><td>menB</td><td width="300">Involved in menaquinone biosynthesis. Convert O-succinylbenzoyl-CoA (OSB-CoA) to 1 4-dihydroxy-2-naphthoic acid (DHNA) [catalytic activity: O-succinylbenzoyl-CoA = 1 4-dihydroxy-2-naphthoate + CoA].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/q5/1q52/1q52_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1Q52">1Q52</a></td><td width="500"> Crystal structure of Mycobacterium tuberculosis MenB, a key enzyme in vitamin K2 biosynthesis.</td><td width=200><ul><li>Truglio, J.J.</li><li>Theis, K.</li><li>Feng, Y.</li><li>Gajda, R.</li><li>Machutta, C.</li><li>Tonge, P.J.</li><li>Kisker, C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/q5/1q51/1q51_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1Q51">1Q51</a></td><td width="500"> Crystal structure of Mycobacterium tuberculosis MenB, a key enzyme in vitamin K2 biosynthesis.</td><td width=200><ul><li>Truglio, J.J.</li><li>Theis, K.</li><li>Feng, Y.</li><li>Gajda, R.</li><li>Machutta, C.</li><li>Tonge, P.J.</li><li>Kisker, C.</li></ul></td></tr></table></td></tr><tr><td>Rv2225</td><td>panB</td><td width="300">Involved in pantothenate biosynthesis.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/oy/1oy0/1oy0_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1OY0">1OY0</a></td><td width="500"> The Crystal Structure of the First Enzyme in the Pantothenate Biosynthetic Pathway, 
                    Ketopantoate Hydroxymethyltransferase, from M. tuberculosis</td><td width=200><ul><li>Chaudhuri, B.N.</li><li>Sawaya, M.R.</li><li>Kim, C.Y.</li><li>Waldo, G.S.</li><li>Park, M.S.</li><li>Terwilliger, T.C.</li><li>Yeates, T.O.</li></ul></td></tr></table></td></tr><tr><td>Rv0137c</td><td>msrA</td><td width="300">Has an important function as a repair enzyme for proteins that have been inactivated by oxidation. Catalyzes the reversible oxidation-reduction of methionine sulfoxide in proteins to methionine [catalytic activity: protein L-methionine + oxidized thioredoxin + H2O = protein-L-methionine-(S)-S-oxide + reduced thioredoxin].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/nw/1nwa/1nwa_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1NWA">1NWA</a></td><td width="500"> Structure of Mycobacterium tuberculosis Methionine Sulfoxide Reductase A in Complex with Protein-bound Methionine</td><td width=200><ul><li>Taylor, A.B.</li><li>Benglis Jr., D.M.</li><li>Dhandayuthapani, S.</li><li>Hart, P.J.</li></ul></td></tr></table></td></tr><tr><td>Rv3602c</td><td>panC</td><td width="300">Involved in pantothenate biosynthesis [catalytic activity: ATP + (R)-pantoate + beta-alanine = AMP + pyrophosphate + (R)-pantothenate].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/n2/1n2o/1n2o_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1N2O">1N2O</a></td><td width="500"> Crystal structures of a pantothenate 
synthetase from M. tuberculosis and its 
complexes with substrates and a 
reaction intermediate</td><td width=200><ul><li>Wang, S.</li><li>Eisenberg, D.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/a8/2a88/2a88_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2A88">2A88</a></td><td width="500"> Crystal Structure of the Pantothenate Synthetase from Mycobacterium tuberculosis, Snapshots of the Enzyme in Action.</td><td width=200><ul><li>Wang, S.</li><li>Eisenberg, D.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/a8/2a84/2a84_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2A84">2A84</a></td><td width="500"> Crystal Structure of the Pantothenate Synthetase from Mycobacterium tuberculosis, Snapshots of the Enzyme in Action.</td><td width=200><ul><li>Wang, S.</li><li>Eisenberg, D.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/a7/2a7x/2a7x_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2A7X">2A7X</a></td><td width="500"> Crystal Structure of the Pantothenate Synthetase from Mycobacterium tuberculosis, Snapshots of the Enzyme in Action.</td><td width=200><ul><li>Wang, S.</li><li>Eisenberg, D.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/a8/2a86/2a86_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2A86">2A86</a></td><td width="500"> Crystal Structure of the Pantothenate Synthetase from Mycobacterium tuberculosis, Snapshots of the Enzyme in Action.</td><td width=200><ul><li>Wang, S.</li><li>Eisenberg, D.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/n2/1n2j/1n2j_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1N2J">1N2J</a></td><td width="500"> Crystal structures of a pantothenate 
synthetase from M. tuberculosis and its 
complexes with substrates and a 
reaction intermediate</td><td width=200><ul><li>Wang, S.</li><li>Eisenberg, D.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/n2/1n2i/1n2i_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1N2I">1N2I</a></td><td width="500"> Crystal structures of a pantothenate 
synthetase from M. tuberculosis and its 
complexes with substrates and a 
reaction intermediate</td><td width=200><ul><li>Wang, S.</li><li>Eisenberg, D.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/n2/1n2h/1n2h_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1N2H">1N2H</a></td><td width="500"> Crystal structures of a pantothenate 
synthetase from M. tuberculosis and its 
complexes with substrates and a 
reaction intermediate</td><td width=200><ul><li>Wang, S.</li><li>Eisenberg, D.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/n2/1n2g/1n2g_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1N2G">1N2G</a></td><td width="500"> Crystal structures of a pantothenate 
synthetase from M. tuberculosis and its 
complexes with substrates and a 
reaction intermediate</td><td width=200><ul><li>Wang, S.</li><li>Eisenberg, D.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/n2/1n2e/1n2e_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1N2E">1N2E</a></td><td width="500"> Crystal structures of a pantothenate 
synthetase from M. tuberculosis and its 
complexes with substrates and a 
reaction intermediate</td><td width=200><ul><li>Wang, S.</li><li>Eisenberg, D.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/n2/1n2b/1n2b_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1N2B">1N2B</a></td><td width="500"> Crystal structures of a pantothenate 
synthetase from M. tuberculosis and its 
complexes with substrates and a 
reaction intermediate</td><td width=200><ul><li>Wang, S.</li><li>Eisenberg, D.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/mo/1mop/1mop_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1MOP">1MOP</a></td><td width="500"> Crystal Structures of a Pantothenate Synthetase from M. tuberculosis and its Complexes with Substrates and a Reaction Intermediate</td><td width=200><ul><li>Wang, S.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv2878c</td><td>mpt53</td><td width="300">Not really known. Despite a weak homology to thioredoxin this cannot serve as a substrate for thioredoxin reductase. Furthermore it has no disulfide reducing activity.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/lu/1lu4/1lu4_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1LU4">1LU4</a></td><td width="500"> Gram-positive DsbE Proteins Function Differently from Gram-negative DsbE Homologs: A STRUCTURE TO FUNCTION ANALYSIS OF DsbE FROM MYCOBACTERIUM TUBERCULOSIS.</td><td width=200><ul><li>Goulding, C.W.</li><li>Apostol, M.I.</li><li>Gleiter, S.</li><li>Parseghian, A.</li><li>Bardwell, J.</li><li>Gennaro, M.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv2794c</td><td>pptT</td><td width="300">Biosynthesis of fatty acids and lipids. Transfers the 4'-phosphopantetheine moiety from coenzyme A to a SER of acyl-carrier protein. Catalyzes the formation of holo-ACP  which mediates the transfer of acyl fatty-acid intermediates during the biosynthesis of fatty acids and lipids [catalytic activity: CoA + APO-[acyl-carrier protein] = adenosine 3' 5'-bisphosphate + holo-[acyl-carrier protein] ].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/n8/7n8e/7n8e_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/7N8E">7N8E</a></td><td width="500"> In Vitro and In Vivo Inhibition of the Mycobacterium tuberculosis Phosphopantetheinyl Transferase PptT by Amidinoureas.</td><td width=200><ul><li>Ottavi, S.</li><li>Scarry, S.M.</li><li>Mosior, J.</li><li>Ling, Y.</li><li>Roberts, J.</li><li>Singh, A.</li><li>Zhang, D.</li><li>Goullieux, L.</li><li>Roubert, C.</li><li>Bacque, E.</li><li>Lagiakos, H.R.</li><li>Vendome, J.</li><li>Moraca, F.</li><li>Li, K.</li><li>Perkowski, A.J.</li><li>Ramesh, R.</li><li>Bowler, M.M.</li><li>Tracy, W.</li><li>Feher, V.A.</li><li>Sacchettini, J.C.</li><li>Gold, B.S.</li><li>Nathan, C.F.</li><li>Aube, J.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/n8/7n8m/7n8m_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/7N8M">7N8M</a></td><td width="500"> In Vitro and In Vivo Inhibition of the Mycobacterium tuberculosis Phosphopantetheinyl Transferase PptT by Amidinoureas.</td><td width=200><ul><li>Ottavi, S.</li><li>Scarry, S.M.</li><li>Mosior, J.</li><li>Ling, Y.</li><li>Roberts, J.</li><li>Singh, A.</li><li>Zhang, D.</li><li>Goullieux, L.</li><li>Roubert, C.</li><li>Bacque, E.</li><li>Lagiakos, H.R.</li><li>Vendome, J.</li><li>Moraca, F.</li><li>Li, K.</li><li>Perkowski, A.J.</li><li>Ramesh, R.</li><li>Bowler, M.M.</li><li>Tracy, W.</li><li>Feher, V.A.</li><li>Sacchettini, J.C.</li><li>Gold, B.S.</li><li>Nathan, C.F.</li><li>Aube, J.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/n8/7n8l/7n8l_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/7N8L">7N8L</a></td><td width="500"> In Vitro and In Vivo Inhibition of the Mycobacterium tuberculosis Phosphopantetheinyl Transferase PptT by Amidinoureas.</td><td width=200><ul><li>Ottavi, S.</li><li>Scarry, S.M.</li><li>Mosior, J.</li><li>Ling, Y.</li><li>Roberts, J.</li><li>Singh, A.</li><li>Zhang, D.</li><li>Goullieux, L.</li><li>Roubert, C.</li><li>Bacque, E.</li><li>Lagiakos, H.R.</li><li>Vendome, J.</li><li>Moraca, F.</li><li>Li, K.</li><li>Perkowski, A.J.</li><li>Ramesh, R.</li><li>Bowler, M.M.</li><li>Tracy, W.</li><li>Feher, V.A.</li><li>Sacchettini, J.C.</li><li>Gold, B.S.</li><li>Nathan, C.F.</li><li>Aube, J.</li></ul></td></tr></table></td></tr><tr><td>Rv2794c</td><td>pptT</td><td width="300">Biosynthesis of fatty acids and lipids. Transfers the 4'-phosphopantetheine moiety from coenzyme A to a SER of acyl-carrier protein. Catalyzes the formation of holo-ACP  which mediates the transfer of acyl fatty-acid intermediates during the biosynthesis of fatty acids and lipids [catalytic activity: CoA + APO-[acyl-carrier protein] = adenosine 3' 5'-bisphosphate + holo-[acyl-carrier protein] ].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/n8/7n8e/7n8e_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/7N8E">7N8E</a></td><td width="500"> In Vitro and In Vivo Inhibition of the Mycobacterium tuberculosis Phosphopantetheinyl Transferase PptT by Amidinoureas.</td><td width=200><ul><li>Ottavi, S.</li><li>Scarry, S.M.</li><li>Mosior, J.</li><li>Ling, Y.</li><li>Roberts, J.</li><li>Singh, A.</li><li>Zhang, D.</li><li>Goullieux, L.</li><li>Roubert, C.</li><li>Bacque, E.</li><li>Lagiakos, H.R.</li><li>Vendome, J.</li><li>Moraca, F.</li><li>Li, K.</li><li>Perkowski, A.J.</li><li>Ramesh, R.</li><li>Bowler, M.M.</li><li>Tracy, W.</li><li>Feher, V.A.</li><li>Sacchettini, J.C.</li><li>Gold, B.S.</li><li>Nathan, C.F.</li><li>Aube, J.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/n8/7n8m/7n8m_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/7N8M">7N8M</a></td><td width="500"> In Vitro and In Vivo Inhibition of the Mycobacterium tuberculosis Phosphopantetheinyl Transferase PptT by Amidinoureas.</td><td width=200><ul><li>Ottavi, S.</li><li>Scarry, S.M.</li><li>Mosior, J.</li><li>Ling, Y.</li><li>Roberts, J.</li><li>Singh, A.</li><li>Zhang, D.</li><li>Goullieux, L.</li><li>Roubert, C.</li><li>Bacque, E.</li><li>Lagiakos, H.R.</li><li>Vendome, J.</li><li>Moraca, F.</li><li>Li, K.</li><li>Perkowski, A.J.</li><li>Ramesh, R.</li><li>Bowler, M.M.</li><li>Tracy, W.</li><li>Feher, V.A.</li><li>Sacchettini, J.C.</li><li>Gold, B.S.</li><li>Nathan, C.F.</li><li>Aube, J.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/n8/7n8l/7n8l_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/7N8L">7N8L</a></td><td width="500"> In Vitro and In Vivo Inhibition of the Mycobacterium tuberculosis Phosphopantetheinyl Transferase PptT by Amidinoureas.</td><td width=200><ul><li>Ottavi, S.</li><li>Scarry, S.M.</li><li>Mosior, J.</li><li>Ling, Y.</li><li>Roberts, J.</li><li>Singh, A.</li><li>Zhang, D.</li><li>Goullieux, L.</li><li>Roubert, C.</li><li>Bacque, E.</li><li>Lagiakos, H.R.</li><li>Vendome, J.</li><li>Moraca, F.</li><li>Li, K.</li><li>Perkowski, A.J.</li><li>Ramesh, R.</li><li>Bowler, M.M.</li><li>Tracy, W.</li><li>Feher, V.A.</li><li>Sacchettini, J.C.</li><li>Gold, B.S.</li><li>Nathan, C.F.</li><li>Aube, J.</li></ul></td></tr></table></td></tr><tr><td>Rv2795c</td><td>Rv2795c</td><td width="300">Function unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/un/6unc/6unc_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6UNC">6UNC</a></td><td width="500"> Structural insights into phosphopantetheinyl hydrolase PptH from Mycobacterium tuberculosis.</td><td width=200><ul><li>Mosior, J.</li><li>Bourland, R.</li><li>Soma, S.</li><li>Nathan, C.</li><li>Sacchettini, J.</li></ul></td></tr></table></td></tr><tr><td>Rv3601c</td><td>panD</td><td width="300">Involved in pantothenate biosynthesis [catalytic activity: L-aspartate = beta-alanine + CO(2)].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/p1/6p1y/6p1y_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6P1Y">6P1Y</a></td><td width="500"> The molecular basis of pyrazinamide activity on Mycobacterium tuberculosis PanD.</td><td width=200><ul><li>Sun, Q.</li><li>Li, X.</li><li>Perez, L.M.</li><li>Shi, W.</li><li>Zhang, Y.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/p0/6p02/6p02_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6P02">6P02</a></td><td width="500"> The molecular basis of pyrazinamide activity on Mycobacterium tuberculosis PanD.</td><td width=200><ul><li>Sun, Q.</li><li>Li, X.</li><li>Perez, L.M.</li><li>Shi, W.</li><li>Zhang, Y.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/oz/6oz8/6oz8_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6OZ8">6OZ8</a></td><td width="500"> The molecular basis of pyrazinamide activity on Mycobacterium tuberculosis PanD.</td><td width=200><ul><li>Sun, Q.</li><li>Li, X.</li><li>Perez, L.M.</li><li>Shi, W.</li><li>Zhang, Y.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/oy/6oyy/6oyy_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6OYY">6OYY</a></td><td width="500"> The molecular basis of pyrazinamide activity on Mycobacterium tuberculosis PanD.</td><td width=200><ul><li>Sun, Q.</li><li>Li, X.</li><li>Perez, L.M.</li><li>Shi, W.</li><li>Zhang, Y.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv3601c</td><td>panD</td><td width="300">Involved in pantothenate biosynthesis [catalytic activity: L-aspartate = beta-alanine + CO(2)].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/p1/6p1y/6p1y_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6P1Y">6P1Y</a></td><td width="500"> The molecular basis of pyrazinamide activity on Mycobacterium tuberculosis PanD.</td><td width=200><ul><li>Sun, Q.</li><li>Li, X.</li><li>Perez, L.M.</li><li>Shi, W.</li><li>Zhang, Y.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/p0/6p02/6p02_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6P02">6P02</a></td><td width="500"> The molecular basis of pyrazinamide activity on Mycobacterium tuberculosis PanD.</td><td width=200><ul><li>Sun, Q.</li><li>Li, X.</li><li>Perez, L.M.</li><li>Shi, W.</li><li>Zhang, Y.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/oz/6oz8/6oz8_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6OZ8">6OZ8</a></td><td width="500"> The molecular basis of pyrazinamide activity on Mycobacterium tuberculosis PanD.</td><td width=200><ul><li>Sun, Q.</li><li>Li, X.</li><li>Perez, L.M.</li><li>Shi, W.</li><li>Zhang, Y.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/oy/6oyy/6oyy_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6OYY">6OYY</a></td><td width="500"> The molecular basis of pyrazinamide activity on Mycobacterium tuberculosis PanD.</td><td width=200><ul><li>Sun, Q.</li><li>Li, X.</li><li>Perez, L.M.</li><li>Shi, W.</li><li>Zhang, Y.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv0014c</td><td>pknB</td><td width="300">Involved in signal transduction (via phosphorylation). Thought to regulate cell division/differentiation. Can phosphorylate the peptide substrate myelin basic protein (MBP) [catalytic activity: ATP + a protein = ADP + a phosphoprotein].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/e1/5e10/5e10_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5E10">5E10</a></td><td width="500"> Structural and Genetic Analyses of the Mycobacterium tuberculosis Protein Kinase B Sensor Domain Identify a Potential Ligand-binding Site.</td><td width=200><ul><li>Prigozhin, D.M.</li><li>Papavinasasundaram, K.G.</li><li>Baer, C.E.</li><li>Murphy, K.C.</li><li>Moskaleva, A.</li><li>Chen, T.Y.</li><li>Alber, T.</li><li>Sassetti, C.M.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/e1/5e12/5e12_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5E12">5E12</a></td><td width="500"> Structural and Genetic Analyses of the Mycobacterium tuberculosis Protein Kinase B Sensor Domain Identify a Potential Ligand-binding Site.</td><td width=200><ul><li>Prigozhin, D.M.</li><li>Papavinasasundaram, K.G.</li><li>Baer, C.E.</li><li>Murphy, K.C.</li><li>Moskaleva, A.</li><li>Chen, T.Y.</li><li>Alber, T.</li><li>Sassetti, C.M.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/e0/5e0z/5e0z_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5E0Z">5E0Z</a></td><td width="500"> Structural and Genetic Analyses of the Mycobacterium tuberculosis Protein Kinase B Sensor Domain Identify a Potential Ligand-binding Site.</td><td width=200><ul><li>Prigozhin, D.M.</li><li>Papavinasasundaram, K.G.</li><li>Baer, C.E.</li><li>Murphy, K.C.</li><li>Moskaleva, A.</li><li>Chen, T.Y.</li><li>Alber, T.</li><li>Sassetti, C.M.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/e0/5e0y/5e0y_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5E0Y">5E0Y</a></td><td width="500"> Structural and Genetic Analyses of the Mycobacterium tuberculosis Protein Kinase B Sensor Domain Identify a Potential Ligand-binding Site.</td><td width=200><ul><li>Prigozhin, D.M.</li><li>Papavinasasundaram, K.G.</li><li>Baer, C.E.</li><li>Murphy, K.C.</li><li>Moskaleva, A.</li><li>Chen, T.Y.</li><li>Alber, T.</li><li>Sassetti, C.M.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/f6/3f61/3f61_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3F61">3F61</a></td><td width="500"> Auto-activation mechanism of the Mycobacterium tuberculosis PknB receptor Ser/Thr kinase.</td><td width=200><ul><li>Mieczkowski, C.</li><li>Iavarone, A.T.</li><li>Alber, T.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/f6/3f69/3f69_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3F69">3F69</a></td><td width="500"> Auto-activation mechanism of the Mycobacterium tuberculosis PknB receptor Ser/Thr kinase.</td><td width=200><ul><li>Mieczkowski, C.</li><li>Iavarone, A.T.</li><li>Alber, T.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/o6/1o6y/1o6y_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1O6Y">1O6Y</a></td><td width="500"> Crystal Structure of the Catalytic Domain of the Pknb Serine/Threonine Kinase from Mycobacterium Tuberculosis</td><td width=200><ul><li>Ortiz-Lombardia, M.</li><li>Pompeo, F.</li><li>Boitel, B.</li><li>Alzari, P.M.</li></ul></td></tr></table></td></tr><tr><td>Rv2671</td><td>ribD</td><td width="300">Involved in riboflavin biosynthesis (at the second and third steps). Converts 2 5-diamino-6-(ribosylamino)-4(3H)-pyrimidinone 5'-phosphate into 5-amino-6-(ribosylamino)-2 4(1H 3H)-pyrimidinedione 5'-phosphate [catalytic activity 1: 2 5-diamino-6-hydroxy-4-(5-phosphoribosylamino)pyrimidine + H(2)O = 5-amino-6-(5-phosphoribosylamino)uracil + NH(3)] [catalytic activity 2: 5-amino-6-(5-phosphoribitylamino)uracil + NADP(+) = 5-amino-6-(5-phosphoribosylamino)uracil + NADPH].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/xt/4xt6/4xt6_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4XT6">4XT6</a></td><td width="500"> Structural Insights into Mycobacterium tuberculosis Rv2671 Protein as a Dihydrofolate Reductase Functional Analogue Contributing to para-Aminosalicylic Acid Resistance.</td><td width=200><ul><li>Cheng, Y.S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/xt/4xt5/4xt5_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4XT5">4XT5</a></td><td width="500"> Structural Insights into Mycobacterium tuberculosis Rv2671 Protein as a Dihydrofolate Reductase Functional Analogue Contributing to para-Aminosalicylic Acid Resistance.</td><td width=200><ul><li>Cheng, Y.S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/xr/4xrb/4xrb_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4XRB">4XRB</a></td><td width="500"> Structural Insights into Mycobacterium tuberculosis Rv2671 Protein as a Dihydrofolate Reductase Functional Analogue Contributing to para-Aminosalicylic Acid Resistance.</td><td width=200><ul><li>Cheng, Y.S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/xt/4xt8/4xt8_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4XT8">4XT8</a></td><td width="500"> Structural Insights into Mycobacterium tuberculosis Rv2671 Protein as a Dihydrofolate Reductase Functional Analogue Contributing to para-Aminosalicylic Acid Resistance.</td><td width=200><ul><li>Cheng, Y.S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/xt/4xt4/4xt4_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4XT4">4XT4</a></td><td width="500"> Structural Insights into Mycobacterium tuberculosis Rv2671 Protein as a Dihydrofolate Reductase Functional Analogue Contributing to para-Aminosalicylic Acid Resistance.</td><td width=200><ul><li>Cheng, Y.S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/xt/4xt7/4xt7_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4XT7">4XT7</a></td><td width="500"> Structural Insights into Mycobacterium tuberculosis Rv2671 Protein as a Dihydrofolate Reductase Functional Analogue Contributing to para-Aminosalicylic Acid Resistance.</td><td width=200><ul><li>Cheng, Y.S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv2671</td><td>ribD</td><td width="300">Involved in riboflavin biosynthesis (at the second and third steps). Converts 2 5-diamino-6-(ribosylamino)-4(3H)-pyrimidinone 5'-phosphate into 5-amino-6-(ribosylamino)-2 4(1H 3H)-pyrimidinedione 5'-phosphate [catalytic activity 1: 2 5-diamino-6-hydroxy-4-(5-phosphoribosylamino)pyrimidine + H(2)O = 5-amino-6-(5-phosphoribosylamino)uracil + NH(3)] [catalytic activity 2: 5-amino-6-(5-phosphoribitylamino)uracil + NADP(+) = 5-amino-6-(5-phosphoribosylamino)uracil + NADPH].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/xt/4xt6/4xt6_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4XT6">4XT6</a></td><td width="500"> Structural Insights into Mycobacterium tuberculosis Rv2671 Protein as a Dihydrofolate Reductase Functional Analogue Contributing to para-Aminosalicylic Acid Resistance.</td><td width=200><ul><li>Cheng, Y.S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/xt/4xt5/4xt5_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4XT5">4XT5</a></td><td width="500"> Structural Insights into Mycobacterium tuberculosis Rv2671 Protein as a Dihydrofolate Reductase Functional Analogue Contributing to para-Aminosalicylic Acid Resistance.</td><td width=200><ul><li>Cheng, Y.S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/xr/4xrb/4xrb_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4XRB">4XRB</a></td><td width="500"> Structural Insights into Mycobacterium tuberculosis Rv2671 Protein as a Dihydrofolate Reductase Functional Analogue Contributing to para-Aminosalicylic Acid Resistance.</td><td width=200><ul><li>Cheng, Y.S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/xt/4xt8/4xt8_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4XT8">4XT8</a></td><td width="500"> Structural Insights into Mycobacterium tuberculosis Rv2671 Protein as a Dihydrofolate Reductase Functional Analogue Contributing to para-Aminosalicylic Acid Resistance.</td><td width=200><ul><li>Cheng, Y.S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/xt/4xt4/4xt4_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4XT4">4XT4</a></td><td width="500"> Structural Insights into Mycobacterium tuberculosis Rv2671 Protein as a Dihydrofolate Reductase Functional Analogue Contributing to para-Aminosalicylic Acid Resistance.</td><td width=200><ul><li>Cheng, Y.S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/xt/4xt7/4xt7_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4XT7">4XT7</a></td><td width="500"> Structural Insights into Mycobacterium tuberculosis Rv2671 Protein as a Dihydrofolate Reductase Functional Analogue Contributing to para-Aminosalicylic Acid Resistance.</td><td width=200><ul><li>Cheng, Y.S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>MAB_1622</td><td>rsfS</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/wc/4wcw/4wcw_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4WCW">4WCW</a></td><td width="500"> Structure of Ribosomal Silencing Factor Bound to Mycobacterium tuberculosis Ribosome.</td><td width=200><ul><li>Li, X.</li><li>Sun, Q.</li><li>Jiang, C.</li><li>Yang, K.</li><li>Hung, L.W.</li><li>Zhang, J.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv2420c</td><td>Rv2420c</td><td width="300">Function unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/wc/4wcw/4wcw_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4WCW">4WCW</a></td><td width="500"> Structure of Ribosomal Silencing Factor Bound to Mycobacterium tuberculosis Ribosome.</td><td width=200><ul><li>Li, X.</li><li>Sun, Q.</li><li>Jiang, C.</li><li>Yang, K.</li><li>Hung, L.W.</li><li>Zhang, J.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv1821</td><td>secA2</td><td width="300">Involved in protein export. May interact with the SECY/SECE subunits. SECA has a central role in coupling the hydrolysis of ATP to the transfer of PRE-secretory periplasmic and outer membrane proteins across the membrane.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ua/4uaq/4uaq_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4UAQ">4UAQ</a></td><td width="500"> Structural Similarities and Differences between Two Functionally Distinct SecA Proteins, Mycobacterium tuberculosis SecA1 and SecA2.</td><td width=200><ul><li>Swanson, S.</li><li>Ioerger, T.R.</li><li>Rigel, N.W.</li><li>Miller, B.K.</li><li>Braunstein, M.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv1821</td><td>secA2</td><td width="300">Involved in protein export. May interact with the SECY/SECE subunits. SECA has a central role in coupling the hydrolysis of ATP to the transfer of PRE-secretory periplasmic and outer membrane proteins across the membrane.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ua/4uaq/4uaq_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4UAQ">4UAQ</a></td><td width="500"> Structural Similarities and Differences between Two Functionally Distinct SecA Proteins, Mycobacterium tuberculosis SecA1 and SecA2.</td><td width=200><ul><li>Swanson, S.</li><li>Ioerger, T.R.</li><li>Rigel, N.W.</li><li>Miller, B.K.</li><li>Braunstein, M.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv3330</td><td>dacB1</td><td width="300">Involved in peptidoglycan synthesis (at final stages). Hydrolyzes the bound D-alanyl-D-alanine [catalytic activity: D-alanyl-D-alanine + H(2)O = 2 D-alanine].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/pp/4ppr/4ppr_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4PPR">4PPR</a></td><td width="500"> Subfamily-specific adaptations in the structures of two penicillin-binding proteins from Mycobacterium tuberculosis.</td><td width=200><ul><li>Prigozhin, D.M.</li><li>Krieger, I.V.</li><li>Huizar, J.P.</li><li>Mavrici, D.</li><li>Waldo, G.S.</li><li>Hung, L.W.</li><li>Sacchettini, J.C.</li><li>Terwilliger, T.C.</li><li>Alber, T.</li></ul></td></tr></table></td></tr><tr><td>Rv3330</td><td>dacB1</td><td width="300">Involved in peptidoglycan synthesis (at final stages). Hydrolyzes the bound D-alanyl-D-alanine [catalytic activity: D-alanyl-D-alanine + H(2)O = 2 D-alanine].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/pp/4ppr/4ppr_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4PPR">4PPR</a></td><td width="500"> Subfamily-specific adaptations in the structures of two penicillin-binding proteins from Mycobacterium tuberculosis.</td><td width=200><ul><li>Prigozhin, D.M.</li><li>Krieger, I.V.</li><li>Huizar, J.P.</li><li>Mavrici, D.</li><li>Waldo, G.S.</li><li>Hung, L.W.</li><li>Sacchettini, J.C.</li><li>Terwilliger, T.C.</li><li>Alber, T.</li></ul></td></tr></table></td></tr><tr><td>MSMEG_2671</td><td>folA</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/m2/4m2x/4m2x_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4M2X">4M2X</a></td><td width="500"> Selective inhibitors for dihydrofolate reductase from mycobacterium tuberculosis</td><td width=200><ul><li>Jung, H.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv2763c</td><td>dfrA</td><td width="300">Essential step for de novo glycine and purine synthesis  DNA precursor synthesis  and for the conversion of dUMP to dTMP [catalytic activity: 5 6 7 8-tetrahydrofolate + NADP(+) = 7 8-dihydrofolate + NADPH].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/m2/4m2x/4m2x_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4M2X">4M2X</a></td><td width="500"> Selective inhibitors for dihydrofolate reductase from mycobacterium tuberculosis</td><td width=200><ul><li>Jung, H.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv2763c</td><td>dfrA</td><td width="300">Essential step for de novo glycine and purine synthesis  DNA precursor synthesis  and for the conversion of dUMP to dTMP [catalytic activity: 5 6 7 8-tetrahydrofolate + NADP(+) = 7 8-dihydrofolate + NADPH].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/m2/4m2x/4m2x_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4M2X">4M2X</a></td><td width="500"> Selective inhibitors for dihydrofolate reductase from mycobacterium tuberculosis</td><td width=200><ul><li>Jung, H.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv0667</td><td>rpoB</td><td width="300">Catalyzes the transcription of DNA into RNA using the four ribonucleoside triphosphates as substrates [catalytic activity: N nucleoside triphosphate = N diphosphate + {RNA}(N)].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/kb/4kbm/4kbm_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4KBM">4KBM</a></td><td width="500"> Structure of the Mtb CarD/RNAP beta-Lobes Complex Reveals the Molecular Basis of Interaction and Presents a Distinct DNA-Binding Domain for Mtb CarD.</td><td width=200><ul><li>Gulten, G.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/kb/4kbj/4kbj_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4KBJ">4KBJ</a></td><td width="500"> Structure of the Mtb CarD/RNAP beta-Lobes Complex Reveals the Molecular Basis of Interaction and Presents a Distinct DNA-Binding Domain for Mtb CarD.</td><td width=200><ul><li>Gulten, G.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv0667</td><td>rpoB</td><td width="300">Catalyzes the transcription of DNA into RNA using the four ribonucleoside triphosphates as substrates [catalytic activity: N nucleoside triphosphate = N diphosphate + {RNA}(N)].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/kb/4kbm/4kbm_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4KBM">4KBM</a></td><td width="500"> Structure of the Mtb CarD/RNAP beta-Lobes Complex Reveals the Molecular Basis of Interaction and Presents a Distinct DNA-Binding Domain for Mtb CarD.</td><td width=200><ul><li>Gulten, G.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/kb/4kbj/4kbj_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4KBJ">4KBJ</a></td><td width="500"> Structure of the Mtb CarD/RNAP beta-Lobes Complex Reveals the Molecular Basis of Interaction and Presents a Distinct DNA-Binding Domain for Mtb CarD.</td><td width=200><ul><li>Gulten, G.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv3583c</td><td>Rv3583c</td><td width="300">Involved in transcriptional mechanism.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/kb/4kbm/4kbm_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4KBM">4KBM</a></td><td width="500"> Structure of the Mtb CarD/RNAP beta-Lobes Complex Reveals the Molecular Basis of Interaction and Presents a Distinct DNA-Binding Domain for Mtb CarD.</td><td width=200><ul><li>Gulten, G.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv2346c</td><td>esxO</td><td width="300">Function unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/gz/4gzr/4gzr_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4GZR">4GZR</a></td><td width="500"> Heterologous expression of mycobacterial Esx complexes in Escherichia coli for structural studies is facilitated by the use of maltose binding protein fusions.</td><td width=200><ul><li>Arbing, M.A.</li><li>Chan, S.</li><li>Harris, L.</li><li>Kuo, E.</li><li>Zhou, T.T.</li><li>Ahn, C.J.</li><li>Nguyen, L.</li><li>He, Q.</li><li>Lu, J.</li><li>Menchavez, P.T.</li><li>Shin, A.</li><li>Holton, T.</li><li>Sawaya, M.R.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv2347c</td><td>esxP</td><td width="300">Function unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/gz/4gzr/4gzr_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4GZR">4GZR</a></td><td width="500"> Heterologous expression of mycobacterial Esx complexes in Escherichia coli for structural studies is facilitated by the use of maltose binding protein fusions.</td><td width=200><ul><li>Arbing, M.A.</li><li>Chan, S.</li><li>Harris, L.</li><li>Kuo, E.</li><li>Zhou, T.T.</li><li>Ahn, C.J.</li><li>Nguyen, L.</li><li>He, Q.</li><li>Lu, J.</li><li>Menchavez, P.T.</li><li>Shin, A.</li><li>Holton, T.</li><li>Sawaya, M.R.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv2764c</td><td>thyA</td><td width="300">Involved in deoxyribonucleotide biosynthesis. Provides the sole de novo source of dTMP for DANA biosynthesis [catalytic activity: 5 10-methylenetetrahydrofolate + dUMP = dihydrofolate + dTMP].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/fo/4fox/4fox_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4FOX">4FOX</a></td><td width="500"> Crystal structure of binary and ternary complexes of thymidylate synthase (ThyA) from Mycobacterium tuberculosis: Insights into the selectivity and mode of inhibition</td><td width=200><ul><li>Reddy, M.C.M.</li><li>Bruning, J.B.</li><li>Sacchettini, J.C.</li><li>Harshbarger, W.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/fo/4fog/4fog_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4FOG">4FOG</a></td><td width="500"> Crystal structure of binary and ternary complexes of thymidylate synthase (ThyA) from Mycobacterium tuberculosis: Insights into the selectivity and mode of inhibition</td><td width=200><ul><li>Reddy, M.C.M.</li><li>Bruning, J.B.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/fo/4foa/4foa_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4FOA">4FOA</a></td><td width="500"> Crystal structure of binary and ternary complexes of thymidylate synthase (ThyA) from Mycobacterium tuberculosis: Insights into the selectivity and mode of inhibition</td><td width=200><ul><li>Reddy, M.C.M.</li><li>Bruning, J.B.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv2764c</td><td>thyA</td><td width="300">Involved in deoxyribonucleotide biosynthesis. Provides the sole de novo source of dTMP for DANA biosynthesis [catalytic activity: 5 10-methylenetetrahydrofolate + dUMP = dihydrofolate + dTMP].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/fo/4fox/4fox_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4FOX">4FOX</a></td><td width="500"> Crystal structure of binary and ternary complexes of thymidylate synthase (ThyA) from Mycobacterium tuberculosis: Insights into the selectivity and mode of inhibition</td><td width=200><ul><li>Reddy, M.C.M.</li><li>Bruning, J.B.</li><li>Sacchettini, J.C.</li><li>Harshbarger, W.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/fo/4fog/4fog_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4FOG">4FOG</a></td><td width="500"> Crystal structure of binary and ternary complexes of thymidylate synthase (ThyA) from Mycobacterium tuberculosis: Insights into the selectivity and mode of inhibition</td><td width=200><ul><li>Reddy, M.C.M.</li><li>Bruning, J.B.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/fo/4foa/4foa_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4FOA">4FOA</a></td><td width="500"> Crystal structure of binary and ternary complexes of thymidylate synthase (ThyA) from Mycobacterium tuberculosis: Insights into the selectivity and mode of inhibition</td><td width=200><ul><li>Reddy, M.C.M.</li><li>Bruning, J.B.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/qj/3qj7/3qj7_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3QJ7">3QJ7</a></td><td width="500"> Crystal structure of binary and ternary complexes of thymidylate synthase (ThyA) from Mycobacterium tuberculosis: insights into the selectivity and mode of inhibition</td><td width=200><ul><li>Reddy, M.C.M.</li><li>Bruning, J.B.</li><li>Harshbarger, W.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv0577</td><td>TB27.3</td><td width="300">Function unknown. Involved in ability to absorb neutral red stain.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ox/3oxh/3oxh_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3OXH">3OXH</a></td><td width="500"> Structural and Biophysical Characterization of the Mycobacterium tuberculosis Protein Rv0577, a Protein Associated with Neutral Red Staining of Virulent Tuberculosis Strains and Homologue of the Streptomyces coelicolor Protein KbpA.</td><td width=200><ul><li>Buchko, G.W.</li><li>Echols, N.</li><li>Flynn, E.M.</li><li>Ng, H.L.</li><li>Stephenson, S.</li><li>Kim, H.B.</li><li>Myler, P.J.</li><li>Terwilliger, T.C.</li><li>Alber, T.</li><li>Kim, C.Y.</li></ul></td></tr></table></td></tr><tr><td>Rv0577</td><td>TB27.3</td><td width="300">Function unknown. Involved in ability to absorb neutral red stain.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ox/3oxh/3oxh_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3OXH">3OXH</a></td><td width="500"> Structural and Biophysical Characterization of the Mycobacterium tuberculosis Protein Rv0577, a Protein Associated with Neutral Red Staining of Virulent Tuberculosis Strains and Homologue of the Streptomyces coelicolor Protein KbpA.</td><td width=200><ul><li>Buchko, G.W.</li><li>Echols, N.</li><li>Flynn, E.M.</li><li>Ng, H.L.</li><li>Stephenson, S.</li><li>Kim, H.B.</li><li>Myler, P.J.</li><li>Terwilliger, T.C.</li><li>Alber, T.</li><li>Kim, C.Y.</li></ul></td></tr></table></td></tr><tr><td>Rv2537c</td><td>aroD</td><td width="300">Involved at the third step in the biosynthesis of chorismate within the biosynthesis of aromatic amino acids (the shikimate pathway). Catalyze a trans-dehydration via an enolate intermediate [catalytic activity: 3-dehydroquinate = 3-dehydroshikimate + H(2)O].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/n8/3n8k/3n8k_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3N8K">3N8K</a></td><td width="500"> Structural investigation of inhibitor designs targeting 3-dehydroquinate dehydratase from the shikimate pathway of Mycobacterium tuberculosis.</td><td width=200><ul><li>Dias, M.V.</li><li>Snee, W.C.</li><li>Bromfield, K.M.</li><li>Payne, R.J.</li><li>Palaninathan, S.K.</li><li>Ciulli, A.</li><li>Howard, N.I.</li><li>Abell, C.</li><li>Sacchettini, J.C.</li><li>Blundell, T.L.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/n5/3n59/3n59_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3N59">3N59</a></td><td width="500"> Structural investigation of inhibitor designs targeting 3-dehydroquinate dehydratase from the shikimate pathway of Mycobacterium tuberculosis.</td><td width=200><ul><li>Dias, M.V.</li><li>Snee, W.C.</li><li>Bromfield, K.M.</li><li>Payne, R.J.</li><li>Palaninathan, S.K.</li><li>Ciulli, A.</li><li>Howard, N.I.</li><li>Abell, C.</li><li>Sacchettini, J.C.</li><li>Blundell, T.L.</li></ul></td></tr></table></td></tr><tr><td>MSMEG_2532</td><td>aroQ</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/n8/3n8k/3n8k_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3N8K">3N8K</a></td><td width="500"> Structural investigation of inhibitor designs targeting 3-dehydroquinate dehydratase from the shikimate pathway of Mycobacterium tuberculosis.</td><td width=200><ul><li>Dias, M.V.</li><li>Snee, W.C.</li><li>Bromfield, K.M.</li><li>Payne, R.J.</li><li>Palaninathan, S.K.</li><li>Ciulli, A.</li><li>Howard, N.I.</li><li>Abell, C.</li><li>Sacchettini, J.C.</li><li>Blundell, T.L.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/n5/3n59/3n59_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3N59">3N59</a></td><td width="500"> Structural investigation of inhibitor designs targeting 3-dehydroquinate dehydratase from the shikimate pathway of Mycobacterium tuberculosis.</td><td width=200><ul><li>Dias, M.V.</li><li>Snee, W.C.</li><li>Bromfield, K.M.</li><li>Payne, R.J.</li><li>Palaninathan, S.K.</li><li>Ciulli, A.</li><li>Howard, N.I.</li><li>Abell, C.</li><li>Sacchettini, J.C.</li><li>Blundell, T.L.</li></ul></td></tr></table></td></tr><tr><td>Rv2537c</td><td>aroD</td><td width="300">Involved at the third step in the biosynthesis of chorismate within the biosynthesis of aromatic amino acids (the shikimate pathway). Catalyze a trans-dehydration via an enolate intermediate [catalytic activity: 3-dehydroquinate = 3-dehydroshikimate + H(2)O].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/n8/3n8k/3n8k_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3N8K">3N8K</a></td><td width="500"> Structural investigation of inhibitor designs targeting 3-dehydroquinate dehydratase from the shikimate pathway of Mycobacterium tuberculosis.</td><td width=200><ul><li>Dias, M.V.</li><li>Snee, W.C.</li><li>Bromfield, K.M.</li><li>Payne, R.J.</li><li>Palaninathan, S.K.</li><li>Ciulli, A.</li><li>Howard, N.I.</li><li>Abell, C.</li><li>Sacchettini, J.C.</li><li>Blundell, T.L.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/n5/3n59/3n59_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3N59">3N59</a></td><td width="500"> Structural investigation of inhibitor designs targeting 3-dehydroquinate dehydratase from the shikimate pathway of Mycobacterium tuberculosis.</td><td width=200><ul><li>Dias, M.V.</li><li>Snee, W.C.</li><li>Bromfield, K.M.</li><li>Payne, R.J.</li><li>Palaninathan, S.K.</li><li>Ciulli, A.</li><li>Howard, N.I.</li><li>Abell, C.</li><li>Sacchettini, J.C.</li><li>Blundell, T.L.</li></ul></td></tr></table></td></tr><tr><td>MAB_3286c</td><td>ddl</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/lw/3lwb/3lwb_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3LWB">3LWB</a></td><td width="500"> Structure of the Mycobacterium tuberculosis D-Alanine:D-Alanine Ligase, a Target of the Antituberculosis Drug D-Cycloserine.</td><td width=200><ul><li>Bruning, J.B.</li><li>Murillo, A.C.</li><li>Chacon, O.</li><li>Barletta, R.G.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv2981c</td><td>ddlA</td><td width="300">Involved in cell wall formation. Along with alanine racemase  it MAKES up the D-alanine branch of the peptidoglycan biosynthetic route. [catalytic activity: ATP + D-alanine + D-alanine = ADP + phosphate + D-alanyl-D-alanine].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/lw/3lwb/3lwb_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3LWB">3LWB</a></td><td width="500"> Structure of the Mycobacterium tuberculosis D-Alanine:D-Alanine Ligase, a Target of the Antituberculosis Drug D-Cycloserine.</td><td width=200><ul><li>Bruning, J.B.</li><li>Murillo, A.C.</li><li>Chacon, O.</li><li>Barletta, R.G.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv2981c</td><td>ddlA</td><td width="300">Involved in cell wall formation. Along with alanine racemase  it MAKES up the D-alanine branch of the peptidoglycan biosynthetic route. [catalytic activity: ATP + D-alanine + D-alanine = ADP + phosphate + D-alanyl-D-alanine].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/lw/3lwb/3lwb_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3LWB">3LWB</a></td><td width="500"> Structure of the Mycobacterium tuberculosis D-Alanine:D-Alanine Ligase, a Target of the Antituberculosis Drug D-Cycloserine.</td><td width=200><ul><li>Bruning, J.B.</li><li>Murillo, A.C.</li><li>Chacon, O.</li><li>Barletta, R.G.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv1653</td><td>argJ</td><td width="300">Arginine biosynthesis.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/it/3it6/3it6_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3IT6">3IT6</a></td><td width="500"> The molecular structure of ornithine acetyltransferase from Mycobacterium tuberculosis bound to ornithine, a competitive inhibitor.</td><td width=200><ul><li>Sankaranarayanan, R.</li><li>Cherney, M.M.</li><li>Garen, C.</li><li>Garen, G.</li><li>Niu, C.</li><li>Yuan, M.</li><li>James, M.N.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/it/3it4/3it4_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3IT4">3IT4</a></td><td width="500"> The molecular structure of ornithine acetyltransferase from Mycobacterium tuberculosis bound to ornithine, a competitive inhibitor.</td><td width=200><ul><li>Sankaranarayanan, R.</li><li>Cherney, M.M.</li><li>Garen, C.</li><li>Garen, G.</li><li>Niu, C.</li><li>Yuan, M.</li><li>James, M.N.</li></ul></td></tr></table></td></tr><tr><td>Rv1653</td><td>argJ</td><td width="300">Arginine biosynthesis.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/it/3it6/3it6_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3IT6">3IT6</a></td><td width="500"> The molecular structure of ornithine acetyltransferase from Mycobacterium tuberculosis bound to ornithine, a competitive inhibitor.</td><td width=200><ul><li>Sankaranarayanan, R.</li><li>Cherney, M.M.</li><li>Garen, C.</li><li>Garen, G.</li><li>Niu, C.</li><li>Yuan, M.</li><li>James, M.N.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/it/3it4/3it4_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3IT4">3IT4</a></td><td width="500"> The molecular structure of ornithine acetyltransferase from Mycobacterium tuberculosis bound to ornithine, a competitive inhibitor.</td><td width=200><ul><li>Sankaranarayanan, R.</li><li>Cherney, M.M.</li><li>Garen, C.</li><li>Garen, G.</li><li>Niu, C.</li><li>Yuan, M.</li><li>James, M.N.</li></ul></td></tr></table></td></tr><tr><td>Rv0407</td><td>fgd1</td><td width="300">Catalyzes oxidation of glucose-6-phosphate to 6-phosphogluconolactone using coenzyme F420 (an *-hydroxy-5-deazaflavin derivative) as the electron acceptor.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/b4/3b4y/3b4y_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3B4Y">3B4Y</a></td><td width="500"> Crystal structures of F420-dependent glucose-6-phosphate dehydrogenase FGD1 involved in the activation of the anti-tuberculosis drug candidate PA-824 reveal the basis of coenzyme and substrate binding</td><td width=200><ul><li>Bashiri, G.</li><li>Squire, C.J.</li><li>Moreland, N.J.</li><li>Baker, E.N.</li></ul></td></tr></table></td></tr><tr><td>Rv0407</td><td>fgd1</td><td width="300">Catalyzes oxidation of glucose-6-phosphate to 6-phosphogluconolactone using coenzyme F420 (an *-hydroxy-5-deazaflavin derivative) as the electron acceptor.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/b4/3b4y/3b4y_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3B4Y">3B4Y</a></td><td width="500"> Crystal structures of F420-dependent glucose-6-phosphate dehydrogenase FGD1 involved in the activation of the anti-tuberculosis drug candidate PA-824 reveal the basis of coenzyme and substrate binding</td><td width=200><ul><li>Bashiri, G.</li><li>Squire, C.J.</li><li>Moreland, N.J.</li><li>Baker, E.N.</li></ul></td></tr></table></td></tr><tr><td>MAB_4230c</td><td>fgd</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/b4/3b4y/3b4y_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3B4Y">3B4Y</a></td><td width="500"> Crystal structures of F420-dependent glucose-6-phosphate dehydrogenase FGD1 involved in the activation of the anti-tuberculosis drug candidate PA-824 reveal the basis of coenzyme and substrate binding</td><td width=200><ul><li>Bashiri, G.</li><li>Squire, C.J.</li><li>Moreland, N.J.</li><li>Baker, E.N.</li></ul></td></tr></table></td></tr><tr><td>Rv1656</td><td>argF</td><td width="300">Involved in arginine biosynthesis [catalytic activity: carbamoyl phosphate + L-ornithine = phosphate + L-citrulline.]</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/p2/2p2g/2p2g_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2P2G">2P2G</a></td><td width="500"> The crystal structures of ornithine carbamoyltransferase from Mycobacterium tuberculosis and its ternary complex with carbamoyl phosphate and L-norvaline reveal the enzyme's catalytic mechanism.</td><td width=200><ul><li>Sankaranarayanan, R.</li><li>Cherney, M.M.</li><li>Cherney, L.T.</li><li>Garen, C.R.</li><li>Moradian, F.</li><li>James, M.N.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/i6/2i6u/2i6u_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2I6U">2I6U</a></td><td width="500"> The crystal structures of ornithine carbamoyltransferase from Mycobacterium tuberculosis and its ternary complex with carbamoyl phosphate and L-norvaline reveal the enzyme's catalytic mechanism</td><td width=200><ul><li>Sankaranarayanan, R.</li><li>Cherney, M.M.</li><li>Cherney, L.T.</li><li>Garen, C.R.</li><li>Moradian, F.</li><li>James, M.N.</li></ul></td></tr></table></td></tr><tr><td>Rv1656</td><td>argF</td><td width="300">Involved in arginine biosynthesis [catalytic activity: carbamoyl phosphate + L-ornithine = phosphate + L-citrulline.]</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/p2/2p2g/2p2g_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2P2G">2P2G</a></td><td width="500"> The crystal structures of ornithine carbamoyltransferase from Mycobacterium tuberculosis and its ternary complex with carbamoyl phosphate and L-norvaline reveal the enzyme's catalytic mechanism.</td><td width=200><ul><li>Sankaranarayanan, R.</li><li>Cherney, M.M.</li><li>Cherney, L.T.</li><li>Garen, C.R.</li><li>Moradian, F.</li><li>James, M.N.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/i6/2i6u/2i6u_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2I6U">2I6U</a></td><td width="500"> The crystal structures of ornithine carbamoyltransferase from Mycobacterium tuberculosis and its ternary complex with carbamoyl phosphate and L-norvaline reveal the enzyme's catalytic mechanism</td><td width=200><ul><li>Sankaranarayanan, R.</li><li>Cherney, M.M.</li><li>Cherney, L.T.</li><li>Garen, C.R.</li><li>Moradian, F.</li><li>James, M.N.</li></ul></td></tr></table></td></tr><tr><td>Rv1293</td><td>lysA</td><td width="300">Involved in biosynthesis of lysine (last step) [catalytic activity: MESO-2 6-diaminoheptanedioate = L-lysine + CO(2)].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/o0/2o0t/2o0t_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2O0T">2O0T</a></td><td width="500"> The three-dimensional structure of diaminopimelate decarboxylase from Mycobacterium tuberculosis reveals a tetrameric enzyme organisation.</td><td width=200><ul><li>Weyand, S.</li><li>Kefala, G.</li><li>Svergun, D.I.</li><li>Weiss, M.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/hk/1hkw/1hkw_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1HKW">1HKW</a></td><td width="500"> Crystal Structure of Mycobacterium Tuberculosis Diaminopimelate Decarboxylase, an Essential Enzyme in Bacterial Lysine Biosynthesis</td><td width=200><ul><li>Gokulan, K.</li><li>Rupp, B.</li><li>Pavelka, M.</li><li>Jacobs, W.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/hk/1hkv/1hkv_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1HKV">1HKV</a></td><td width="500"> Crystal Structure of Mycobacterium Tuberculosis Diaminopimelate Decarboxylase, an Essential Enzyme in Bacterial Lysine Biosynthesis</td><td width=200><ul><li>Gokulan, K.</li><li>Rupp, B.</li><li>Pavelka, M.</li><li>Jacobs, W.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv1293</td><td>lysA</td><td width="300">Involved in biosynthesis of lysine (last step) [catalytic activity: MESO-2 6-diaminoheptanedioate = L-lysine + CO(2)].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/o0/2o0t/2o0t_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2O0T">2O0T</a></td><td width="500"> The three-dimensional structure of diaminopimelate decarboxylase from Mycobacterium tuberculosis reveals a tetrameric enzyme organisation.</td><td width=200><ul><li>Weyand, S.</li><li>Kefala, G.</li><li>Svergun, D.I.</li><li>Weiss, M.S.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/hk/1hkw/1hkw_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1HKW">1HKW</a></td><td width="500"> Crystal Structure of Mycobacterium Tuberculosis Diaminopimelate Decarboxylase, an Essential Enzyme in Bacterial Lysine Biosynthesis</td><td width=200><ul><li>Gokulan, K.</li><li>Rupp, B.</li><li>Pavelka, M.</li><li>Jacobs, W.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/hk/1hkv/1hkv_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1HKV">1HKV</a></td><td width="500"> Crystal Structure of Mycobacterium Tuberculosis Diaminopimelate Decarboxylase, an Essential Enzyme in Bacterial Lysine Biosynthesis</td><td width=200><ul><li>Gokulan, K.</li><li>Rupp, B.</li><li>Pavelka, M.</li><li>Jacobs, W.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv1652</td><td>argC</td><td width="300">Involved in arginine biosynthesis (at the third step) [catalytic activity: N-acetyl-L-glutamate 5-semialdehyde + NADP(+) + phosphate = N-acetyl-5-glutamyl phosphate + NADPH].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/nq/2nqt/2nqt_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2NQT">2NQT</a></td><td width="500"> Crystal Structure of N-acetyl-gamma-glutamyl-phosphate Reductase from Mycobacterium tuberculosis in Complex with NADP(+).</td><td width=200><ul><li>Cherney, L.T.</li><li>Cherney, M.M.</li><li>Garen, C.R.</li><li>Niu, C.</li><li>Moradian, F.</li><li>James, M.N.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/i3/2i3g/2i3g_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2I3G">2I3G</a></td><td width="500"> Crystal Structure of N-acetyl-gamma-glutamyl-phosphate Reductase from Mycobacterium tuberculosis in Complex with NADP(+).</td><td width=200><ul><li>Cherney, L.T.</li><li>Cherney, M.M.</li><li>Garen, C.R.</li><li>Niu, C.</li><li>Moradian, F.</li><li>James, M.N.G.</li></ul></td></tr></table></td></tr><tr><td>Rv2246</td><td>kasB</td><td width="300">Involved in fatty acid biosynthesis (mycolic acids synthesis); involved in meromycolate extension. Catalyzes the condensation reaction of fatty acid synthesis by the addition to an acyl acceptor of two carbons from malonyl-ACP [catalytic activity: acyl-[acyl-carrier protein] + malonyl-[acyl-carrier protein] = 3-oxoacyl-[acyl-carrier protein] + [acyl-carrier protein] + CO(2)].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/gp/2gp6/2gp6_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2GP6">2GP6</a></td><td width="500"> X-Ray Crystal Structure of Mycobacterium tuberculosis beta-Ketoacyl Acyl Carrier Protein Synthase II (mtKasB).</td><td width=200><ul><li>Sridharan, S.</li><li>Wang, L.</li><li>Brown, A.K.</li><li>Dover, L.G.</li><li>Kremer, L.</li><li>Besra, G.S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv2827c</td><td>Rv2827c</td><td width="300">Unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ze/1zel/1zel_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1ZEL">1ZEL</a></td><td width="500"> Structural analysis reveals DNA binding properties of Rv2827c, a hypothetical protein from Mycobacterium tuberculosis.</td><td width=200><ul><li>Janowski, R.</li><li>Panjikar, S.</li><li>Eddine, A.N.</li><li>Kaufmann, S.H.</li><li>Weiss, M.S.</li></ul></td></tr></table></td></tr><tr><td>Rv1347c</td><td>mbtK</td><td width="300">Involved in the biogenesis of the hydroxyphenyloxazoline-containing siderophore mycobactins</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/yk/1yk3/1yk3_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1YK3">1YK3</a></td><td width="500"> The crystal structure of Rv1347c, a putative antibiotic resistance protein from Mycobacterium tuberculosis, reveals a GCN5-related fold and suggests an alternative function in siderophore biosynthesis</td><td width=200><ul><li>Card, G.L.</li><li>Peterson, N.A.</li><li>Smith, C.A.</li><li>Rupp, B.</li><li>Schick, B.M.</li><li>Baker, E.N.</li></ul></td></tr></table></td></tr><tr><td>Rv2753c</td><td>dapA</td><td width="300">Involved in biosynthesis of diaminopimelate and lysine from aspartate semialdehyde (at the first step) [catalytic activity: L-aspartate 4-semialdehyde + pyruvate = dihydrodipicolinate + 2 H(2)O].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/xx/1xxx/1xxx_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1XXX">1XXX</a></td><td width="500"> Crystal structure and kinetic study of dihydrodipicolinate synthase from Mycobacterium tuberculosis.</td><td width=200><ul><li>Kefala, G.</li><li>Evans, G.L.</li><li>Griffin, M.D.</li><li>Devenish, S.R.</li><li>Pearce, F.G.</li><li>Perugini, M.A.</li><li>Gerrard, J.A.</li><li>Weiss, M.S.</li><li>Dobson, R.C.</li></ul></td></tr></table></td></tr><tr><td>Rv1695</td><td>ppnK</td><td width="300">Catalyzes the phosphorylation of NAD to NADP. Utilizes ATP and other nucleoside triphosphates as well as inorganic polyphosphate as a source of phosphorus [catalytic activity: ATP + NAD+ = ADP + NADP+].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/u0/1u0t/1u0t_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1U0T">1U0T</a></td><td width="500"> A novel fold revealed by Mycobacterium tuberculosis NAD kinase, a key allosteric enzyme in NADP biosynthesis</td><td width=200><ul><li>Garavaglia, S.</li><li>Raffaelli, N.</li><li>Finaurini, L.</li><li>Magni, G.</li><li>Rizzi, M.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/u0/1u0r/1u0r_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1U0R">1U0R</a></td><td width="500"> A novel fold revealed by Mycobacterium tuberculosis NAD kinase, a key allosteric enzyme in NADP biosynthesis</td><td width=200><ul><li>Garavaglia, S.</li><li>Raffaelli, N.</li><li>Finaurini, L.</li><li>Magni, G.</li><li>Rizzi, M.</li></ul></td></tr></table></td></tr><tr><td>Rv2697c</td><td>dut</td><td width="300">Involved in biosynthesis of thymidylate. This enzyme is involved in nucleotide metabolism: it produces dUMP  the immediate precursor of thymidine nucleotides and it decreases the intracellular concentration of dUTP so that uracil cannot be incorporated into DNA [catalytic activity: dUTP + H(2)O = dUMP + pyrophosphate].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/sn/1snf/1snf_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1SNF">1SNF</a></td><td width="500"> Crystal structure of the Mycobacterium tuberculosis dUTPase: insights into the catalytic mechanism.</td><td width=200><ul><li>Chan, S.</li><li>Segelke, B.</li><li>Lekin, T.</li><li>Krupka, H.</li><li>Cho, U.S.</li><li>Kim, M.-Y.</li><li>So, M.</li><li>Kim, C.-Y.</li><li>Naranjo, C.M.</li><li>Rogers, Y.C.</li><li>Park, M.S.</li><li>Waldo, G.S.</li><li>Pashkov, I.</li><li>Cascio, D.</li><li>Perry, J.L.</li><li>Sawaya, M.R.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/sm/1smc/1smc_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1SMC">1SMC</a></td><td width="500"> Crystal structure of the Mycobacterium tuberculosis dUTPase: insights into the catalytic mechanism.</td><td width=200><ul><li>Chan, S.</li><li>Segelke, B.</li><li>Lekin, T.</li><li>Krupka, H.</li><li>Cho, U.S.</li><li>Kim, M.-Y.</li><li>So, M.</li><li>Kim, C.-Y.</li><li>Naranjo, C.M.</li><li>Rogers, Y.C.</li><li>Park, M.S.</li><li>Waldo, G.S.</li><li>Pashkov, I.</li><li>Cascio, D.</li><li>Perry, J.L.</li><li>Sawaya, M.R.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/sm/1sm8/1sm8_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1SM8">1SM8</a></td><td width="500"> Crystal structure of the Mycobacterium tuberculosis dUTPase: insights into the catalytic mechanism.</td><td width=200><ul><li>Chan, S.</li><li>Segelke, B.</li><li>Lekin, T.</li><li>Krupka, H.</li><li>Cho, U.S.</li><li>Kim, M.-Y.</li><li>So, M.</li><li>Kim, C.-Y.</li><li>Naranjo, C.M.</li><li>Rogers, Y.C.</li><li>Park, M.S.</li><li>Waldo, G.S.</li><li>Pashkov, I.</li><li>Cascio, D.</li><li>Perry, J.L.</li><li>Sawaya, M.R.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/sl/1slh/1slh_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1SLH">1SLH</a></td><td width="500"> Crystal structure of the Mycobacterium tuberculosis dUTPase: insights into the catalytic mechanism.</td><td width=200><ul><li>Chan, S.</li><li>Segelke, B.</li><li>Lekin, T.</li><li>Krupka, H.</li><li>Cho, U.S.</li><li>Kim, M.-Y.</li><li>So, M.</li><li>Kim, C.-Y.</li><li>Naranjo, C.M.</li><li>Rogers, Y.C.</li><li>Park, M.S.</li><li>Waldo, G.S.</li><li>Pashkov, I.</li><li>Cascio, D.</li><li>Perry, J.L.</li><li>Sawaya, M.R.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/sj/1sjn/1sjn_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1SJN">1SJN</a></td><td width="500"> Crystal structure of the Mycobacterium tuberculosis dUTPase: insights into the catalytic mechanism.</td><td width=200><ul><li>Chan, S.</li><li>Segelke, B.</li><li>Lekin, T.</li><li>Krupka, H.</li><li>Cho, U.S.</li><li>Kim, M.-Y.</li><li>So, M.</li><li>Kim, C.-Y.</li><li>Naranjo, C.M.</li><li>Rogers, Y.C.</li><li>Park, M.S.</li><li>Waldo, G.S.</li><li>Pashkov, I.</li><li>Cascio, D.</li><li>Perry, J.L.</li><li>Sawaya, M.R.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/si/1six/1six_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1SIX">1SIX</a></td><td width="500"> Crystal structure of the Mycobacterium tuberculosis dUTPase: insights into the catalytic mechanism.</td><td width=200><ul><li>Chan, S.</li><li>Segelke, B.</li><li>Lekin, T.</li><li>Krupka, H.</li><li>Cho, U.S.</li><li>Kim, M.-Y.</li><li>So, M.</li><li>Kim, C.-Y.</li><li>Naranjo, C.M.</li><li>Rogers, Y.C.</li><li>Park, M.S.</li><li>Waldo, G.S.</li><li>Pashkov, I.</li><li>Cascio, D.</li><li>Perry, J.L.</li><li>Sawaya, M.R.</li></ul></td></tr></table></td></tr><tr><td>Rv2697c</td><td>dut</td><td width="300">Involved in biosynthesis of thymidylate. This enzyme is involved in nucleotide metabolism: it produces dUMP  the immediate precursor of thymidine nucleotides and it decreases the intracellular concentration of dUTP so that uracil cannot be incorporated into DNA [catalytic activity: dUTP + H(2)O = dUMP + pyrophosphate].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/sn/1snf/1snf_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1SNF">1SNF</a></td><td width="500"> Crystal structure of the Mycobacterium tuberculosis dUTPase: insights into the catalytic mechanism.</td><td width=200><ul><li>Chan, S.</li><li>Segelke, B.</li><li>Lekin, T.</li><li>Krupka, H.</li><li>Cho, U.S.</li><li>Kim, M.-Y.</li><li>So, M.</li><li>Kim, C.-Y.</li><li>Naranjo, C.M.</li><li>Rogers, Y.C.</li><li>Park, M.S.</li><li>Waldo, G.S.</li><li>Pashkov, I.</li><li>Cascio, D.</li><li>Perry, J.L.</li><li>Sawaya, M.R.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/sm/1smc/1smc_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1SMC">1SMC</a></td><td width="500"> Crystal structure of the Mycobacterium tuberculosis dUTPase: insights into the catalytic mechanism.</td><td width=200><ul><li>Chan, S.</li><li>Segelke, B.</li><li>Lekin, T.</li><li>Krupka, H.</li><li>Cho, U.S.</li><li>Kim, M.-Y.</li><li>So, M.</li><li>Kim, C.-Y.</li><li>Naranjo, C.M.</li><li>Rogers, Y.C.</li><li>Park, M.S.</li><li>Waldo, G.S.</li><li>Pashkov, I.</li><li>Cascio, D.</li><li>Perry, J.L.</li><li>Sawaya, M.R.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/sm/1sm8/1sm8_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1SM8">1SM8</a></td><td width="500"> Crystal structure of the Mycobacterium tuberculosis dUTPase: insights into the catalytic mechanism.</td><td width=200><ul><li>Chan, S.</li><li>Segelke, B.</li><li>Lekin, T.</li><li>Krupka, H.</li><li>Cho, U.S.</li><li>Kim, M.-Y.</li><li>So, M.</li><li>Kim, C.-Y.</li><li>Naranjo, C.M.</li><li>Rogers, Y.C.</li><li>Park, M.S.</li><li>Waldo, G.S.</li><li>Pashkov, I.</li><li>Cascio, D.</li><li>Perry, J.L.</li><li>Sawaya, M.R.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/sl/1slh/1slh_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1SLH">1SLH</a></td><td width="500"> Crystal structure of the Mycobacterium tuberculosis dUTPase: insights into the catalytic mechanism.</td><td width=200><ul><li>Chan, S.</li><li>Segelke, B.</li><li>Lekin, T.</li><li>Krupka, H.</li><li>Cho, U.S.</li><li>Kim, M.-Y.</li><li>So, M.</li><li>Kim, C.-Y.</li><li>Naranjo, C.M.</li><li>Rogers, Y.C.</li><li>Park, M.S.</li><li>Waldo, G.S.</li><li>Pashkov, I.</li><li>Cascio, D.</li><li>Perry, J.L.</li><li>Sawaya, M.R.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/sj/1sjn/1sjn_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1SJN">1SJN</a></td><td width="500"> Crystal structure of the Mycobacterium tuberculosis dUTPase: insights into the catalytic mechanism.</td><td width=200><ul><li>Chan, S.</li><li>Segelke, B.</li><li>Lekin, T.</li><li>Krupka, H.</li><li>Cho, U.S.</li><li>Kim, M.-Y.</li><li>So, M.</li><li>Kim, C.-Y.</li><li>Naranjo, C.M.</li><li>Rogers, Y.C.</li><li>Park, M.S.</li><li>Waldo, G.S.</li><li>Pashkov, I.</li><li>Cascio, D.</li><li>Perry, J.L.</li><li>Sawaya, M.R.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/si/1six/1six_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1SIX">1SIX</a></td><td width="500"> Crystal structure of the Mycobacterium tuberculosis dUTPase: insights into the catalytic mechanism.</td><td width=200><ul><li>Chan, S.</li><li>Segelke, B.</li><li>Lekin, T.</li><li>Krupka, H.</li><li>Cho, U.S.</li><li>Kim, M.-Y.</li><li>So, M.</li><li>Kim, C.-Y.</li><li>Naranjo, C.M.</li><li>Rogers, Y.C.</li><li>Park, M.S.</li><li>Waldo, G.S.</li><li>Pashkov, I.</li><li>Cascio, D.</li><li>Perry, J.L.</li><li>Sawaya, M.R.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/mq/1mq7/1mq7_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1MQ7">1MQ7</a></td><td width="500"> Crystal structure of the Mycobacterium tuberculosis dUTPase: insights into the catalytic mechanism.</td><td width=200><ul><li>Chan, S.</li><li>Segelke, B.</li><li>Lekin, T.</li><li>Krupka, H.</li><li>Cho, U.S.</li><li>Kim, M.-Y.</li><li>So, M.</li><li>Kim, C.-Y.</li><li>Naranjo, C.M.</li><li>Rogers, Y.C.</li><li>Park, M.S.</li><li>Waldo, G.S.</li><li>Pashkov, I.</li><li>Cascio, D.</li><li>Perry, J.L.</li><li>Sawaya, M.R.</li></ul></td></tr></table></td></tr><tr><td>Mb2716c</td><td>dut</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/sn/1snf/1snf_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1SNF">1SNF</a></td><td width="500"> Crystal structure of the Mycobacterium tuberculosis dUTPase: insights into the catalytic mechanism.</td><td width=200><ul><li>Chan, S.</li><li>Segelke, B.</li><li>Lekin, T.</li><li>Krupka, H.</li><li>Cho, U.S.</li><li>Kim, M.-Y.</li><li>So, M.</li><li>Kim, C.-Y.</li><li>Naranjo, C.M.</li><li>Rogers, Y.C.</li><li>Park, M.S.</li><li>Waldo, G.S.</li><li>Pashkov, I.</li><li>Cascio, D.</li><li>Perry, J.L.</li><li>Sawaya, M.R.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/sm/1smc/1smc_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1SMC">1SMC</a></td><td width="500"> Crystal structure of the Mycobacterium tuberculosis dUTPase: insights into the catalytic mechanism.</td><td width=200><ul><li>Chan, S.</li><li>Segelke, B.</li><li>Lekin, T.</li><li>Krupka, H.</li><li>Cho, U.S.</li><li>Kim, M.-Y.</li><li>So, M.</li><li>Kim, C.-Y.</li><li>Naranjo, C.M.</li><li>Rogers, Y.C.</li><li>Park, M.S.</li><li>Waldo, G.S.</li><li>Pashkov, I.</li><li>Cascio, D.</li><li>Perry, J.L.</li><li>Sawaya, M.R.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/sm/1sm8/1sm8_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1SM8">1SM8</a></td><td width="500"> Crystal structure of the Mycobacterium tuberculosis dUTPase: insights into the catalytic mechanism.</td><td width=200><ul><li>Chan, S.</li><li>Segelke, B.</li><li>Lekin, T.</li><li>Krupka, H.</li><li>Cho, U.S.</li><li>Kim, M.-Y.</li><li>So, M.</li><li>Kim, C.-Y.</li><li>Naranjo, C.M.</li><li>Rogers, Y.C.</li><li>Park, M.S.</li><li>Waldo, G.S.</li><li>Pashkov, I.</li><li>Cascio, D.</li><li>Perry, J.L.</li><li>Sawaya, M.R.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/sl/1slh/1slh_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1SLH">1SLH</a></td><td width="500"> Crystal structure of the Mycobacterium tuberculosis dUTPase: insights into the catalytic mechanism.</td><td width=200><ul><li>Chan, S.</li><li>Segelke, B.</li><li>Lekin, T.</li><li>Krupka, H.</li><li>Cho, U.S.</li><li>Kim, M.-Y.</li><li>So, M.</li><li>Kim, C.-Y.</li><li>Naranjo, C.M.</li><li>Rogers, Y.C.</li><li>Park, M.S.</li><li>Waldo, G.S.</li><li>Pashkov, I.</li><li>Cascio, D.</li><li>Perry, J.L.</li><li>Sawaya, M.R.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/sj/1sjn/1sjn_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1SJN">1SJN</a></td><td width="500"> Crystal structure of the Mycobacterium tuberculosis dUTPase: insights into the catalytic mechanism.</td><td width=200><ul><li>Chan, S.</li><li>Segelke, B.</li><li>Lekin, T.</li><li>Krupka, H.</li><li>Cho, U.S.</li><li>Kim, M.-Y.</li><li>So, M.</li><li>Kim, C.-Y.</li><li>Naranjo, C.M.</li><li>Rogers, Y.C.</li><li>Park, M.S.</li><li>Waldo, G.S.</li><li>Pashkov, I.</li><li>Cascio, D.</li><li>Perry, J.L.</li><li>Sawaya, M.R.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/si/1six/1six_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1SIX">1SIX</a></td><td width="500"> Crystal structure of the Mycobacterium tuberculosis dUTPase: insights into the catalytic mechanism.</td><td width=200><ul><li>Chan, S.</li><li>Segelke, B.</li><li>Lekin, T.</li><li>Krupka, H.</li><li>Cho, U.S.</li><li>Kim, M.-Y.</li><li>So, M.</li><li>Kim, C.-Y.</li><li>Naranjo, C.M.</li><li>Rogers, Y.C.</li><li>Park, M.S.</li><li>Waldo, G.S.</li><li>Pashkov, I.</li><li>Cascio, D.</li><li>Perry, J.L.</li><li>Sawaya, M.R.</li></ul></td></tr></table></td></tr><tr><td>Rv0440</td><td>groEL2</td><td width="300">Prevents misfolding and promotes the refolding and proper assembly of unfolded polypeptides generated under stress conditions.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/sj/1sjp/1sjp_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1SJP">1SJP</a></td><td width="500"> Crystal Structure of the 65-Kilodalton Heat Shock Protein, Chaperonin 60.2, of Mycobacterium tuberculosis</td><td width=200><ul><li>Qamra, R.</li><li>Mande, S.C.</li></ul></td></tr></table></td></tr><tr><td>Rv0931c</td><td>pknD</td><td width="300">Involved in signal transduction (via phosphorylation). Thought to regulate phosphate transport. Can phosphorylate the peptide substrate myelin basic protein (MBP) at serine and threonine residues. Can be autophosphorylated on threonine residues [catalytic activity: ATP + a protein = ADP + a phosphoprotein].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/rw/1rwl/1rwl_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1RWL">1RWL</a></td><td width="500"> Sensor Domain of the Mycobacterium tuberculosis Receptor Ser/Thr Protein Kinase, PknD, forms a Highly Symmetric beta Propeller.</td><td width=200><ul><li>Good, M.C.</li><li>Greenstein, A.E.</li><li>Young, T.A.</li><li>Ng, H.L.</li><li>Alber, T.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/rw/1rwi/1rwi_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1RWI">1RWI</a></td><td width="500"> Sensor Domain of the Mycobacterium tuberculosis Receptor Ser/Thr Protein Kinase, PknD, forms a Highly Symmetric beta Propeller.</td><td width=200><ul><li>Good, M.C.</li><li>Greenstein, A.E.</li><li>Young, T.A.</li><li>Ng, H.L.</li><li>Alber, T.</li></ul></td></tr></table></td></tr><tr><td>Rv0931c</td><td>pknD</td><td width="300">Involved in signal transduction (via phosphorylation). Thought to regulate phosphate transport. Can phosphorylate the peptide substrate myelin basic protein (MBP) at serine and threonine residues. Can be autophosphorylated on threonine residues [catalytic activity: ATP + a protein = ADP + a phosphoprotein].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/rw/1rwl/1rwl_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1RWL">1RWL</a></td><td width="500"> Sensor Domain of the Mycobacterium tuberculosis Receptor Ser/Thr Protein Kinase, PknD, forms a Highly Symmetric beta Propeller.</td><td width=200><ul><li>Good, M.C.</li><li>Greenstein, A.E.</li><li>Young, T.A.</li><li>Ng, H.L.</li><li>Alber, T.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/rw/1rwi/1rwi_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1RWI">1RWI</a></td><td width="500"> Sensor Domain of the Mycobacterium tuberculosis Receptor Ser/Thr Protein Kinase, PknD, forms a Highly Symmetric beta Propeller.</td><td width=200><ul><li>Good, M.C.</li><li>Greenstein, A.E.</li><li>Young, T.A.</li><li>Ng, H.L.</li><li>Alber, T.</li></ul></td></tr></table></td></tr><tr><td>Rv1596</td><td>nadC</td><td width="300">de novo biosynthesis of NAD and NADP [catalytic activity: nicotinate D-ribonucleotide + diphosphate + CO(2) = pyridine-2 3-dicarboxylate + 5-phospho-alpha-D-ribose 1-diphosphate]</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/qp/1qpq/1qpq_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1QPQ">1QPQ</a></td><td width="500"> Crystal structure of quinolinic acid phosphoribosyltransferase from Mycobacterium tuberculosis: a potential TB drug target.</td><td width=200><ul><li>Sharma, V.</li><li>Grubmeyer, C.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/qp/1qpo/1qpo_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1QPO">1QPO</a></td><td width="500"> Crystal structure of quinolinic acid phosphoribosyltransferase from Mycobacterium tuberculosis: a potential TB drug target.</td><td width=200><ul><li>Sharma, V.</li><li>Grubmeyer, C.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/qp/1qpn/1qpn_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1QPN">1QPN</a></td><td width="500"> Crystal structure of quinolinic acid phosphoribosyltransferase from Mycobacterium tuberculosis: a potential TB drug target.</td><td width=200><ul><li>Sharma, V.</li><li>Grubmeyer, C.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/qp/1qpr/1qpr_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1QPR">1QPR</a></td><td width="500"> Crystal structure of quinolinic acid phosphoribosyltransferase from Mycobacterium tuberculosis: a potential TB drug target.</td><td width=200><ul><li>Sharma, V.</li><li>Grubmeyer, C.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv1596</td><td>nadC</td><td width="300">de novo biosynthesis of NAD and NADP [catalytic activity: nicotinate D-ribonucleotide + diphosphate + CO(2) = pyridine-2 3-dicarboxylate + 5-phospho-alpha-D-ribose 1-diphosphate]</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/qp/1qpq/1qpq_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1QPQ">1QPQ</a></td><td width="500"> Crystal structure of quinolinic acid phosphoribosyltransferase from Mycobacterium tuberculosis: a potential TB drug target.</td><td width=200><ul><li>Sharma, V.</li><li>Grubmeyer, C.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/qp/1qpo/1qpo_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1QPO">1QPO</a></td><td width="500"> Crystal structure of quinolinic acid phosphoribosyltransferase from Mycobacterium tuberculosis: a potential TB drug target.</td><td width=200><ul><li>Sharma, V.</li><li>Grubmeyer, C.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/qp/1qpn/1qpn_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1QPN">1QPN</a></td><td width="500"> Crystal structure of quinolinic acid phosphoribosyltransferase from Mycobacterium tuberculosis: a potential TB drug target.</td><td width=200><ul><li>Sharma, V.</li><li>Grubmeyer, C.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/qp/1qpr/1qpr_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1QPR">1QPR</a></td><td width="500"> Crystal structure of quinolinic acid phosphoribosyltransferase from Mycobacterium tuberculosis: a potential TB drug target.</td><td width=200><ul><li>Sharma, V.</li><li>Grubmeyer, C.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv1170</td><td>mshB</td><td width="300">Involved in mycothiol biosynthesis. 1-D-myo-inosityl-2-acetamido-2-deoxy-alpha-D-glucopyranoside (GlcNAc-Ins) is converted to 1-D-myo-inosityl-2-amino-2-deoxy-alpha-D-glucopyranoside (GlcN-Ins) by this enzyme. Seems to possess weak mycothiol conjugate amidase activity but SHOWS substantial deacetylation activity with 1-D-myo-inosityl-2-acetamido-2-deoxy-alpha-D-glucopyranoside (GlcNAc-Ins)  a hypothetical mycothiol biosynthetic precursor. GlcNAc-Ins is an intermediate in MSH biosynthesis.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/q7/1q74/1q74_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1Q74">1Q74</a></td><td width="500"> The Crystal Structure of 1-D-myo-Inositol 2-Acetamido-2-deoxy-alpha-D-glucopyranoside Deacetylase (MshB) from Mycobacterium tuberculosis Reveals a Zinc Hydrolase with a Lactate Dehydrogenase Fold.</td><td width=200><ul><li>Maynes, J.T.</li><li>Garen, C.</li><li>Cherney, M.M.</li><li>Newton, G.</li><li>Arad, D.</li><li>Av-Gay, Y.</li><li>Fahey, R.C.</li><li>James, M.N.</li></ul></td></tr></table></td></tr><tr><td>MMAR_5403</td><td>menG</td><td width="300"></td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/nx/1nxj/1nxj_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1NXJ">1NXJ</a></td><td width="500"> Crystal Structure of a Putative Methyltransferase from Mycobacterium tuberculosis: Misannotation of a Genome Clarified by Protein Structural Analysis</td><td width=200><ul><li>Johnston, J.M.</li><li>Arcus, V.L.</li><li>Morton, C.J.</li><li>Parker, M.W.</li><li>Baker, E.N.</li></ul></td></tr></table></td></tr><tr><td>Rv2121c</td><td>hisG</td><td width="300">Involved in histidine biosynthesis.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/nh/1nh7/1nh7_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1NH7">1NH7</a></td><td width="500"> Crystal Structure of ATP Phosphoribosyltransferase from Mycobacterium Tuberculosis</td><td width=200><ul><li>Cho, Y.</li><li>Sharma, V.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/nh/1nh8/1nh8_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1NH8">1NH8</a></td><td width="500"> Crystal structure of ATP phosphoribosyltransferase from Mycobacterium tuberculosis</td><td width=200><ul><li>Cho, Y.</li><li>Sharma, V.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv2121c</td><td>hisG</td><td width="300">Involved in histidine biosynthesis.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/nh/1nh7/1nh7_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1NH7">1NH7</a></td><td width="500"> Crystal Structure of ATP Phosphoribosyltransferase from Mycobacterium Tuberculosis</td><td width=200><ul><li>Cho, Y.</li><li>Sharma, V.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/nh/1nh8/1nh8_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1NH8">1NH8</a></td><td width="500"> Crystal structure of ATP phosphoribosyltransferase from Mycobacterium tuberculosis</td><td width=200><ul><li>Cho, Y.</li><li>Sharma, V.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv3392c</td><td>cmaA1</td><td width="300">Has cyclopropane function. Transfers a methylene group from S-adenosyl-L-methionine to the cis double bond of an unsaturated fatty acid chain resulting in the replacement of the double bond with a methylene bridge. Mycolic acids  which represent the major constituent of mycobacterial cell wall complex  act as substrates [catalytic activity: S-adenosyl-L-methionine + phospholipidolefinic fatty acid = S-adenosyl-L-homocysteine + phospholipid cyclopropane fatty acid].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/kp/1kpg/1kpg_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1KPG">1KPG</a></td><td width="500"> Crystal structures of mycolic acid cyclopropane synthases from Mycobacterium tuberculosis</td><td width=200><ul><li>Huang, C.-C.</li><li>Smith, C.V.</li><li>Glickman, M.S.</li><li>Jacobs Jr., W.R.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/kp/1kph/1kph_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1KPH">1KPH</a></td><td width="500"> Crystal structures of mycolic acid cyclopropane synthases from Mycobacterium tuberculosis</td><td width=200><ul><li>Huang, C.-C.</li><li>Smith, C.V.</li><li>Glickman, M.S.</li><li>Jacobs Jr., W.R.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/kp/1kp9/1kp9_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1KP9">1KP9</a></td><td width="500"> Crystal structures of mycolic acid cyclopropane synthases from Mycobacterium tuberculosis</td><td width=200><ul><li>Huang, C.-C.</li><li>Smith, C.V.</li><li>Glickman, M.S.</li><li>Jacobs Jr., W.R.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv2118c</td><td>Rv2118c</td><td width="300">Involved in transfer of methyl group (from S-adenosyl-L-methionine to the substrate).</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/i9/1i9g/1i9g_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1I9G">1I9G</a></td><td width="500"> Crystal structure of Rv2118c: an AdoMet-dependent methyltransferase from Mycobacterium tuberculosis H37Rv.</td><td width=200><ul><li>Gupta, A.</li><li>Kumar, P.H.</li><li>Dineshkumar, T.K.</li><li>Varshney, U.</li><li>Subramanya, H.S.</li></ul></td></tr></table></td></tr><tr><td>MMAR_0792</td><td>icl</td><td width="300"></td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/f6/1f61/1f61_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1F61">1F61</a></td><td width="500"> Structure of isocitrate lyase, a persistence factor of Mycobacterium tuberculosis.</td><td width=200><ul><li>Sharma, V.</li><li>Sharma, S.</li><li>Hoener zu Bentrup, K.</li><li>McKinney, J.D.</li><li>Russell, D.G.</li><li>Jacobs Jr., W.R.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/f8/1f8m/1f8m_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1F8M">1F8M</a></td><td width="500"> Structure of isocitrate lyase, a persistence factor of Mycobacterium tuberculosis.</td><td width=200><ul><li>Sharma, V.</li><li>Sharma, S.</li><li>Hoener zu Bentrup, K.</li><li>McKinney, J.D.</li><li>Russell, D.G.</li><li>Jacobs Jr., W.R.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/f8/1f8i/1f8i_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1F8I">1F8I</a></td><td width="500"> Structure of isocitrate lyase, a persistence factor of Mycobacterium tuberculosis.</td><td width=200><ul><li>Sharma, V.</li><li>Sharma, S.</li><li>Hoener zu Bentrup, K.</li><li>McKinney, J.D.</li><li>Russell, D.G.</li><li>Jacobs Jr., W.R.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv0467</td><td>icl1</td><td width="300">Involved in glyoxylate bypass (at the first step)  an alternative to the tricarboxylic acid cycle (in bacteria  plants  and fungi) [catalytic activity: isocitrate = succinate + glyoxylate]. Involved in the persistence in the host.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/f6/1f61/1f61_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1F61">1F61</a></td><td width="500"> Structure of isocitrate lyase, a persistence factor of Mycobacterium tuberculosis.</td><td width=200><ul><li>Sharma, V.</li><li>Sharma, S.</li><li>Hoener zu Bentrup, K.</li><li>McKinney, J.D.</li><li>Russell, D.G.</li><li>Jacobs Jr., W.R.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/f8/1f8m/1f8m_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1F8M">1F8M</a></td><td width="500"> Structure of isocitrate lyase, a persistence factor of Mycobacterium tuberculosis.</td><td width=200><ul><li>Sharma, V.</li><li>Sharma, S.</li><li>Hoener zu Bentrup, K.</li><li>McKinney, J.D.</li><li>Russell, D.G.</li><li>Jacobs Jr., W.R.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/f8/1f8i/1f8i_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1F8I">1F8I</a></td><td width="500"> Structure of isocitrate lyase, a persistence factor of Mycobacterium tuberculosis.</td><td width=200><ul><li>Sharma, V.</li><li>Sharma, S.</li><li>Hoener zu Bentrup, K.</li><li>McKinney, J.D.</li><li>Russell, D.G.</li><li>Jacobs Jr., W.R.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv2502c</td><td>accD1</td><td width="300">Involved in fatty acid metabolism.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/q0/4q0g/4q0g_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4Q0G">4Q0G</a></td><td width="500"> Crystal structure of Beta subunit of acyl-CoA carboxylase AccD1 from Mycobacterium tuberculosis</td><td width=200><ul><li>Bie, H.Y.</li><li>Yin, J.</li><li>James, M.N.G.</li><li>TB Structural Genomics Consortium (TBSGC)</li></ul></td></tr></table></td></tr><tr><td>Rv2502c</td><td>accD1</td><td width="300">Involved in fatty acid metabolism.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/q0/4q0g/4q0g_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4Q0G">4Q0G</a></td><td width="500"> Crystal structure of Beta subunit of acyl-CoA carboxylase AccD1 from Mycobacterium tuberculosis</td><td width=200><ul><li>Bie, H.Y.</li><li>Yin, J.</li><li>James, M.N.G.</li><li>TB Structural Genomics Consortium (TBSGC)</li></ul></td></tr></table></td></tr><tr><td>Rv1837c</td><td>glcB</td><td width="300">Involved in glyoxylate bypass (second step)  an alternative to the tricarboxylic acid cycle [catalytic activity: L-malate + CoA = acetyl-CoA + H(2)O + glyoxylate]</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/dl/6dlj/6dlj_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6DLJ">6DLJ</a></td><td width="500"> Anion-pi Interactions in Computer-Aided Drug Design: Modeling the Inhibition of Malate Synthase by Phenyl-Diketo Acids.</td><td width=200><ul><li>Ellenbarger, J.F.</li><li>Krieger, I.V.</li><li>Huang, H.L.</li><li>Gomez-Coca, S.</li><li>Ioerger, T.R.</li><li>Sacchettini, J.C.</li><li>Wheeler, S.E.</li><li>Dunbar, K.R.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/dl/6dl9/6dl9_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6DL9">6DL9</a></td><td width="500"> Anion-pi Interactions in Computer-Aided Drug Design: Modeling the Inhibition of Malate Synthase by Phenyl-Diketo Acids.</td><td width=200><ul><li>Ellenbarger, J.F.</li><li>Krieger, I.V.</li><li>Huang, H.L.</li><li>Gomez-Coca, S.</li><li>Ioerger, T.R.</li><li>Sacchettini, J.C.</li><li>Wheeler, S.E.</li><li>Dunbar, K.R.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/dk/6dko/6dko_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6DKO">6DKO</a></td><td width="500"> Anion-pi Interactions in Computer-Aided Drug Design: Modeling the Inhibition of Malate Synthase by Phenyl-Diketo Acids.</td><td width=200><ul><li>Ellenbarger, J.F.</li><li>Krieger, I.V.</li><li>Huang, H.L.</li><li>Gomez-Coca, S.</li><li>Ioerger, T.R.</li><li>Sacchettini, J.C.</li><li>Wheeler, S.E.</li><li>Dunbar, K.R.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c7/6c7b/6c7b_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6C7B">6C7B</a></td><td width="500"> Anion-pi Interactions in Computer-Aided Drug Design: Modeling the Inhibition of Malate Synthase by Phenyl-Diketo Acids.</td><td width=200><ul><li>Ellenbarger, J.F.</li><li>Krieger, I.V.</li><li>Huang, H.L.</li><li>Gomez-Coca, S.</li><li>Ioerger, T.R.</li><li>Sacchettini, J.C.</li><li>Wheeler, S.E.</li><li>Dunbar, K.R.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/as/6as6/6as6_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6AS6">6AS6</a></td><td width="500"> Anion-pi Interactions in Computer-Aided Drug Design: Modeling the Inhibition of Malate Synthase by Phenyl-Diketo Acids.</td><td width=200><ul><li>Ellenbarger, J.F.</li><li>Krieger, I.V.</li><li>Huang, H.L.</li><li>Gomez-Coca, S.</li><li>Ioerger, T.R.</li><li>Sacchettini, J.C.</li><li>Wheeler, S.E.</li><li>Dunbar, K.R.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/n8/1n8w/1n8w_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1N8W">1N8W</a></td><td width="500"> Biochemical and structural studies of malate synthase from Mycobacterium tuberculosis</td><td width=200><ul><li>Smith, C.V.</li><li>Huang, C.C.</li><li>Miczak, A.</li><li>Russell, D.G.</li><li>Sacchettini, J.C.</li><li>Honer Zu Bentrup, K.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/n8/1n8i/1n8i_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1N8I">1N8I</a></td><td width="500"> Biochemical and structural studies of malate synthase from Mycobacterium tuberculosis</td><td width=200><ul><li>Smith, C.V.</li><li>Huang, C.C.</li><li>Miczak, A.</li><li>Russell, D.G.</li><li>Sacchettini, J.C.</li><li>Honer Zu Bentrup, K.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ap/6apz/6apz_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6APZ">6APZ</a></td><td width="500"> Anion-pi Interactions in Computer-Aided Drug Design: Modeling the Inhibition of Malate Synthase by Phenyl-Diketo Acids.</td><td width=200><ul><li>Ellenbarger, J.F.</li><li>Krieger, I.V.</li><li>Huang, H.L.</li><li>Gomez-Coca, S.</li><li>Ioerger, T.R.</li><li>Sacchettini, J.C.</li><li>Wheeler, S.E.</li><li>Dunbar, K.R.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/dn/6dnp/6dnp_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6DNP">6DNP</a></td><td width="500"> Anion-pi Interactions in Computer-Aided Drug Design: Modeling the Inhibition of Malate Synthase by Phenyl-Diketo Acids.</td><td width=200><ul><li>Ellenbarger, J.F.</li><li>Krieger, I.V.</li><li>Huang, H.L.</li><li>Gomez-Coca, S.</li><li>Ioerger, T.R.</li><li>Sacchettini, J.C.</li><li>Wheeler, S.E.</li><li>Dunbar, K.R.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c8/6c8p/6c8p_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6C8P">6C8P</a></td><td width="500"> Anion-pi Interactions in Computer-Aided Drug Design: Modeling the Inhibition of Malate Synthase by Phenyl-Diketo Acids.</td><td width=200><ul><li>Ellenbarger, J.F.</li><li>Krieger, I.V.</li><li>Huang, H.L.</li><li>Gomez-Coca, S.</li><li>Ioerger, T.R.</li><li>Sacchettini, J.C.</li><li>Wheeler, S.E.</li><li>Dunbar, K.R.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c6/6c6o/6c6o_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6C6O">6C6O</a></td><td width="500"> Anion-pi Interactions in Computer-Aided Drug Design: Modeling the Inhibition of Malate Synthase by Phenyl-Diketo Acids.</td><td width=200><ul><li>Ellenbarger, J.F.</li><li>Krieger, I.V.</li><li>Huang, H.L.</li><li>Gomez-Coca, S.</li><li>Ioerger, T.R.</li><li>Sacchettini, J.C.</li><li>Wheeler, S.E.</li><li>Dunbar, K.R.</li></ul></td></tr></table></td></tr><tr><td>Rv2202c</td><td>adoK</td><td width="300">Adenosine phosphorylation</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c9/6c9s/6c9s_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6C9S">6C9S</a></td><td width="500"> Structure-Guided Drug Design of 6-Substituted Adenosine Analogues as Potent Inhibitors of Mycobacterium tuberculosis Adenosine Kinase.</td><td width=200><ul><li>Crespo, R.A.</li><li>Dang, Q.</li><li>Zhou, N.E.</li><li>Guthrie, L.M.</li><li>Snavely, T.C.</li><li>Dong, W.</li><li>Loesch, K.A.</li><li>Suzuki, T.</li><li>You, L.</li><li>Wang, W.</li><li>O'Malley, T.</li><li>Parish, T.</li><li>Olsen, D.B.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c9/6c9q/6c9q_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6C9Q">6C9Q</a></td><td width="500"> Structure-Guided Drug Design of 6-Substituted Adenosine Analogues as Potent Inhibitors of Mycobacterium tuberculosis Adenosine Kinase.</td><td width=200><ul><li>Crespo, R.A.</li><li>Dang, Q.</li><li>Zhou, N.E.</li><li>Guthrie, L.M.</li><li>Snavely, T.C.</li><li>Dong, W.</li><li>Loesch, K.A.</li><li>Suzuki, T.</li><li>You, L.</li><li>Wang, W.</li><li>O'Malley, T.</li><li>Parish, T.</li><li>Olsen, D.B.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/pk/2pkn/2pkn_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2PKN">2PKN</a></td><td width="500"> High resolution crystal structures of Mycobacterium tuberculosis adenosine kinase: insights into the mechanism and specificity of this novel prokaryotic enzyme</td><td width=200><ul><li>Reddy, M.C.M.</li><li>Palaninathan, S.K.</li><li>Shetty, N.D.</li><li>Owen, J.L.</li><li>Watson, M.D.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/pk/2pkk/2pkk_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2PKK">2PKK</a></td><td width="500"> High resolution crystal structures of Mycobacterium tuberculosis adenosine kinase: insights into the mechanism and specificity of this novel prokaryotic enzyme</td><td width=200><ul><li>Reddy, M.C.M.</li><li>Palaninathan, S.K.</li><li>Shetty, N.D.</li><li>Owen, J.L.</li><li>Watson, M.D.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c9/6c9v/6c9v_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6C9V">6C9V</a></td><td width="500"> Structure-Guided Drug Design of 6-Substituted Adenosine Analogues as Potent Inhibitors of Mycobacterium tuberculosis Adenosine Kinase.</td><td width=200><ul><li>Crespo, R.A.</li><li>Dang, Q.</li><li>Zhou, N.E.</li><li>Guthrie, L.M.</li><li>Snavely, T.C.</li><li>Dong, W.</li><li>Loesch, K.A.</li><li>Suzuki, T.</li><li>You, L.</li><li>Wang, W.</li><li>O'Malley, T.</li><li>Parish, T.</li><li>Olsen, D.B.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c9/6c9r/6c9r_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6C9R">6C9R</a></td><td width="500"> Structure-Guided Drug Design of 6-Substituted Adenosine Analogues as Potent Inhibitors of Mycobacterium tuberculosis Adenosine Kinase.</td><td width=200><ul><li>Crespo, R.A.</li><li>Dang, Q.</li><li>Zhou, N.E.</li><li>Guthrie, L.M.</li><li>Snavely, T.C.</li><li>Dong, W.</li><li>Loesch, K.A.</li><li>Suzuki, T.</li><li>You, L.</li><li>Wang, W.</li><li>O'Malley, T.</li><li>Parish, T.</li><li>Olsen, D.B.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c9/6c9p/6c9p_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6C9P">6C9P</a></td><td width="500"> Structure-Guided Drug Design of 6-Substituted Adenosine Analogues as Potent Inhibitors of Mycobacterium tuberculosis Adenosine Kinase.</td><td width=200><ul><li>Crespo, R.A.</li><li>Dang, Q.</li><li>Zhou, N.E.</li><li>Guthrie, L.M.</li><li>Snavely, T.C.</li><li>Dong, W.</li><li>Loesch, K.A.</li><li>Suzuki, T.</li><li>You, L.</li><li>Wang, W.</li><li>O'Malley, T.</li><li>Parish, T.</li><li>Olsen, D.B.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c9/6c9n/6c9n_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6C9N">6C9N</a></td><td width="500"> Structure-Guided Drug Design of 6-Substituted Adenosine Analogues as Potent Inhibitors of Mycobacterium tuberculosis Adenosine Kinase.</td><td width=200><ul><li>Crespo, R.A.</li><li>Dang, Q.</li><li>Zhou, N.E.</li><li>Guthrie, L.M.</li><li>Snavely, T.C.</li><li>Dong, W.</li><li>Loesch, K.A.</li><li>Suzuki, T.</li><li>You, L.</li><li>Wang, W.</li><li>O'Malley, T.</li><li>Parish, T.</li><li>Olsen, D.B.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c6/6c67/6c67_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6C67">6C67</a></td><td width="500"> Structure-Guided Drug Design of 6-Substituted Adenosine Analogues as Potent Inhibitors of Mycobacterium tuberculosis Adenosine Kinase.</td><td width=200><ul><li>Crespo, R.A.</li><li>Dang, Q.</li><li>Zhou, N.E.</li><li>Guthrie, L.M.</li><li>Snavely, T.C.</li><li>Dong, W.</li><li>Loesch, K.A.</li><li>Suzuki, T.</li><li>You, L.</li><li>Wang, W.</li><li>O'Malley, T.</li><li>Parish, T.</li><li>Olsen, D.B.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>MMAR_3246</td><td>cbhK</td><td width="300"></td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c9/6c9s/6c9s_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6C9S">6C9S</a></td><td width="500"> Structure-Guided Drug Design of 6-Substituted Adenosine Analogues as Potent Inhibitors of Mycobacterium tuberculosis Adenosine Kinase.</td><td width=200><ul><li>Crespo, R.A.</li><li>Dang, Q.</li><li>Zhou, N.E.</li><li>Guthrie, L.M.</li><li>Snavely, T.C.</li><li>Dong, W.</li><li>Loesch, K.A.</li><li>Suzuki, T.</li><li>You, L.</li><li>Wang, W.</li><li>O'Malley, T.</li><li>Parish, T.</li><li>Olsen, D.B.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c9/6c9q/6c9q_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6C9Q">6C9Q</a></td><td width="500"> Structure-Guided Drug Design of 6-Substituted Adenosine Analogues as Potent Inhibitors of Mycobacterium tuberculosis Adenosine Kinase.</td><td width=200><ul><li>Crespo, R.A.</li><li>Dang, Q.</li><li>Zhou, N.E.</li><li>Guthrie, L.M.</li><li>Snavely, T.C.</li><li>Dong, W.</li><li>Loesch, K.A.</li><li>Suzuki, T.</li><li>You, L.</li><li>Wang, W.</li><li>O'Malley, T.</li><li>Parish, T.</li><li>Olsen, D.B.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c9/6c9v/6c9v_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6C9V">6C9V</a></td><td width="500"> Structure-Guided Drug Design of 6-Substituted Adenosine Analogues as Potent Inhibitors of Mycobacterium tuberculosis Adenosine Kinase.</td><td width=200><ul><li>Crespo, R.A.</li><li>Dang, Q.</li><li>Zhou, N.E.</li><li>Guthrie, L.M.</li><li>Snavely, T.C.</li><li>Dong, W.</li><li>Loesch, K.A.</li><li>Suzuki, T.</li><li>You, L.</li><li>Wang, W.</li><li>O'Malley, T.</li><li>Parish, T.</li><li>Olsen, D.B.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c9/6c9r/6c9r_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6C9R">6C9R</a></td><td width="500"> Structure-Guided Drug Design of 6-Substituted Adenosine Analogues as Potent Inhibitors of Mycobacterium tuberculosis Adenosine Kinase.</td><td width=200><ul><li>Crespo, R.A.</li><li>Dang, Q.</li><li>Zhou, N.E.</li><li>Guthrie, L.M.</li><li>Snavely, T.C.</li><li>Dong, W.</li><li>Loesch, K.A.</li><li>Suzuki, T.</li><li>You, L.</li><li>Wang, W.</li><li>O'Malley, T.</li><li>Parish, T.</li><li>Olsen, D.B.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c9/6c9p/6c9p_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6C9P">6C9P</a></td><td width="500"> Structure-Guided Drug Design of 6-Substituted Adenosine Analogues as Potent Inhibitors of Mycobacterium tuberculosis Adenosine Kinase.</td><td width=200><ul><li>Crespo, R.A.</li><li>Dang, Q.</li><li>Zhou, N.E.</li><li>Guthrie, L.M.</li><li>Snavely, T.C.</li><li>Dong, W.</li><li>Loesch, K.A.</li><li>Suzuki, T.</li><li>You, L.</li><li>Wang, W.</li><li>O'Malley, T.</li><li>Parish, T.</li><li>Olsen, D.B.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c9/6c9n/6c9n_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6C9N">6C9N</a></td><td width="500"> Structure-Guided Drug Design of 6-Substituted Adenosine Analogues as Potent Inhibitors of Mycobacterium tuberculosis Adenosine Kinase.</td><td width=200><ul><li>Crespo, R.A.</li><li>Dang, Q.</li><li>Zhou, N.E.</li><li>Guthrie, L.M.</li><li>Snavely, T.C.</li><li>Dong, W.</li><li>Loesch, K.A.</li><li>Suzuki, T.</li><li>You, L.</li><li>Wang, W.</li><li>O'Malley, T.</li><li>Parish, T.</li><li>Olsen, D.B.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c6/6c67/6c67_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6C67">6C67</a></td><td width="500"> Structure-Guided Drug Design of 6-Substituted Adenosine Analogues as Potent Inhibitors of Mycobacterium tuberculosis Adenosine Kinase.</td><td width=200><ul><li>Crespo, R.A.</li><li>Dang, Q.</li><li>Zhou, N.E.</li><li>Guthrie, L.M.</li><li>Snavely, T.C.</li><li>Dong, W.</li><li>Loesch, K.A.</li><li>Suzuki, T.</li><li>You, L.</li><li>Wang, W.</li><li>O'Malley, T.</li><li>Parish, T.</li><li>Olsen, D.B.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv1837c</td><td>glcB</td><td width="300">Involved in glyoxylate bypass (second step)  an alternative to the tricarboxylic acid cycle [catalytic activity: L-malate + CoA = acetyl-CoA + H(2)O + glyoxylate]</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/au/6au9/6au9_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6AU9">6AU9</a></td><td width="500"> Anion-pi Interactions in Computer-Aided Drug Design: Modeling the Inhibition of Malate Synthase by Phenyl-Diketo Acids.</td><td width=200><ul><li>Ellenbarger, J.F.</li><li>Krieger, I.V.</li><li>Huang, H.L.</li><li>Gomez-Coca, S.</li><li>Ioerger, T.R.</li><li>Sacchettini, J.C.</li><li>Wheeler, S.E.</li><li>Dunbar, K.R.</li></ul></td></tr></table></td></tr><tr><td>Rv2386c</td><td>mbtI</td><td width="300">Involved in the biogenesis of the hydroxyphenyloxazoline-containing siderophore mycobactins. Possibly plays a role in the conversion of chorismate to salicilate (the starter unit for mycobactin siderophore construction).</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/st/3st6/3st6_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3ST6">3ST6</a></td><td width="500"> Implications of binding mode and active site flexibility for inhibitor potency against the salicylate synthase from Mycobacterium tuberculosis</td><td width=200><ul><li>Chi, G.</li><li>Manos-Turvey, A.</li><li>O'Connor, P.D.</li><li>Johnston, J.M.</li><li>Evans, G.L.</li><li>Baker, E.N.</li><li>Payne, R.J.</li><li>Lott, J.S.</li><li>Bulloch, E.M.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/rv/3rv9/3rv9_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3RV9">3RV9</a></td><td width="500"> Implications of binding mode and active site flexibility for inhibitor potency against the salicylate synthase from Mycobacterium tuberculosis</td><td width=200><ul><li>Chi, G.</li><li>Manos-Turvey, A.</li><li>O'Connor, P.D.</li><li>Johnston, J.M.</li><li>Evans, G.L.</li><li>Baker, E.N.</li><li>Payne, R.J.</li><li>Lott, J.S.</li><li>Bulloch, E.M.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/rv/3rv8/3rv8_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3RV8">3RV8</a></td><td width="500"> Implications of binding mode and active site flexibility for inhibitor potency against the salicylate synthase from Mycobacterium tuberculosis</td><td width=200><ul><li>Chi, G.</li><li>Manos-Turvey, A.</li><li>O'Connor, P.D.</li><li>Johnston, J.M.</li><li>Evans, G.L.</li><li>Baker, E.N.</li><li>Payne, R.J.</li><li>Lott, J.S.</li><li>Bulloch, E.M.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/rv/3rv7/3rv7_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3RV7">3RV7</a></td><td width="500"> Implications of binding mode and active site flexibility for inhibitor potency against the salicylate synthase from Mycobacterium tuberculosis</td><td width=200><ul><li>Chi, G.</li><li>Manos-Turvey, A.</li><li>O'Connor, P.D.</li><li>Johnston, J.M.</li><li>Evans, G.L.</li><li>Baker, E.N.</li><li>Payne, R.J.</li><li>Lott, J.S.</li><li>Bulloch, E.M.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/rv/3rv6/3rv6_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3RV6">3RV6</a></td><td width="500"> Implications of binding mode and active site flexibility for inhibitor potency against the salicylate synthase from Mycobacterium tuberculosis</td><td width=200><ul><li>Chi, G.</li><li>Manos-Turvey, A.</li><li>O'Connor, P.D.</li><li>Johnston, J.M.</li><li>Evans, G.L.</li><li>Baker, E.N.</li><li>Payne, R.J.</li><li>Lott, J.S.</li><li>Bulloch, E.M.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ve/3veh/3veh_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3VEH">3VEH</a></td><td width="500"> Implications of binding mode and active site flexibility for inhibitor potency against the salicylate synthase from Mycobacterium tuberculosis.</td><td width=200><ul><li>Chi, G.</li><li>Manos-Turvey, A.</li><li>O'Connor, P.D.</li><li>Johnston, J.M.</li><li>Evans, G.L.</li><li>Baker, E.N.</li><li>Payne, R.J.</li><li>Lott, J.S.</li><li>Bulloch, E.M.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/g5/2g5f/2g5f_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2G5F">2G5F</a></td><td width="500"> The Structure of MbtI from Mycobacterium tuberculosis, the First Enzyme in the Biosynthesis of the Siderophore Mycobactin, Reveals It To Be a Salicylate Synthase</td><td width=200><ul><li>Harrison, A.J.</li><li>Gardenborg, T.</li><li>Yu, M.</li><li>Middleditch, M.</li><li>Ramsay, R.J.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table></td></tr><tr><td>Rv3357</td><td>relJ</td><td width="300">Function unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/oe/3oei/3oei_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3OEI">3OEI</a></td><td width="500"> Crystal structure of Mycobacterium tuberculosis RelJK (Rv3357-Rv3358-RelBE3)</td><td width=200><ul><li>Miallau, L.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv3358</td><td>relK</td><td width="300">Function unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/oe/3oei/3oei_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3OEI">3OEI</a></td><td width="500"> Crystal structure of Mycobacterium tuberculosis RelJK (Rv3357-Rv3358-RelBE3)</td><td width=200><ul><li>Miallau, L.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv3676</td><td>crp</td><td width="300">Involved in transcriptional mechanism.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/i5/3i59/3i59_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3I59">3I59</a></td><td width="500"> Structural Insights into the Mechanism of the Allosteric Transitions of Mycobacterium tuberculosis cAMP Receptor Protein.</td><td width=200><ul><li>Reddy, M.C.</li><li>Palaninathan, S.K.</li><li>Bruning, J.B.</li><li>Thurman, C.</li><li>Smith, D.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/i5/3i54/3i54_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3I54">3I54</a></td><td width="500"> Structural Insights into the Mechanism of the Allosteric Transitions of Mycobacterium tuberculosis cAMP Receptor Protein.</td><td width=200><ul><li>Reddy, M.C.</li><li>Palaninathan, S.K.</li><li>Bruning, J.B.</li><li>Thurman, C.</li><li>Smith, D.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>MSMEG_1843</td><td>ahcY</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/dh/3dhy/3dhy_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3DHY">3DHY</a></td><td width="500"> Crystal structures of Mycobacterium tuberculosis S-adenosyl-L-homocysteine hydrolase in ternary complex with substrate and inhibitors.</td><td width=200><ul><li>Reddy, M.C.</li><li>Kuppan, G.</li><li>Shetty, N.D.</li><li>Owen, J.L.</li><li>Ioerger, T.R.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv3248c</td><td>sahH</td><td width="300">Thioester hydrolase which acting on ether bounds. Could be involved in methionine and selenoamino acid metabolisms. Also involved in activated methyl. Cycle adenosylhomocysteine is a competitive inhibitor of S-adenosyl-L-methionine-dependent methyl transferase reactions; therefore adenosylhomocysteinase may play a key role in the control of methylations via regulation of the intracellular concentration of adenosylhomocysteine [catalytic activity: S-adenosyl-L-homocysteine + H(2)O = adenosine + L-homocysteine].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/dh/3dhy/3dhy_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3DHY">3DHY</a></td><td width="500"> Crystal structures of Mycobacterium tuberculosis S-adenosyl-L-homocysteine hydrolase in ternary complex with substrate and inhibitors.</td><td width=200><ul><li>Reddy, M.C.</li><li>Kuppan, G.</li><li>Shetty, N.D.</li><li>Owen, J.L.</li><li>Ioerger, T.R.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv3248c</td><td>sahH</td><td width="300">Thioester hydrolase which acting on ether bounds. Could be involved in methionine and selenoamino acid metabolisms. Also involved in activated methyl. Cycle adenosylhomocysteine is a competitive inhibitor of S-adenosyl-L-methionine-dependent methyl transferase reactions; therefore adenosylhomocysteinase may play a key role in the control of methylations via regulation of the intracellular concentration of adenosylhomocysteine [catalytic activity: S-adenosyl-L-homocysteine + H(2)O = adenosine + L-homocysteine].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/dh/3dhy/3dhy_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3DHY">3DHY</a></td><td width="500"> Crystal structures of Mycobacterium tuberculosis S-adenosyl-L-homocysteine hydrolase in ternary complex with substrate and inhibitors.</td><td width=200><ul><li>Reddy, M.C.</li><li>Kuppan, G.</li><li>Shetty, N.D.</li><li>Owen, J.L.</li><li>Ioerger, T.R.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv1018c</td><td>glmU</td><td width="300">Peptidoglycan and lipopolysaccharide biosynthesis</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/qk/2qkx/2qkx_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2QKX">2QKX</a></td><td width="500"> Structure and function of GlmU from Mycobacterium tuberculosis.</td><td width=200><ul><li>Zhang, Z.</li><li>Bulloch, E.M.</li><li>Bunker, R.D.</li><li>Baker, E.N.</li><li>Squire, C.J.</li></ul></td></tr></table></td></tr><tr><td>Rv1018c</td><td>glmU</td><td width="300">Peptidoglycan and lipopolysaccharide biosynthesis</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/qk/2qkx/2qkx_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2QKX">2QKX</a></td><td width="500"> Structure and function of GlmU from Mycobacterium tuberculosis.</td><td width=200><ul><li>Zhang, Z.</li><li>Bulloch, E.M.</li><li>Bunker, R.D.</li><li>Baker, E.N.</li><li>Squire, C.J.</li></ul></td></tr></table></td></tr><tr><td>Rv1652</td><td>argC</td><td width="300">Involved in arginine biosynthesis (at the third step) [catalytic activity: N-acetyl-L-glutamate 5-semialdehyde + NADP(+) + phosphate = N-acetyl-5-glutamyl phosphate + NADPH].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/i3/2i3g/2i3g_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2I3G">2I3G</a></td><td width="500"> Crystal Structure of N-acetyl-gamma-glutamyl-phosphate Reductase from Mycobacterium tuberculosis in Complex with NADP(+).</td><td width=200><ul><li>Cherney, L.T.</li><li>Cherney, M.M.</li><li>Garen, C.R.</li><li>Niu, C.</li><li>Moradian, F.</li><li>James, M.N.G.</li></ul></td></tr></table></td></tr><tr><td>Rv2431c</td><td>PE25</td><td width="300">Function unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/g3/2g38/2g38_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2G38">2G38</a></td><td width="500"> Toward the structural genomics of complexes: Crystal structure of a PE/PPE protein complex from Mycobacterium tuberculosis.</td><td width=200><ul><li>Strong, M.</li><li>Sawaya, M.R.</li><li>Wang, S.</li><li>Phillips, M.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv2430c</td><td>PPE41</td><td width="300">Function unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/g3/2g38/2g38_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2G38">2G38</a></td><td width="500"> Toward the structural genomics of complexes: Crystal structure of a PE/PPE protein complex from Mycobacterium tuberculosis.</td><td width=200><ul><li>Strong, M.</li><li>Sawaya, M.R.</li><li>Wang, S.</li><li>Phillips, M.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv2430c</td><td>PPE41</td><td width="300">Function unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/g3/2g38/2g38_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2G38">2G38</a></td><td width="500"> Toward the structural genomics of complexes: Crystal structure of a PE/PPE protein complex from Mycobacterium tuberculosis.</td><td width=200><ul><li>Strong, M.</li><li>Sawaya, M.R.</li><li>Wang, S.</li><li>Phillips, M.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv1625c</td><td>cya</td><td width="300">Involved in cAMP synthesis [catalytic activity: ATP = 3' 5'-cyclic AMP + diphosphate]. Inhibited by polyphosphates.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/yk/1yk9/1yk9_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1YK9">1YK9</a></td><td width="500"> A Structural Basis for the Role of Nucleotide Specifying Residues in Regulating the Oligomerization of the Rv1625c Adenylyl Cyclase from M.tuberculosis</td><td width=200><ul><li>Ketkar, A.D.</li><li>Shenoy, A.R.</li><li>Ramagopal, U.A.</li><li>Visweswariah, S.S.</li><li>Suguna, K.</li></ul></td></tr></table></td></tr><tr><td>Rv2996c</td><td>serA1</td><td width="300">Involved at the first committed step in the 'phosphorylated' pathway of L-serine biosynthesis [catalytic activity: 3-phosphoglycerate + NAD(+) = 3-phosphohydroxypyruvate + NADH].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/yg/1ygy/1ygy_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1YGY">1YGY</a></td><td width="500"> Crystal Structure of Mycobacterium tuberculosis D-3-Phosphoglycerate Dehydrogenase: EXTREME ASYMMETRY IN A TETRAMER OF IDENTICAL SUBUNITS</td><td width=200><ul><li>Dey, S.</li><li>Grant, G.A.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv0533c</td><td>fabH</td><td width="300">Involved in fatty acid biosynthesis. Catalyzes the condensation reaction of fatty acid synthesis by the addition to an acyl acceptor of two carbons from malonyl-ACP. KAS III catalyzes the first condensation reaction which initiates fatty acid synthesis and may therefore play a role in governing the total rate of fatty acid production. Possesses both acetoacetyl-ACP synthase and acetyl transacylase activities [catalytic activity: acyl-[acyl-carrier protein] + malonyl-[acyl-carrier protein] = 3-oxoacyl-[acyl-carrier protein] + CO2 + [acyl-carrier protein]].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/m1/1m1m/1m1m_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1M1M">1M1M</a></td><td width="500"> Probing the mechanism of the Mycobacterium tuberculosis beta-ketoacyl-acyl carrier protein synthase III mtFabH: factors influencing catalysis and substrate specificity.</td><td width=200><ul><li>Brown, A.K.</li><li>Sridharan, S.</li><li>Kremer, L.</li><li>Lindenberg, S.</li><li>Dover, L.G.</li><li>Sacchettini, J.C.</li><li>Besra, G.S.</li></ul></td></tr></table></td></tr><tr><td>Rv0470c</td><td>pcaA</td><td width="300">Involved in the mycolic acid modification or synthesis; essential for the cyclopropanation function. Required for cording and mycolic acid cyclopropane ring synthesis in the cell wall.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/l1/1l1e/1l1e_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1L1E">1L1E</a></td><td width="500"> Crystal structures of mycolic acid cyclopropane synthases from Mycobacterium tuberculosis.</td><td width=200><ul><li>Huang, C.-C.</li><li>Smith, C.V.</li><li>Glickman, M.S.</li><li>Jacobs Jr., W.R.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv0503c</td><td>cmaA2</td><td width="300">Essential for the cyclopropanation function. Transfers a methylene group from S-adenosyl-L-methionine to the cis double bond of an unsaturated fatty acid chain resulting in the replacement of the double bond with a methylene bridge. Mycolic acids  which represent the major constituent of mycobacterial cell wall complex  act as substrates [catalytic activity: S-adenosyl-L-methionine + phospholipid olefinic fatty acid = S-adenosyl-L-homocysteine + phospholipid cyclopropane fatty acid].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/kp/1kpi/1kpi_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1KPI">1KPI</a></td><td width="500"> Crystal structures of mycolic acid cyclopropane synthases from Mycobacterium tuberculosis</td><td width=200><ul><li>Huang, C.-C.</li><li>Smith, C.V.</li><li>Glickman, M.S.</li><li>Jacobs Jr., W.R.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv3763</td><td>lpqH</td><td width="300">Shown to inhibit gamma interferon regulated HLA-DR protein and mRNA expression in human macrophages</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/zj/4zjm/4zjm_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4ZJM">4ZJM</a></td><td width="500"> Crystal Structure of Mycobacterium tuberculosis LpqH (Rv3763)</td><td width=200><ul><li>Arbing, M.A.</li><li>Chan, S.</li><li>Kuo, E.</li><li>Harris, L.R.</li><li>Zhou, T.T.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv3763</td><td>lpqH</td><td width="300">Shown to inhibit gamma interferon regulated HLA-DR protein and mRNA expression in human macrophages</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/zj/4zjm/4zjm_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4ZJM">4ZJM</a></td><td width="500"> Crystal Structure of Mycobacterium tuberculosis LpqH (Rv3763)</td><td width=200><ul><li>Arbing, M.A.</li><li>Chan, S.</li><li>Kuo, E.</li><li>Harris, L.R.</li><li>Zhou, T.T.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>MSMEG_6933</td><td>trxB</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/a8/2a87/2a87_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2A87">2A87</a></td><td width="500"> Conformational flexibility of Mycobacterium tuberculosis thioredoxin reductase: crystal structure and normal-mode analysis.</td><td width=200><ul><li>Akif, M.</li><li>Suhre, K.</li><li>Verma, C.</li><li>Mande, S.C.</li></ul></td></tr></table></td></tr><tr><td>Rv0938</td><td>ligD</td><td width="300">Involved in DNA double-strand break repair  by nonhomologous end joining (NHEJ). Interacts with KU (Rv0937c) [catalytic activity:ATP + {deoxyribonucleotide}(N) + {deoxyribonucleotide}(M) = AMP + diphosphate + {deoxyribonucleotide}(N+M)].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/vs/1vs0/1vs0_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1VS0">1VS0</a></td><td width="500"> Crystal Structure and Nonhomologous End-joining Function of the Ligase Component of Mycobacterium DNA Ligase D.</td><td width=200><ul><li>Akey, D.</li><li>Martins, A.</li><li>Aniukwu, J.</li><li>Glickman, M.S.</li><li>Shuman, S.</li><li>Berger, J.M.</li></ul></td></tr></table></td></tr><tr><td>Rv0938</td><td>ligD</td><td width="300">Involved in DNA double-strand break repair  by nonhomologous end joining (NHEJ). Interacts with KU (Rv0937c) [catalytic activity:ATP + {deoxyribonucleotide}(N) + {deoxyribonucleotide}(M) = AMP + diphosphate + {deoxyribonucleotide}(N+M)].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/vs/1vs0/1vs0_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1VS0">1VS0</a></td><td width="500"> Crystal Structure and Nonhomologous End-joining Function of the Ligase Component of Mycobacterium DNA Ligase D.</td><td width=200><ul><li>Akey, D.</li><li>Martins, A.</li><li>Aniukwu, J.</li><li>Glickman, M.S.</li><li>Shuman, S.</li><li>Berger, J.M.</li></ul></td></tr></table></td></tr><tr><td>MSMEG_3706</td><td>aceA</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/vb/6vb9/6vb9_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6VB9">6VB9</a></td><td width="500"> Covalent Inactivation of Mycobacterium tuberculosis Isocitrate Lyase by cis -2,3-Epoxy-Succinic Acid.</td><td width=200><ul><li>Pham, T.V.</li><li>Mellott, D.M.</li><li>Moghadamchargari, Z.</li><li>Chen, K.</li><li>Krieger, I.</li><li>Laganowsky, A.</li><li>Sacchettini, J.C.</li><li>Meek, T.D.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ws/6wsi/6wsi_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6WSI">6WSI</a></td><td width="500"> Covalent Inactivation of Mycobacterium tuberculosis Isocitrate Lyase by cis -2,3-Epoxy-Succinic Acid.</td><td width=200><ul><li>Pham, T.V.</li><li>Mellott, D.M.</li><li>Moghadamchargari, Z.</li><li>Chen, K.</li><li>Krieger, I.</li><li>Laganowsky, A.</li><li>Sacchettini, J.C.</li><li>Meek, T.D.</li></ul></td></tr></table></td></tr><tr><td>Rv2995c</td><td>leuB</td><td width="300">Involved in leucine biosynthesis (at the third step) [catalytic activity: 3-carboxy-2-hydroxy-4-methylpentanoate + NAD(+) = 3-carboxy-4-methyl-2-oxopentanoate + NADH (the product decarboxylates to 4-methyl-2-oxopentanoate)].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/w0/1w0d/1w0d_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1W0D">1W0D</a></td><td width="500"> The High Resolution Structure of Leub (Rv2995C) from Mycobacterium Tuberculosis</td><td width=200><ul><li>Singh, R.K.</li><li>Kefala, G.</li><li>Janowski, R.</li><li>Mueller-Dieckmann, C.</li><li>Von Kries, J.P.</li><li>Weiss, M.S.</li></ul></td></tr></table></td></tr><tr><td>Rv2995c</td><td>leuB</td><td width="300">Involved in leucine biosynthesis (at the third step) [catalytic activity: 3-carboxy-2-hydroxy-4-methylpentanoate + NAD(+) = 3-carboxy-4-methyl-2-oxopentanoate + NADH (the product decarboxylates to 4-methyl-2-oxopentanoate)].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/w0/1w0d/1w0d_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1W0D">1W0D</a></td><td width="500"> The High Resolution Structure of Leub (Rv2995C) from Mycobacterium Tuberculosis</td><td width=200><ul><li>Singh, R.K.</li><li>Kefala, G.</li><li>Janowski, R.</li><li>Mueller-Dieckmann, C.</li><li>Von Kries, J.P.</li><li>Weiss, M.S.</li></ul></td></tr></table></td></tr><tr><td>Rv1346</td><td>mbtN</td><td width="300">Thought to be involved in the biogenesis of the hydroxyphenyloxazoline-containing siderophore mycobactins</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/xv/4xvx/4xvx_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4XVX">4XVX</a></td><td width="500"> A covalent adduct of MbtN, an acyl-ACP dehydrogenase from Mycobacterium tuberculosis, reveals an unusual acyl-binding pocket.</td><td width=200><ul><li>Chai, A.F.</li><li>Bulloch, E.M.</li><li>Evans, G.L.</li><li>Lott, J.S.</li><li>Baker, E.N.</li><li>Johnston, J.M.</li></ul></td></tr></table></td></tr><tr><td>Rv1346</td><td>mbtN</td><td width="300">Thought to be involved in the biogenesis of the hydroxyphenyloxazoline-containing siderophore mycobactins</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/xv/4xvx/4xvx_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4XVX">4XVX</a></td><td width="500"> A covalent adduct of MbtN, an acyl-ACP dehydrogenase from Mycobacterium tuberculosis, reveals an unusual acyl-binding pocket.</td><td width=200><ul><li>Chai, A.F.</li><li>Bulloch, E.M.</li><li>Evans, G.L.</li><li>Lott, J.S.</li><li>Baker, E.N.</li><li>Johnston, J.M.</li></ul></td></tr></table></td></tr><tr><td>Rv3356c</td><td>folD</td><td width="300">Necessary for the biosynthesis of purines  thymydylate  methionine  histidine  pantothenate  and formyl tRNA-met [catalytic activity: 5 10-methylenetetrahydrofolate + NADP(+) = 5 10-methenyltetrahydrofolate + NADPH] [catalytic activity: 5 10-methenyltetrahydrofolate + H(2)O = 10-formyltetrahydrofolate].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c2/2c2y/2c2y_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2C2Y">2C2Y</a></td><td width="500"> Three Dimensional Structure of Bifunctional Methylenetetrahydrofolate Dehydrogenase-Cyclohydrolase from Mycobacterium tuberculosis</td><td width=200><ul><li>Javid-Majd, F.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c2/2c2x/2c2x_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2C2X">2C2X</a></td><td width="500"> Three Dimensional Structure of Bifunctional Methylenetetrahydrofolate Dehydrogenase- Cyclohydrolase from Mycobacterium tuberculosis</td><td width=200><ul><li>Javid-Majd, F.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv3356c</td><td>folD</td><td width="300">Necessary for the biosynthesis of purines  thymydylate  methionine  histidine  pantothenate  and formyl tRNA-met [catalytic activity: 5 10-methylenetetrahydrofolate + NADP(+) = 5 10-methenyltetrahydrofolate + NADPH] [catalytic activity: 5 10-methenyltetrahydrofolate + H(2)O = 10-formyltetrahydrofolate].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c2/2c2y/2c2y_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2C2Y">2C2Y</a></td><td width="500"> Three Dimensional Structure of Bifunctional Methylenetetrahydrofolate Dehydrogenase-Cyclohydrolase from Mycobacterium tuberculosis</td><td width=200><ul><li>Javid-Majd, F.</li><li>Sacchettini, J.C.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c2/2c2x/2c2x_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2C2X">2C2X</a></td><td width="500"> Three Dimensional Structure of Bifunctional Methylenetetrahydrofolate Dehydrogenase- Cyclohydrolase from Mycobacterium tuberculosis</td><td width=200><ul><li>Javid-Majd, F.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv3875</td><td>esxA</td><td width="300">Unknown. Elicits high level of inf-gamma from memory effector cells during the first phase of a protective immune response. Exported protein cotranscribed with Rv3874|MT3988|MTV027.09|LHP|CFP10.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/wa/1wa8/1wa8_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1WA8">1WA8</a></td><td width="500"> Structure and Function of the Complex Formed by the Tuberculosis Virulence Factors Cfp-10 and Esat-6</td><td width=200><ul><li>Renshaw, P.S.</li><li>Lightbody, K.L.</li><li>Veverka, V.</li><li>Muskett, F.W.</li><li>Kelly, G.</li><li>Frenkiel, T.A.</li><li>Gordon, S.V.</li><li>Hewinson, R.G.</li><li>Burke, B.</li><li>Norman, J.</li><li>Williamson, R.A.</li><li>Carr, M.D.</li></ul></td></tr></table></td></tr><tr><td>Rv3875</td><td>esxA</td><td width="300">Unknown. Elicits high level of inf-gamma from memory effector cells during the first phase of a protective immune response. Exported protein cotranscribed with Rv3874|MT3988|MTV027.09|LHP|CFP10.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/wa/1wa8/1wa8_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1WA8">1WA8</a></td><td width="500"> Structure and Function of the Complex Formed by the Tuberculosis Virulence Factors Cfp-10 and Esat-6</td><td width=200><ul><li>Renshaw, P.S.</li><li>Lightbody, K.L.</li><li>Veverka, V.</li><li>Muskett, F.W.</li><li>Kelly, G.</li><li>Frenkiel, T.A.</li><li>Gordon, S.V.</li><li>Hewinson, R.G.</li><li>Burke, B.</li><li>Norman, J.</li><li>Williamson, R.A.</li><li>Carr, M.D.</li></ul></td></tr></table></td></tr><tr><td>MAB_3112</td><td>MAB_3112</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/i0/4i0x/4i0x_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4I0X">4I0X</a></td><td width="500"> Heterologous expression of mycobacterial Esx complexes in Escherichia coli for structural studies is facilitated by the use of maltose binding protein fusions.</td><td width=200><ul><li>Arbing, M.A.</li><li>Chan, S.</li><li>Harris, L.</li><li>Kuo, E.</li><li>Zhou, T.T.</li><li>Ahn, C.J.</li><li>Nguyen, L.</li><li>He, Q.</li><li>Lu, J.</li><li>Menchavez, P.T.</li><li>Shin, A.</li><li>Holton, T.</li><li>Sawaya, M.R.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>MAB_3113</td><td>MAB_3113</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/i0/4i0x/4i0x_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/4I0X">4I0X</a></td><td width="500"> Heterologous expression of mycobacterial Esx complexes in Escherichia coli for structural studies is facilitated by the use of maltose binding protein fusions.</td><td width=200><ul><li>Arbing, M.A.</li><li>Chan, S.</li><li>Harris, L.</li><li>Kuo, E.</li><li>Zhou, T.T.</li><li>Ahn, C.J.</li><li>Nguyen, L.</li><li>He, Q.</li><li>Lu, J.</li><li>Menchavez, P.T.</li><li>Shin, A.</li><li>Holton, T.</li><li>Sawaya, M.R.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv3020c</td><td>esxS</td><td width="300">Function unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/h6/3h6p/3h6p_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3H6P">3H6P</a></td><td width="500"> Crystal structure of Rv3019c-Rv3020c from Mycobacterium tuberculosis</td><td width=200><ul><li>Chan, S.</li><li>Arbing, M.</li><li>Phan, T.</li><li>Kaufmann, M.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv3019c</td><td>esxR</td><td width="300">Unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/h6/3h6p/3h6p_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3H6P">3H6P</a></td><td width="500"> Crystal structure of Rv3019c-Rv3020c from Mycobacterium tuberculosis</td><td width=200><ul><li>Chan, S.</li><li>Arbing, M.</li><li>Phan, T.</li><li>Kaufmann, M.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv3020c</td><td>esxS</td><td width="300">Function unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/h6/3h6p/3h6p_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3H6P">3H6P</a></td><td width="500"> Crystal structure of Rv3019c-Rv3020c from Mycobacterium tuberculosis</td><td width=200><ul><li>Chan, S.</li><li>Arbing, M.</li><li>Phan, T.</li><li>Kaufmann, M.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv3019c</td><td>esxR</td><td width="300">Unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/h6/3h6p/3h6p_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3H6P">3H6P</a></td><td width="500"> Crystal structure of Rv3019c-Rv3020c from Mycobacterium tuberculosis</td><td width=200><ul><li>Chan, S.</li><li>Arbing, M.</li><li>Phan, T.</li><li>Kaufmann, M.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv0066c</td><td>icd2</td><td width="300">Involved in the KREBS cycle [catalytic activity: isocitrate + NADP+ = 2-oxoglutarate + CO(2) + NADPH].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/kv/5kvu/5kvu_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5KVU">5KVU</a></td><td width="500"> Structural and kinetic characterization of isocitrate dehydrogenase-2 from Mycobacterium tuberculosis</td><td width=200><ul><li>Sacchettini, J.C.</li><li>Cheng, Y.S.</li></ul></td></tr></table></td></tr><tr><td>Rv0066c</td><td>icd2</td><td width="300">Involved in the KREBS cycle [catalytic activity: isocitrate + NADP+ = 2-oxoglutarate + CO(2) + NADPH].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/kv/5kvu/5kvu_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5KVU">5KVU</a></td><td width="500"> Structural and kinetic characterization of isocitrate dehydrogenase-2 from Mycobacterium tuberculosis</td><td width=200><ul><li>Sacchettini, J.C.</li><li>Cheng, Y.S.</li></ul></td></tr></table></td></tr><tr><td>MMAR_0548</td><td>espG3</td><td width="300"></td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/dl/5dlb/5dlb_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5DLB">5DLB</a></td><td width="500"> Structural Variability of EspG Chaperones from Mycobacterial ESX-1, ESX-3, and ESX-5 Type VII Secretion Systems.</td><td width=200><ul><li>Tuukkanen, A.T.</li><li>Freire, D.</li><li>Chan, S.</li><li>Arbing, M.A.</li><li>Reed, R.W.</li><li>Evans, T.J.</li><li>Zenkeviciute, G.</li><li>Kim, J.</li><li>Kahng, S.</li><li>Sawaya, M.R.</li><li>Chaton, C.T.</li><li>Wilmanns, M.</li><li>Eisenberg, D.</li><li>Parret, A.H.A.</li><li>Korotkov, K.V.</li></ul></td></tr></table></td></tr><tr><td>Rv3275c</td><td>purE</td><td width="300">Involved in purine biosynthesis (sixth step). This subunit can alone transform air to CAIR  but in association with PURK  which possesses an ATPase activity  an enzyme complex is produced which is capable of converting air to CAIR efficiently under physiological condition [catalytic activity: 1-(5-phosphoribosyl)-5-amino-4-imidazole-carboxylate = 1-(5-phosphoribosyl)-5-aminoimidazole + CO(2)].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/lp/3lp6/3lp6_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3LP6">3LP6</a></td><td width="500"> Crystal Structure of Rv3275c-E60A from Mycobacterium tuberculosis at 1.7A 
Resolution</td><td width=200><ul><li>Kim, H.</li><li>Yu, M.</li><li>Hung, L.-W.</li><li>Terwilliger, T.C.</li><li>Kim, C.-Y.</li></ul></td></tr></table></td></tr><tr><td>Rv3275c</td><td>purE</td><td width="300">Involved in purine biosynthesis (sixth step). This subunit can alone transform air to CAIR  but in association with PURK  which possesses an ATPase activity  an enzyme complex is produced which is capable of converting air to CAIR efficiently under physiological condition [catalytic activity: 1-(5-phosphoribosyl)-5-amino-4-imidazole-carboxylate = 1-(5-phosphoribosyl)-5-aminoimidazole + CO(2)].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/lp/3lp6/3lp6_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3LP6">3LP6</a></td><td width="500"> Crystal Structure of Rv3275c-E60A from Mycobacterium tuberculosis at 1.7A 
Resolution</td><td width=200><ul><li>Kim, H.</li><li>Yu, M.</li><li>Hung, L.-W.</li><li>Terwilliger, T.C.</li><li>Kim, C.-Y.</li></ul></td></tr></table></td></tr><tr><td>Rv0006</td><td>gyrA</td><td width="300">DNA gyrase negatively supercoils closed circular double-stranded DNA in an ATP-dependent manner and also catalyzes the interconversion of other topological isomers of double-stranded DNA rings  including catenanes and knotted rings [catalytic activity: ATP-dependent breakage  passage and rejoining of double-stranded DNA].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/il/3ilw/3ilw_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3ILW">3ILW</a></td><td width="500"> Crystal structure of the DNA gyrase GyrA N-terminal domain from Mycobacterium tuberculosis.</td><td width=200><ul><li>Tretter, E.M.</li><li>Schoeffler, A.J.</li><li>Weisfield, S.R.</li><li>Berger, J.M.</li></ul></td></tr></table></td></tr><tr><td>Rv0006</td><td>gyrA</td><td width="300">DNA gyrase negatively supercoils closed circular double-stranded DNA in an ATP-dependent manner and also catalyzes the interconversion of other topological isomers of double-stranded DNA rings  including catenanes and knotted rings [catalytic activity: ATP-dependent breakage  passage and rejoining of double-stranded DNA].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/il/3ilw/3ilw_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3ILW">3ILW</a></td><td width="500"> Crystal structure of the DNA gyrase GyrA N-terminal domain from Mycobacterium tuberculosis.</td><td width=200><ul><li>Tretter, E.M.</li><li>Schoeffler, A.J.</li><li>Weisfield, S.R.</li><li>Berger, J.M.</li></ul></td></tr></table></td></tr><tr><td>Rv2865</td><td>relF</td><td width="300">Function unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/g5/3g5o/3g5o_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3G5O">3G5O</a></td><td width="500"> The crystal structure of the toxin-antitoxin complex RelBE2 (Rv2865-2866) from Mycobacterium tuberculosis</td><td width=200><ul><li>Miallau, L.</li><li>Chernishof, I.</li><li>Chiang, J.</li><li>Arbing, M.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv2865</td><td>relF</td><td width="300">Function unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/g5/3g5o/3g5o_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3G5O">3G5O</a></td><td width="500"> The crystal structure of the toxin-antitoxin complex RelBE2 (Rv2865-2866) from Mycobacterium tuberculosis</td><td width=200><ul><li>Miallau, L.</li><li>Chernishof, I.</li><li>Chiang, J.</li><li>Arbing, M.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv2866</td><td>relG</td><td width="300">Function unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/g5/3g5o/3g5o_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3G5O">3G5O</a></td><td width="500"> The crystal structure of the toxin-antitoxin complex RelBE2 (Rv2865-2866) from Mycobacterium tuberculosis</td><td width=200><ul><li>Miallau, L.</li><li>Chernishof, I.</li><li>Chiang, J.</li><li>Arbing, M.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv2866</td><td>relG</td><td width="300">Function unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/g5/3g5o/3g5o_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3G5O">3G5O</a></td><td width="500"> The crystal structure of the toxin-antitoxin complex RelBE2 (Rv2865-2866) from Mycobacterium tuberculosis</td><td width=200><ul><li>Miallau, L.</li><li>Chernishof, I.</li><li>Chiang, J.</li><li>Arbing, M.</li><li>Cascio, D.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv0858c</td><td>dapC</td><td width="300">Involved in biosynthesis of diaminopimelate and lysine from aspartate semialdehyde (at the fourth step) [catalytic activity: L-glutamate + N-succinyl-2-amino-6-ketopimelate = 2-oxoglutarate + N-succinyl-L L-2 6-diaminopimelate</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/o0/2o0r/2o0r_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2O0R">2O0R</a></td><td width="500"> The Three-dimensional Structure of N-Succinyldiaminopimelate Aminotransferase from Mycobacterium tuberculosis</td><td width=200><ul><li>Weyand, S.</li><li>Kefala, G.</li><li>Weiss, M.S.</li></ul></td></tr></table></td></tr><tr><td>Rv0858c</td><td>dapC</td><td width="300">Involved in biosynthesis of diaminopimelate and lysine from aspartate semialdehyde (at the fourth step) [catalytic activity: L-glutamate + N-succinyl-2-amino-6-ketopimelate = 2-oxoglutarate + N-succinyl-L L-2 6-diaminopimelate</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/o0/2o0r/2o0r_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2O0R">2O0R</a></td><td width="500"> The Three-dimensional Structure of N-Succinyldiaminopimelate Aminotransferase from Mycobacterium tuberculosis</td><td width=200><ul><li>Weyand, S.</li><li>Kefala, G.</li><li>Weiss, M.S.</li></ul></td></tr></table></td></tr><tr><td>MAB_4049c</td><td>gpmA</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ri/1rii/1rii_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1RII">1RII</a></td><td width="500"> The 1.70 angstroms X-ray crystal structure of Mycobacterium tuberculosis phosphoglycerate mutase.</td><td width=200><ul><li>Muller, P.</li><li>Sawaya, M.R.</li><li>Pashkov, I.</li><li>Chan, S.</li><li>Nguyen, C.</li><li>Wu, Y.</li><li>Perry, L.J.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>MSMEG_2136</td><td>pgm</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ri/1rii/1rii_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1RII">1RII</a></td><td width="500"> The 1.70 angstroms X-ray crystal structure of Mycobacterium tuberculosis phosphoglycerate mutase.</td><td width=200><ul><li>Muller, P.</li><li>Sawaya, M.R.</li><li>Pashkov, I.</li><li>Chan, S.</li><li>Nguyen, C.</li><li>Wu, Y.</li><li>Perry, L.J.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv0489</td><td>gpm1</td><td width="300">Involved in glycolysis [catalytic activity: 1 3-diphosphoglycerate + 3-phosphoglycerate = 2 3-diphosphoglycerate + 3-phosphoglycerate].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ri/1rii/1rii_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1RII">1RII</a></td><td width="500"> The 1.70 angstroms X-ray crystal structure of Mycobacterium tuberculosis phosphoglycerate mutase.</td><td width=200><ul><li>Muller, P.</li><li>Sawaya, M.R.</li><li>Pashkov, I.</li><li>Chan, S.</li><li>Nguyen, C.</li><li>Wu, Y.</li><li>Perry, L.J.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv0489</td><td>gpm1</td><td width="300">Involved in glycolysis [catalytic activity: 1 3-diphosphoglycerate + 3-phosphoglycerate = 2 3-diphosphoglycerate + 3-phosphoglycerate].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ri/1rii/1rii_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1RII">1RII</a></td><td width="500"> The 1.70 angstroms X-ray crystal structure of Mycobacterium tuberculosis phosphoglycerate mutase.</td><td width=200><ul><li>Muller, P.</li><li>Sawaya, M.R.</li><li>Pashkov, I.</li><li>Chan, S.</li><li>Nguyen, C.</li><li>Wu, Y.</li><li>Perry, L.J.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Mb0499</td><td>gpm1</td><td width="300">unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ri/1rii/1rii_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/1RII">1RII</a></td><td width="500"> The 1.70 angstroms X-ray crystal structure of Mycobacterium tuberculosis phosphoglycerate mutase.</td><td width=200><ul><li>Muller, P.</li><li>Sawaya, M.R.</li><li>Pashkov, I.</li><li>Chan, S.</li><li>Nguyen, C.</li><li>Wu, Y.</li><li>Perry, L.J.</li><li>Eisenberg, D.</li></ul></td></tr></table></td></tr><tr><td>Rv2202c</td><td>adoK</td><td width="300">Adenosine phosphorylation</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/c6/6c67/6c67_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6C67">6C67</a></td><td width="500"> Structure-Guided Drug Design of 6-Substituted Adenosine Analogues as Potent Inhibitors of Mycobacterium tuberculosis Adenosine Kinase.</td><td width=200><ul><li>Crespo, R.A.</li><li>Dang, Q.</li><li>Zhou, N.E.</li><li>Guthrie, L.M.</li><li>Snavely, T.C.</li><li>Dong, W.</li><li>Loesch, K.A.</li><li>Suzuki, T.</li><li>You, L.</li><li>Wang, W.</li><li>O'Malley, T.</li><li>Parish, T.</li><li>Olsen, D.B.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv1240</td><td>mdh</td><td width="300">Involved in the conversion of malate to oxaloacetate [catalytic activity: (S)-malate + NAD+ = oxaloacetate + NADH].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/kv/5kvv/5kvv_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5KVV">5KVV</a></td><td width="500"> Structure of Malate Dehydrogenase in complex with NADH from Mycobacterium tuberculosis</td><td width=200><ul><li>Cheng, Y.S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv1240</td><td>mdh</td><td width="300">Involved in the conversion of malate to oxaloacetate [catalytic activity: (S)-malate + NAD+ = oxaloacetate + NADH].</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/kv/5kvv/5kvv_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/5KVV">5KVV</a></td><td width="500"> Structure of Malate Dehydrogenase in complex with NADH from Mycobacterium tuberculosis</td><td width=200><ul><li>Cheng, Y.S.</li><li>Sacchettini, J.C.</li></ul></td></tr></table></td></tr><tr><td>Rv2386c</td><td>mbtI</td><td width="300">Involved in the biogenesis of the hydroxyphenyloxazoline-containing siderophore mycobactins. Possibly plays a role in the conversion of chorismate to salicilate (the starter unit for mycobactin siderophore construction).</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ve/3veh/3veh_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3VEH">3VEH</a></td><td width="500"> Implications of binding mode and active site flexibility for inhibitor potency against the salicylate synthase from Mycobacterium tuberculosis.</td><td width=200><ul><li>Chi, G.</li><li>Manos-Turvey, A.</li><li>O'Connor, P.D.</li><li>Johnston, J.M.</li><li>Evans, G.L.</li><li>Baker, E.N.</li><li>Payne, R.J.</li><li>Lott, J.S.</li><li>Bulloch, E.M.</li></ul></td></tr></table><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/g5/2g5f/2g5f_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2G5F">2G5F</a></td><td width="500"> The Structure of MbtI from Mycobacterium tuberculosis, the First Enzyme in the Biosynthesis of the Siderophore Mycobactin, Reveals It To Be a Salicylate Synthase</td><td width=200><ul><li>Harrison, A.J.</li><li>Gardenborg, T.</li><li>Yu, M.</li><li>Middleditch, M.</li><li>Ramsay, R.J.</li><li>Baker, E.N.</li><li>Lott, J.S.</li></ul></td></tr></table></td></tr><tr><td>Rv1873</td><td>Rv1873</td><td width="300">Unknown</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/je/2jek/2jek_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/2JEK">2JEK</a></td><td width="500"> The Molecular Structure of Rv1873, a Conserved Hypothetical Protein from Mycobacterium Tuberculosis, at 1.38A Resolution</td><td width=200><ul><li>Garen, C.R.</li><li>Cherney, M.M.</li><li>Bergmann, E.M.</li><li>James, M.N.G.</li></ul></td></tr></table></td></tr><tr><td>Rv1747</td><td>Rv1747</td><td width="300">Thought to be involved in active transport of undetermined substrate (possibly lipooligosaccharide) across the membrane. Responsible for energy coupling to the transport system and for the translocation of the substrate across the membrane.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/cc/6ccd/6ccd_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6CCD">6CCD</a></td><td width="500"> Biophysical Characterization of the Tandem FHA Domain Regulatory Module from the Mycobacterium tuberculosis ABC Transporter Rv1747.</td><td width=200><ul><li>Heinkel, F.</li><li>Shen, L.</li><li>Richard-Greenblatt, M.</li><li>Okon, M.</li><li>Bui, J.M.</li><li>Gee, C.L.</li><li>Gay, L.M.</li><li>Alber, T.</li><li>Av-Gay, Y.</li><li>Gsponer, J.</li><li>McIntosh, L.P.</li></ul></td></tr></table></td></tr><tr><td>Rv0467</td><td>icl1</td><td width="300">Involved in glyoxylate bypass (at the first step)  an alternative to the tricarboxylic acid cycle (in bacteria  plants  and fungi) [catalytic activity: isocitrate = succinate + glyoxylate]. Involved in the persistence in the host.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/ws/6wsi/6wsi_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/6WSI">6WSI</a></td><td width="500"> Covalent Inactivation of Mycobacterium tuberculosis Isocitrate Lyase by cis -2,3-Epoxy-Succinic Acid.</td><td width=200><ul><li>Pham, T.V.</li><li>Mellott, D.M.</li><li>Moghadamchargari, Z.</li><li>Chen, K.</li><li>Krieger, I.</li><li>Laganowsky, A.</li><li>Sacchettini, J.C.</li><li>Meek, T.D.</li></ul></td></tr></table></td></tr><tr><td>Rv0223c</td><td>Rv0223c</td><td width="300">Thought to oxidize a wide variety of aliphatic and aromatic aldehydes.</td><td><table><tr><td width="200"><img src=https://cdn.rcsb.org/images/structures/b4/3b4w/3b4w_assembly-1.jpeg width=200/></td><td width="100"><a href="https://www.rcsb.org/structure/3B4W">3B4W</a></td><td width="500"> X-ray crystal structure of aldehyde dehydrogenase from Mycobacterium tuberculosis complexed with NAD+.</td><td width=200><ul><li>Moon, J.H.</li><li>Lyon, A.E.</li><li>Yu, M.</li><li>Hung, L.-W.</li><li>Terwilliger, T.</li><li>Kim, C.-Y.</li></ul></td></tr></table></td></tr></table></body></html>